<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey for Swathi" class="md-header__button md-logo" aria-label="Literature Survey for Swathi" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for Swathi
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              21fcaf99ddc745b1c9ce7be98fe589c812c89ddf
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Immunology/" class="md-tabs__link">
        
  
    
  
  Immunology

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Oncology/" class="md-tabs__link">
        
  
    
  
  Oncology

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey for Swathi" class="md-nav__button md-logo" aria-label="Literature Survey for Swathi" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for Swathi
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Immunology/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Immunology
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Oncology/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Oncology
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-09 09:14:56 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Mitochondrial complex I activity in microglia sustains neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="[This retracts the article DOI: 10.1155/2021/1574109.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67bc7c1a1b13af9e2bdae5652e9164d25f864f22" target='_blank'>Retracted: Enriched Environment Regulates Dendritic Cells to Alleviate Inflammation in Cerebral Infarction Lesions</a></td>
          <td>
            Computational and Mathematical Methods in Medicine
          </td>
          <td>2023-11-01</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The brain needs more energy than other organs in the body. Mitochondria are the generator of vital power in the living organism. Not only do mitochondria sense signals from the outside of a cell, but they also orchestrate the cascade of subcellular events by supplying adenosine-5′-triphosphate (ATP), the biochemical energy. It is known that impaired mitochondrial function and oxidative stress contribute or lead to neuronal damage and degeneration of the brain. This mini-review focuses on addressing how mitochondrial dysfunction and oxidative stress are associated with the pathogenesis of neurodegenerative disorders including Alzheimer’s disease, amyotrophic lateral sclerosis, Huntington’s disease, and Parkinson’s disease. In addition, we discuss state-of-the-art computational models of mitochondrial functions in relation to oxidative stress and neurodegeneration. Together, a better understanding of brain disease-specific mitochondrial dysfunction and oxidative stress can pave the way to developing antioxidant therapeutic strategies to ameliorate neuronal activity and prevent neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fbf34fc8afb8d9e248fe00c0a39c40f4620732c" target='_blank'>Power Failure of Mitochondria and Oxidative Stress in Neurodegeneration and Its Computational Models</a></td>
          <td>
            Junhyuk Woo, Hyesun Cho, YunHee Seol, Soon Ho Kim, Chanhyeok Park, Ali Yousefian-Jazi, Seung Jae Hyeon, Junghee Lee, H. Ryu
          </td>
          <td>2021-02-01</td>
          <td>Antioxidants</td>
          <td>21</td>
          <td>50</td>
        </tr>

        <tr id="Background Ischemic stroke (IS) is a severe neurological disorder with a pathogenesis that remains incompletely understood. Recently, a novel form of cell death known as disulfidptosis has garnered significant attention in the field of ischemic stroke research. This study aims to investigate the mechanistic roles of disulfidptosis-related genes (DRGs) in the context of IS and to examine their correlation with immunopathological features. Methods To enhance our understanding of the mechanistic underpinnings of disulfidptosis in IS, we initially retrieved the expression profile of peripheral blood from human IS patients from the GEO database. We then utilized a suite of machine learning algorithms, including LASSO, random forest, and SVM-RFE, to identify and validate pivotal genes. Furthermore, we developed a predictive nomogram model, integrating multifactorial logistic regression analysis and calibration curves, to evaluate the risk of IS. For the analysis of single-cell sequencing data, we employed a range of analytical tools, such as "Monocle" and "CellChat," to assess the status of immune cell infiltration and to characterize intercellular communication networks. Additionally, we utilized an oxygen–glucose deprivation (OGD) model to investigate the effects of SLC7A11 overexpression on microglial polarization. Results This study successfully identified key genes associated with disulfidptosis and developed a reliable nomogram model using machine learning algorithms to predict the risk of ischemic stroke. Examination of single-cell sequencing data showed a robust correlation between disulfidptosis levels and the infiltration of immune cells. Furthermore, "CellChat" analysis elucidated the intricate characteristics of intercellular communication networks. Notably, the TNF signaling pathway was found to be intimately linked with the disulfidptosis signature in ischemic stroke. In an intriguing finding, the OGD model demonstrated that SLC7A11 expression suppresses M1 polarization while promoting M2 polarization in microglia. Conclusion The significance of our findings lies in their potential to shed light on the pathogenesis of ischemic stroke, particularly by underscoring the pivotal role of disulfidptosis-related genes (DRGs). These insights could pave the way for novel therapeutic strategies targeting DRGs to mitigate the impact of ischemic stroke. Supplementary Information The online version contains supplementary material available at 10.1007/s12017-024-08804-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7a6a2a6301afa935d9e20ab5fd0cfe8dab900c" target='_blank'>Identification of Disulfidptosis-Related Genes in Ischemic Stroke by Combining Single-Cell Sequencing, Machine Learning Algorithms, and In Vitro Experiments</a></td>
          <td>
            Songyun Zhao, Zhuang Hao, Wei Ji, Chao Cheng, Yuankun Liu
          </td>
          <td>2024-09-15</td>
          <td>Neuromolecular Medicine</td>
          <td>7</td>
          <td>14</td>
        </tr>

        <tr id="Astrocytes and microglia react to Aβ plaques, neurofibrillary tangles, and neurodegeneration in the Alzheimer’s disease (AD) brain. Single-nuclei and single-cell RNA-seq have revealed multiple states or subpopulations of these glial cells but lack spatial information. We have developed a methodology of cyclic multiplex fluorescent immunohistochemistry on human postmortem brains and image analysis that enables a comprehensive morphological quantitative characterization of astrocytes and microglia in the context of their spatial relationships with plaques and tangles. Single FFPE sections from the temporal association cortex of control and AD subjects were subjected to 8 cycles of multiplex fluorescent immunohistochemistry, including 7 astroglial, 6 microglial, 1 neuronal, Aβ, and phospho-tau markers. Our analysis pipeline consisted of: (1) image alignment across cycles; (2) background subtraction; (3) manual annotation of 5172 ALDH1L1+ astrocytic and 6226 IBA1+ microglial profiles; (4) local thresholding and segmentation of profiles; (5) machine learning on marker intensity data; and (6) deep learning on image features. Spectral clustering identified three phenotypes of astrocytes and microglia, which we termed “homeostatic,” “intermediate,” and “reactive.” Reactive and, to a lesser extent, intermediate astrocytes and microglia were closely associated with AD pathology (≤ 50 µm). Compared to homeostatic, reactive astrocytes contained substantially higher GFAP and YKL-40, modestly elevated vimentin and TSPO as well as EAAT1, and reduced GS. Intermediate astrocytes had markedly increased EAAT2, moderately increased GS, and intermediate GFAP and YKL-40 levels. Relative to homeostatic, reactive microglia showed increased expression of all markers (CD68, ferritin, MHC2, TMEM119, TSPO), whereas intermediate microglia exhibited increased ferritin and TMEM119 as well as intermediate CD68 levels. Machine learning models applied on either high-plex signal intensity data (gradient boosting machines) or directly on image features (convolutional neural networks) accurately discriminated control vs. AD diagnoses at the single-cell level. Cyclic multiplex fluorescent immunohistochemistry combined with machine learning models holds promise to advance our understanding of the complexity and heterogeneity of glial responses as well as inform transcriptomics studies. Three distinct phenotypes emerged with our combination of markers, thus expanding the classic binary “homeostatic vs. reactive” classification to a third state, which could represent “transitional” or “resilient” glia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30baa2b791d2f94464bd8ccbc6fb674e60c8902c" target='_blank'>Cyclic multiplex fluorescent immunohistochemistry and machine learning reveal distinct states of astrocytes and microglia in normal aging and Alzheimer’s disease</a></td>
          <td>
            C. Muñoz-Castro, Ayush Noori, C. Magdamo, Zhaozhi Li, Jordan D Marks, M. Frosch, Sudeshna Das, B. Hyman, A. Serrano‐Pozo
          </td>
          <td>2022-02-02</td>
          <td>Journal of Neuroinflammation</td>
          <td>22</td>
          <td>184</td>
        </tr>

        <tr id="Microglia are the immune cell in the central nervous system (CNS) and exist in a surveillant state characterized by a ramified form in the healthy brain. In response to brain injury or disease including neurodegenerative diseases, they become activated and change their morphology. Due to known correlation between this activation and neuroinflammation, there is great interest in improved approaches for studying microglial activation in the context of CNS disease mechanisms. One classic approach has utilized Microglia's morphology as one of the key indicators of its activation and correlated with its functional state. More recently microglial activation has been shown to have intrinsic NADH metabolic signatures that are detectable via fluorescence lifetime imaging (FLIM). Despite the promise of morphology and metabolism as key fingerprints of microglial function, they has not been analyzed together due to lack of an appropriate computational framework. Here we present a deep neural network to study the effect of both morphology and FLIM metabolic signatures toward identifying its activation status. Our model is tested on 1, 000+ cells (ground truth generated using LPS treatment) and provides a state-of-the-art framework to identify microglial activation and its role in neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7f55a95c602ede6221ec93232ccc948b0ea701" target='_blank'>A deep learning framework for classifying microglia activation state using morphology and intrinsic fluorescence lifetime data</a></td>
          <td>
            L. Mukherjee, M. A. Sagar, Jonathan N. Ouellette, J. Watters, K. Eliceiri
          </td>
          <td>2022-12-16</td>
          <td>Frontiers in Neuroinformatics</td>
          <td>4</td>
          <td>49</td>
        </tr>

        <tr id="Multiple sclerosis (MS) is a neurodegenerative disease affecting millions worldwide, highlighting the complex relationship between the immune system and the central nervous system. Astrocytes are recognized as significant contributors to the disease's pathogenesis. In this work, a biophysically realistic astrocytic model was created to investigate astrocytes' role in MS development, focusing on their impact on axonal conduction and enhanced sodium channel facilitation in demyelinated axons. Through the advancement of comprehension about the involvement of astrocytes in the pathophysiology of MS, this study explores the processes underlying the disease. The study also examines the morphology of astrocytes and its influence on cellular activity, providing insights into cell instability drivers and the interaction between morphological changes and functional modifications. This approach aims to understand the complex connections between cellular characteristics and physiological attributes, enhancing our understanding of multiple sclerosis and potentially developing groundbreaking therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b4c295918668f31465720399d5d6a4cfabce43" target='_blank'>Deciphering Astroglial Dynamics and Interactions Through Multi-Scale Computational Modeling in Multiple Sclerosis Evolution</a></td>
          <td>
            Chrysoula Tsimperi, Konstantinos Michmizos, Leontios J. Hadjileontiadis
          </td>
          <td>2024-12-31</td>
          <td>Global Clinical Engineering Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Ischemic stroke (IS) represents a significant contributor to morbidity and mortality globally. The relationship between IS and mitochondrial unfolded protein response (UPRmt) was presently uncertain. This study endeavors to explore the fundamental mechanism of UPRmt in IS by utilizing bioinformatics methods. Methods In GSE58294, differentially expressed genes (DEGs) were obtained, which were overlapped with key module genes of UPRmt -related gene ( UPRmt -RGs) for producing candidate genes. The biomarkers were identified from the candidate genes through machine learning, expression validation, and receiver operating characteristic (ROC) curves. In order to verify the biomarkers, reverse transcription-quantitative PCR (RT-qPCR) experiments were performed on human peripheral blood. Subsequently, a predictive nomogram was created to estimate the likelihood of developing IS. Next, the mechanisms and functions related to the biomarkers were explored by enrichment analysis and immune infiltration. In addition, cells enriched with biomarkers were identified, and the biological processes involved in these cells were analyzed through intercellular communication analysis and virtual knockout experiments. Results MCEMP1, CACNA1E, and CLEC4D were identified as biomarkers and subsequently validated by RT-qPCR. RT-qPCR revealed that CLEC4D is the most sensitive biomarker. The nomogram analysis revealed that these biomarkers possess strong diagnostic value. Immune infiltration analysis indicated that all three biomarkers are strongly correlated with neutrophils. Additionally, in the single-cell transcriptome data, these biomarkers were predominantly enriched in neutrophils. Compared to the sham group, the middle cerebral artery occlusion (MCAO) group exhibited enhanced immune-inflammatory responses. Virtual knockout experiments provide preliminary evidence that CLEC4D functions as a regulatory molecule in neutrophil-mediated inflammation, rather than serving merely as a passive marker. Conclusion CLEC4D was identified as the most sensitive biomarker for IS related to UPRmt -RGs, offering a new reference for IS diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3221c100d41eb26882a7f7a4e070068e017b2e" target='_blank'>Comprehensive transcriptomic analysis integrating bulk and single-cell RNA-seq with machine learning to identify and validate mitochondrial unfolded protein response biomarkers in patients with ischemic stroke</a></td>
          <td>
            Lu Zhang, Lei Yue, Peng Jia, Ziqi Cheng, Jiwen Liu
          </td>
          <td>2025-04-17</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
A multitude of high-quality imaging modalities exist that provide structural data at unprecedented levels of detail. Tissue ultrastructure greatly influences the rate of transport of proteins and other molecules that contribute to neurodegeneration. However, our ability to model flow and diffusion processes in the brain lags behind the quality of the neuroimaging data. Here, a new framework will be presented for converting neuroimaging data into computational fluid dynamics (CFD) models in a scalable manner suitable for high performance computing. TEM data from the Allen Institute will illustrate its capabilities.


METHODS
Segmented TEM images were downloaded from the MicronsExplorer public storage cache. Segmented image stack volumes (4 × 4 × 4 µm) were converted into surface meshes using the Lewiner Marching Cubes algorithm. Convection and diffusion along the surface of individual cells was modeled. After each timestep mass was allowed to exchange between neighboring cells, implemented on a high performance computing cluster using the MPI protocol on 40 or more computing cores. Multiple volumes were analyzed simultaneously, with updates of the flux between cells in different volumes occurring at each time step. These steps were performed with new custom CFD code written in C and Julia.


RESULTS
Segmented z-stacks of TEM data from the MICrONS Explorer website were downloaded in 4 × 4 × 4 µm blocks, which typically contain ≈500 distinct cells (Fig. 1A&B). The meshing algorithm leads to perfect matching between adjacent cells (Fig. 1C). The convection-diffusion equations are solved, modeling flow/diffusion of solutes between cells (Fig. 1D&E). Equations are solved for all ≈500 distinct cells in parallel, with concentrations at matching faces re-balanced at each time step (Fig. 2A&B). A manager-worker MPI model allows for solution on an arbitrary number of cores (Fig. 2C). The interfaces between cell surfaces across volumes are also perfectly matched, allowing for adjustment of the flux of mass between volumes at each timestep (Fig. 2D).


CONCLUSIONS
A new modeling framework is described for solution of transport equations on high performance computing clusters for any imaging modality that produces segmented z-stacked images.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/442e21cebdf5c35d5db269b1a675a5bbe438eb87" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            D. Elbert
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
In Alzheimer's disease (AD), specific brain regions become vulnerable to pathology while others remain resilient. New methods of imaging such as highly multiplexed immunofluorescence (MxIF) provide an abundance of spatial information, while analytical techniques like machine learning (ML) can address questions of cellular contributors to this regional vulnerability.


METHOD
We performed MxIF staining for 26 markers and compared postmortem human samples from an AD-susceptible brain area, the prefrontal cortex (PFC, Brodmann's areas 9, 10 or 46) to an AD-resilient brain area, the primary visual cortex (V1, area 17). Subjects included AD (n = 3, clinical dementia rating CDR 3, Thal stages 3-4, Braak stages V-VI), mild cognitive impairment (n = 4, CDR 0.5, Thal stages 1-3, Braak stages I-V), and age-matched healthy controls (n = 5, CDR 0, Thal stages 0-1, Braak stages I-II), including both females and males. We first used a custom Fiji plugin that leverages iterative ML to segment the images via DAPI or HUD signals. For the DAPI segmentation, ML subsequently classifies the cell type based on relevant channels. The second method used QuPath to segment exclusively using DAPI signal followed by cell-type classifications. Segmentations and classifications in QuPath used static thresholds.


RESULT
Using Fiji for HUD segmentation and filtering by the presence of nuclei, we found a decrease specifically in layer 5 PFC neurons from CDR 0 to CDR 0.5. Surprisingly, this decrease did not hold in QuPath when segmenting first using DAPI. Both methods found an increase in GFAP-expressing astrocytes primarily in layer 1. However, the pan-astrocyte marker ALDH1L1 showed an inverse relationship, with higher density found in layers 2 through 6. Both findings held true regardless of brain region or CDR. Finally, in addition to the white matter, both methods found an increase in oligodendrocytes (MBP+, MBP+/HUD+) within layer 4 of V1 and driven by CDR 0.5.


CONCLUSION
Comparisons between the image analysis modalities demonstrate that the method by which cells are segmented and classified can impact the interpretation of MxIF data. Combining knowledge of disease biology with a deeper understanding of when and how to use these techniques will improve data precision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f661168de6c8530f41d73a876647de7773ee3b98" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            Alec K McKendell, Elizabeth McDonough, Lisa Lowery, Samantha Abate, Daniel Meyer, Jennifer Luebke, Patrick R Hof, M. Varghese
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>25</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>