---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-24 09:14:26 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Mitochondrial complex I activity in microglia sustains neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="[This retracts the article DOI: 10.1155/2021/1574109.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67bc7c1a1b13af9e2bdae5652e9164d25f864f22" target='_blank'>Retracted: Enriched Environment Regulates Dendritic Cells to Alleviate Inflammation in Cerebral Infarction Lesions</a></td>
          <td>
            Computational and Mathematical Methods in Medicine
          </td>
          <td>2023-11-01</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7a6a2a6301afa935d9e20ab5fd0cfe8dab900c" target='_blank'>Identification of Disulfidptosis-Related Genes in Ischemic Stroke by Combining Single-Cell Sequencing, Machine Learning Algorithms, and In Vitro Experiments</a></td>
          <td>
            Songyun Zhao, Zhuang Hao, Wei Ji, Chao Cheng, Yuankun Liu
          </td>
          <td>2024-09-15</td>
          <td>Neuromolecular Medicine</td>
          <td>4</td>
          <td>13</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30baa2b791d2f94464bd8ccbc6fb674e60c8902c" target='_blank'>Cyclic multiplex fluorescent immunohistochemistry and machine learning reveal distinct states of astrocytes and microglia in normal aging and Alzheimer’s disease</a></td>
          <td>
            C. Muñoz-Castro, Ayush Noori, C. Magdamo, Zhaozhi Li, Jordan D Marks, M. Frosch, Sudeshna Das, B. Hyman, A. Serrano‐Pozo
          </td>
          <td>2022-02-02</td>
          <td>Journal of Neuroinflammation</td>
          <td>21</td>
          <td>183</td>
        </tr>
    
        <tr id="Microglia are the immune cell in the central nervous system (CNS) and exist in a surveillant state characterized by a ramified form in the healthy brain. In response to brain injury or disease including neurodegenerative diseases, they become activated and change their morphology. Due to known correlation between this activation and neuroinflammation, there is great interest in improved approaches for studying microglial activation in the context of CNS disease mechanisms. One classic approach has utilized Microglia's morphology as one of the key indicators of its activation and correlated with its functional state. More recently microglial activation has been shown to have intrinsic NADH metabolic signatures that are detectable via fluorescence lifetime imaging (FLIM). Despite the promise of morphology and metabolism as key fingerprints of microglial function, they has not been analyzed together due to lack of an appropriate computational framework. Here we present a deep neural network to study the effect of both morphology and FLIM metabolic signatures toward identifying its activation status. Our model is tested on 1, 000+ cells (ground truth generated using LPS treatment) and provides a state-of-the-art framework to identify microglial activation and its role in neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7f55a95c602ede6221ec93232ccc948b0ea701" target='_blank'>A deep learning framework for classifying microglia activation state using morphology and intrinsic fluorescence lifetime data</a></td>
          <td>
            L. Mukherjee, M. A. Sagar, Jonathan N. Ouellette, J. Watters, K. Eliceiri
          </td>
          <td>2022-12-16</td>
          <td>Frontiers in Neuroinformatics</td>
          <td>4</td>
          <td>49</td>
        </tr>
    
        <tr id="BACKGROUND
A multitude of high-quality imaging modalities exist that provide structural data at unprecedented levels of detail. Tissue ultrastructure greatly influences the rate of transport of proteins and other molecules that contribute to neurodegeneration. However, our ability to model flow and diffusion processes in the brain lags behind the quality of the neuroimaging data. Here, a new framework will be presented for converting neuroimaging data into computational fluid dynamics (CFD) models in a scalable manner suitable for high performance computing. TEM data from the Allen Institute will illustrate its capabilities.


METHODS
Segmented TEM images were downloaded from the MicronsExplorer public storage cache. Segmented image stack volumes (4 × 4 × 4 µm) were converted into surface meshes using the Lewiner Marching Cubes algorithm. Convection and diffusion along the surface of individual cells was modeled. After each timestep mass was allowed to exchange between neighboring cells, implemented on a high performance computing cluster using the MPI protocol on 40 or more computing cores. Multiple volumes were analyzed simultaneously, with updates of the flux between cells in different volumes occurring at each time step. These steps were performed with new custom CFD code written in C and Julia.


RESULTS
Segmented z-stacks of TEM data from the MICrONS Explorer website were downloaded in 4 × 4 × 4 µm blocks, which typically contain ≈500 distinct cells (Fig. 1A&B). The meshing algorithm leads to perfect matching between adjacent cells (Fig. 1C). The convection-diffusion equations are solved, modeling flow/diffusion of solutes between cells (Fig. 1D&E). Equations are solved for all ≈500 distinct cells in parallel, with concentrations at matching faces re-balanced at each time step (Fig. 2A&B). A manager-worker MPI model allows for solution on an arbitrary number of cores (Fig. 2C). The interfaces between cell surfaces across volumes are also perfectly matched, allowing for adjustment of the flux of mass between volumes at each timestep (Fig. 2D).


CONCLUSIONS
A new modeling framework is described for solution of transport equations on high performance computing clusters for any imaging modality that produces segmented z-stacked images.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/442e21cebdf5c35d5db269b1a675a5bbe438eb87" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            D. Elbert
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="BACKGROUND
In Alzheimer's disease (AD), specific brain regions become vulnerable to pathology while others remain resilient. New methods of imaging such as highly multiplexed immunofluorescence (MxIF) provide an abundance of spatial information, while analytical techniques like machine learning (ML) can address questions of cellular contributors to this regional vulnerability.


METHOD
We performed MxIF staining for 26 markers and compared postmortem human samples from an AD-susceptible brain area, the prefrontal cortex (PFC, Brodmann's areas 9, 10 or 46) to an AD-resilient brain area, the primary visual cortex (V1, area 17). Subjects included AD (n = 3, clinical dementia rating CDR 3, Thal stages 3-4, Braak stages V-VI), mild cognitive impairment (n = 4, CDR 0.5, Thal stages 1-3, Braak stages I-V), and age-matched healthy controls (n = 5, CDR 0, Thal stages 0-1, Braak stages I-II), including both females and males. We first used a custom Fiji plugin that leverages iterative ML to segment the images via DAPI or HUD signals. For the DAPI segmentation, ML subsequently classifies the cell type based on relevant channels. The second method used QuPath to segment exclusively using DAPI signal followed by cell-type classifications. Segmentations and classifications in QuPath used static thresholds.


RESULT
Using Fiji for HUD segmentation and filtering by the presence of nuclei, we found a decrease specifically in layer 5 PFC neurons from CDR 0 to CDR 0.5. Surprisingly, this decrease did not hold in QuPath when segmenting first using DAPI. Both methods found an increase in GFAP-expressing astrocytes primarily in layer 1. However, the pan-astrocyte marker ALDH1L1 showed an inverse relationship, with higher density found in layers 2 through 6. Both findings held true regardless of brain region or CDR. Finally, in addition to the white matter, both methods found an increase in oligodendrocytes (MBP+, MBP+/HUD+) within layer 4 of V1 and driven by CDR 0.5.


CONCLUSION
Comparisons between the image analysis modalities demonstrate that the method by which cells are segmented and classified can impact the interpretation of MxIF data. Combining knowledge of disease biology with a deeper understanding of when and how to use these techniques will improve data precision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f661168de6c8530f41d73a876647de7773ee3b98" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            Alec K McKendell, Elizabeth McDonough, Lisa Lowery, Samantha Abate, Daniel Meyer, Jennifer Luebke, Patrick R Hof, M. Varghese
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr id="Ferroptosis is crucial in neuronal cell death, associated with various neurological disorders. However, the role of ferroptosis-related genes (FRGs) in ischaemic stroke (IS) has yet to be well elucidated. We downloaded IS-related gene information and FRGs from the Gene Expression Omnibus (GEO) and Ferritin databases (FerrDb). 22 IS-related DE-FRGs were obtained. Functional enrichment analysis showed that these genes involve multiple regulatory pathways related to IS pathogenesis, including redox, amino acid metabolism, and cell cycle. Subsequently, DUOX2, MDM2, EGFR, MAP3K14, and TRIM46 were identified as core marker genes among the 22 DE-FRGs by lasso and SVM-RFE algorithms. The construction of the nomogram using the five marker genes had excellent diagnostic value. In addition, CIBERSORT analysis showed that changes in the immune microenvironment of IS patients might be associated with TRIM46, MDM2, and DUOX2. In addition, a total of 66 drugs targeting two characteristic FRGs were obtained. The ceRNA networks revealed complex regulatory relationships based on characteristic FRGs. These findings provide new insights into the diagnosis and treatment of IS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bea31b38fe4842d5c6a5da6e332fb25b8c946a51" target='_blank'>Identification of five characteristic ferroptosis-related genes as novel biomarkers for ischemic stroke using bioinformatic and machine learning strategies</a></td>
          <td>
            Xin Feng, Chunhua Huang, Ye Lin, Zhongbo Xu
          </td>
          <td>2023-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d22c0e1c57e1637a0c74b7f59109ccf4f609752" target='_blank'>Identifying neuroinflammation in omics datasets with a computational biological network model of reactive astrogliosis</a></td>
          <td>
            M. Barkhuizen, C. Mathis, M. Peitsch, M. Talikka
          </td>
          <td>2021-12-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>62</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc05a1e9efded8d9259f6f958732b96404f2a30e" target='_blank'>Dynamic changes in pyroptosis following spinal cord injury and the identification of crucial molecular signatures through machine learning and single-cell sequencing.</a></td>
          <td>
            Chuang Li, Qingyang Li, Ruizhi Jiang, Chi Zhang, Enling Qi, Mingxin Wu, Mingzhe Zhang, Hua Zhao, Fenge Zhao, Hengxing Zhou
          </td>
          <td>2024-08-01</td>
          <td>Journal of pharmaceutical and biomedical analysis</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>