---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-19 09:15:19 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Mitochondrial complex I activity in microglia sustains neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="The brain needs more energy than other organs in the body. Mitochondria are the generator of vital power in the living organism. Not only do mitochondria sense signals from the outside of a cell, but they also orchestrate the cascade of subcellular events by supplying adenosine-5′-triphosphate (ATP), the biochemical energy. It is known that impaired mitochondrial function and oxidative stress contribute or lead to neuronal damage and degeneration of the brain. This mini-review focuses on addressing how mitochondrial dysfunction and oxidative stress are associated with the pathogenesis of neurodegenerative disorders including Alzheimer’s disease, amyotrophic lateral sclerosis, Huntington’s disease, and Parkinson’s disease. In addition, we discuss state-of-the-art computational models of mitochondrial functions in relation to oxidative stress and neurodegeneration. Together, a better understanding of brain disease-specific mitochondrial dysfunction and oxidative stress can pave the way to developing antioxidant therapeutic strategies to ameliorate neuronal activity and prevent neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fbf34fc8afb8d9e248fe00c0a39c40f4620732c" target='_blank'>Power Failure of Mitochondria and Oxidative Stress in Neurodegeneration and Its Computational Models</a></td>
          <td>
            Junhyuk Woo, Hyesun Cho, YunHee Seol, Soon Ho Kim, Chanhyeok Park, Ali Yousefian-Jazi, Seung Jae Hyeon, Junghee Lee, H. Ryu
          </td>
          <td>2021-02-01</td>
          <td>Antioxidants</td>
          <td>21</td>
          <td>50</td>
        </tr>
    
        <tr id="Astrocytes and microglia react to Aβ plaques, neurofibrillary tangles, and neurodegeneration in the Alzheimer’s disease (AD) brain. Single-nuclei and single-cell RNA-seq have revealed multiple states or subpopulations of these glial cells but lack spatial information. We have developed a methodology of cyclic multiplex fluorescent immunohistochemistry on human postmortem brains and image analysis that enables a comprehensive morphological quantitative characterization of astrocytes and microglia in the context of their spatial relationships with plaques and tangles. Single FFPE sections from the temporal association cortex of control and AD subjects were subjected to 8 cycles of multiplex fluorescent immunohistochemistry, including 7 astroglial, 6 microglial, 1 neuronal, Aβ, and phospho-tau markers. Our analysis pipeline consisted of: (1) image alignment across cycles; (2) background subtraction; (3) manual annotation of 5172 ALDH1L1+ astrocytic and 6226 IBA1+ microglial profiles; (4) local thresholding and segmentation of profiles; (5) machine learning on marker intensity data; and (6) deep learning on image features. Spectral clustering identified three phenotypes of astrocytes and microglia, which we termed “homeostatic,” “intermediate,” and “reactive.” Reactive and, to a lesser extent, intermediate astrocytes and microglia were closely associated with AD pathology (≤ 50 µm). Compared to homeostatic, reactive astrocytes contained substantially higher GFAP and YKL-40, modestly elevated vimentin and TSPO as well as EAAT1, and reduced GS. Intermediate astrocytes had markedly increased EAAT2, moderately increased GS, and intermediate GFAP and YKL-40 levels. Relative to homeostatic, reactive microglia showed increased expression of all markers (CD68, ferritin, MHC2, TMEM119, TSPO), whereas intermediate microglia exhibited increased ferritin and TMEM119 as well as intermediate CD68 levels. Machine learning models applied on either high-plex signal intensity data (gradient boosting machines) or directly on image features (convolutional neural networks) accurately discriminated control vs. AD diagnoses at the single-cell level. Cyclic multiplex fluorescent immunohistochemistry combined with machine learning models holds promise to advance our understanding of the complexity and heterogeneity of glial responses as well as inform transcriptomics studies. Three distinct phenotypes emerged with our combination of markers, thus expanding the classic binary “homeostatic vs. reactive” classification to a third state, which could represent “transitional” or “resilient” glia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30baa2b791d2f94464bd8ccbc6fb674e60c8902c" target='_blank'>Cyclic multiplex fluorescent immunohistochemistry and machine learning reveal distinct states of astrocytes and microglia in normal aging and Alzheimer’s disease</a></td>
          <td>
            C. Muñoz-Castro, Ayush Noori, C. Magdamo, Zhaozhi Li, Jordan D Marks, M. Frosch, Sudeshna Das, B. Hyman, A. Serrano‐Pozo
          </td>
          <td>2022-02-02</td>
          <td>Journal of Neuroinflammation</td>
          <td>22</td>
          <td>184</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4310b87b8933d306be9d8481edadbbe192917920" target='_blank'>Identification of mitophagy-related genes and analysis of immune infiltration in the astrocytes based on machine learning in the pathogenesis of major depressive disorder.</a></td>
          <td>
            Lan Lei, Cong-Ya Chen, Yu-Fei Wang, Yi Zhang
          </td>
          <td>2024-09-01</td>
          <td>Journal of affective disorders</td>
          <td>5</td>
          <td>5</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c8de4b41f2f1be7b8045a14a0453ff1ad21d6b" target='_blank'>Neuroinflammation — using big data to inform clinical practice</a></td>
          <td>
            C. Dendrou, G. McVean, L. Fugger
          </td>
          <td>2016-12-01</td>
          <td>Nature Reviews Neurology</td>
          <td>30</td>
          <td>94</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d22c0e1c57e1637a0c74b7f59109ccf4f609752" target='_blank'>Identifying neuroinflammation in omics datasets with a computational biological network model of reactive astrogliosis</a></td>
          <td>
            M. Barkhuizen, C. Mathis, M. Peitsch, M. Talikka
          </td>
          <td>2021-12-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>62</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc05a1e9efded8d9259f6f958732b96404f2a30e" target='_blank'>Dynamic changes in pyroptosis following spinal cord injury and the identification of crucial molecular signatures through machine learning and single-cell sequencing.</a></td>
          <td>
            Chuang Li, Qingyang Li, Ruizhi Jiang, Chi Zhang, Enling Qi, Mingxin Wu, Mingzhe Zhang, Hua Zhao, Fenge Zhao, Hengxing Zhou
          </td>
          <td>2024-08-01</td>
          <td>Journal of pharmaceutical and biomedical analysis</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr id="Numerous sub-cellular through system-level disturbances have been identified in over 1300 articles examining the superoxide dismutase-1 guanine 93 to alanine (SOD1-G93A) transgenic mouse amyotrophic lateral sclerosis (ALS) pathophysiology. Manual assessment of such a broad literature base is daunting. We performed a comprehensive informatics-based systematic review or ‘field analysis’ to agnostically compute and map the current state of the field. Text mining of recaptured articles was used to quantify published data topic breadth and frequency. We constructed a nine-category pathophysiological function-based ontology to systematically organize and quantify the field's primary data. Results demonstrated that the distribution of primary research belonging to each category is: systemic measures an motor function, 59%; inflammation, 46%; cellular energetics, 37%; proteomics, 31%; neural excitability, 22%; apoptosis, 20%; oxidative stress, 18%; aberrant cellular chemistry, 14%; axonal transport, 10%. We constructed a SOD1-G93A field map that visually illustrates and categorizes the 85% most frequently assessed sub-topics. Finally, we present the literature-cited significance of frequently published terms and uncover thinly investigated areas. In conclusion, most articles individually examine at least two categories, which is indicative of the numerous underlying pathophysiological interrelationships. An essential future path is examination of cross-category pathophysiological interrelationships and their co-correspondence to homeostatic regulation and disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bc13bee12af61816f4c5a101b44a1a7eb5527ee" target='_blank'>State of the field: An informatics-based systematic review of the SOD1-G93A amyotrophic lateral sclerosis transgenic mouse model</a></td>
          <td>
            Renaid B. Kim, Cameron Irvin, Keval R. Tilva, Cassie S. Mitchell
          </td>
          <td>2015-05-22</td>
          <td>Amyotrophic Lateral Sclerosis & Frontotemporal Degeneration</td>
          <td>41</td>
          <td>19</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02a4e4801d9cdc43d9ea072c79e5b55d50a8852e" target='_blank'>MIC-MAC: High-throughput analysis of three-dimensional microglia morphology in mammalian brains</a></td>
          <td>
            L. Salamanca, N. Mechawar, Keith K. Murai, Rudi Balling, David S Bouvier, A. Skupin
          </td>
          <td>2018-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>