---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-17 09:14:28 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Mitochondrial complex I activity in microglia sustains neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="[This retracts the article DOI: 10.1155/2021/1574109.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67bc7c1a1b13af9e2bdae5652e9164d25f864f22" target='_blank'>Retracted: Enriched Environment Regulates Dendritic Cells to Alleviate Inflammation in Cerebral Infarction Lesions</a></td>
          <td>
            Computational and Mathematical Methods in Medicine
          </td>
          <td>2023-11-01</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7a6a2a6301afa935d9e20ab5fd0cfe8dab900c" target='_blank'>Identification of Disulfidptosis-Related Genes in Ischemic Stroke by Combining Single-Cell Sequencing, Machine Learning Algorithms, and In Vitro Experiments</a></td>
          <td>
            Songyun Zhao, Zhuang Hao, Wei Ji, Chao Cheng, Yuankun Liu
          </td>
          <td>2024-09-15</td>
          <td>Neuromolecular Medicine</td>
          <td>4</td>
          <td>13</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30baa2b791d2f94464bd8ccbc6fb674e60c8902c" target='_blank'>Cyclic multiplex fluorescent immunohistochemistry and machine learning reveal distinct states of astrocytes and microglia in normal aging and Alzheimer’s disease</a></td>
          <td>
            C. Muñoz-Castro, Ayush Noori, C. Magdamo, Zhaozhi Li, Jordan D Marks, M. Frosch, Sudeshna Das, B. Hyman, A. Serrano‐Pozo
          </td>
          <td>2022-02-02</td>
          <td>Journal of Neuroinflammation</td>
          <td>21</td>
          <td>183</td>
        </tr>
    
        <tr id="Multiple sclerosis (MS) is a neurodegenerative disease affecting millions worldwide, highlighting the complex relationship between the immune system and the central nervous system. Astrocytes are recognized as significant contributors to the disease's pathogenesis. In this work, a biophysically realistic astrocytic model was created to investigate astrocytes' role in MS development, focusing on their impact on axonal conduction and enhanced sodium channel facilitation in demyelinated axons. Through the advancement of comprehension about the involvement of astrocytes in the pathophysiology of MS, this study explores the processes underlying the disease. The study also examines the morphology of astrocytes and its influence on cellular activity, providing insights into cell instability drivers and the interaction between morphological changes and functional modifications. This approach aims to understand the complex connections between cellular characteristics and physiological attributes, enhancing our understanding of multiple sclerosis and potentially developing groundbreaking therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b4c295918668f31465720399d5d6a4cfabce43" target='_blank'>Deciphering Astroglial Dynamics and Interactions Through Multi-Scale Computational Modeling in Multiple Sclerosis Evolution</a></td>
          <td>
            Chrysoula Tsimperi, Konstantinos Michmizos, Leontios J. Hadjileontiadis
          </td>
          <td>2024-12-31</td>
          <td>Global Clinical Engineering Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="BACKGROUND
In Alzheimer's disease (AD), specific brain regions become vulnerable to pathology while others remain resilient. New methods of imaging such as highly multiplexed immunofluorescence (MxIF) provide an abundance of spatial information, while analytical techniques like machine learning (ML) can address questions of cellular contributors to this regional vulnerability.


METHOD
We performed MxIF staining for 26 markers and compared postmortem human samples from an AD-susceptible brain area, the prefrontal cortex (PFC, Brodmann's areas 9, 10 or 46) to an AD-resilient brain area, the primary visual cortex (V1, area 17). Subjects included AD (n = 3, clinical dementia rating CDR 3, Thal stages 3-4, Braak stages V-VI), mild cognitive impairment (n = 4, CDR 0.5, Thal stages 1-3, Braak stages I-V), and age-matched healthy controls (n = 5, CDR 0, Thal stages 0-1, Braak stages I-II), including both females and males. We first used a custom Fiji plugin that leverages iterative ML to segment the images via DAPI or HUD signals. For the DAPI segmentation, ML subsequently classifies the cell type based on relevant channels. The second method used QuPath to segment exclusively using DAPI signal followed by cell-type classifications. Segmentations and classifications in QuPath used static thresholds.


RESULT
Using Fiji for HUD segmentation and filtering by the presence of nuclei, we found a decrease specifically in layer 5 PFC neurons from CDR 0 to CDR 0.5. Surprisingly, this decrease did not hold in QuPath when segmenting first using DAPI. Both methods found an increase in GFAP-expressing astrocytes primarily in layer 1. However, the pan-astrocyte marker ALDH1L1 showed an inverse relationship, with higher density found in layers 2 through 6. Both findings held true regardless of brain region or CDR. Finally, in addition to the white matter, both methods found an increase in oligodendrocytes (MBP+, MBP+/HUD+) within layer 4 of V1 and driven by CDR 0.5.


CONCLUSION
Comparisons between the image analysis modalities demonstrate that the method by which cells are segmented and classified can impact the interpretation of MxIF data. Combining knowledge of disease biology with a deeper understanding of when and how to use these techniques will improve data precision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f661168de6c8530f41d73a876647de7773ee3b98" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            Alec K McKendell, Elizabeth McDonough, Lisa Lowery, Samantha Abate, Daniel Meyer, Jennifer Luebke, Patrick R Hof, M. Varghese
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c8de4b41f2f1be7b8045a14a0453ff1ad21d6b" target='_blank'>Neuroinflammation — using big data to inform clinical practice</a></td>
          <td>
            C. Dendrou, G. McVean, L. Fugger
          </td>
          <td>2016-12-01</td>
          <td>Nature Reviews Neurology</td>
          <td>30</td>
          <td>93</td>
        </tr>
    
        <tr id="Microglia, the resident immune cells of the brain, exhibit complex and diverse phenotypes depending on their physiological context including brain regions and disease states. While single-cell RNA-sequencing has recently resolved this heterogeneity, it does not capture tissue and intercellular mechanisms. Our complementary imaging-based approach enables automatic 3D-morphology characterization and classification of thousands of individual microglia in situ and revealed species- and disease-specific morphological phenotypes in mouse and human brain samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02a4e4801d9cdc43d9ea072c79e5b55d50a8852e" target='_blank'>MIC-MAC: High-throughput analysis of three-dimensional microglia morphology in mammalian brains</a></td>
          <td>
            L. Salamanca, N. Mechawar, Keith K. Murai, Rudi Balling, David S Bouvier, A. Skupin
          </td>
          <td>2018-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>
    
        <tr id="As rapid responders to their environments, microglia engage in functions that are mirrored by their cellular morphology. Microglia are classically thought to exhibit a ramified morphology under homeostatic conditions which switches to an ameboid form during inflammatory conditions. However, microglia display a wide spectrum of morphologies outside of this dichotomy, including rod-like, ramified, ameboid, and hypertrophic states, which have been observed across brain regions, neurodevelopmental timepoints, and various pathological contexts. We used dimensionality reduction and clustering approaches to consider contributions of multiple morphology measures together to define a spectrum of microglial morphological states. Using ImageJ tools, we first developed a semi-automated approach to characterize 27 morphology features from hundreds to thousands of individual microglial cells in a brain subregion-specific manner. Within this pool of morphology measures, we defined distinct sets of highly correlated features that describe different aspects of morphology, including branch length, branching complexity, territory span, and cell circularity. When considered together, these sets of features drove different morphological clusters. Furthermore, our analysis toolset captured morphological states similarly and robustly when applied to independent datasets and using different immunofluorescent markers for microglia. We have compiled our morphology analysis pipeline into an accessible, easy to use, and fully open-source ImageJ macro and R package that the neuroscience community can expand upon and directly apply to their own analyses. Outcomes from this work will supply the field with new tools to systematically evaluate the heterogeneity of microglia morphological states across various experimental models and research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2dd5433a20a8b0dada8fe60f4e507d656fed823" target='_blank'>Development of a High-Throughput Pipeline to Characterize Microglia Morphological States at a Single-Cell Resolution</a></td>
          <td>
            Jennifer Kim, Paul Pavlidis, A. Ciernia
          </td>
          <td>2023-11-05</td>
          <td>eNeuro</td>
          <td>4</td>
          <td>10</td>
        </tr>
    
        <tr id="Automated computational analysis techniques utilizing machine learning have been demonstrated to be able to extract more data from different imaging modalities compared to traditional analysis techniques. One new approach is to use machine learning techniques to existing multiphoton imaging modalities to better interpret intrinsically fluorescent cellular signals to characterize different cell types. Fluorescence Lifetime Imaging Microscopy (FLIM) is a high-resolution quantitative imaging tool that can detect metabolic cellular signatures based on the lifetime variations of intrinsically fluorescent metabolic co-factors such as nicotinamide adenine dinucleotide [NAD(P)H]. NAD(P)H lifetime-based discrimination techniques have previously been used to develop metabolic cell signatures for diverse cell types including immune cells such as macrophages. However, FLIM could be even more effective in characterizing cell types if machine learning was used to classify cells by utilizing FLIM parameters for classification. Here, we demonstrate the potential for FLIM-based, label-free NAD(P)H imaging to distinguish different cell types using Artificial Neural Network (ANN)-based machine learning. For our biological use case, we used the challenge of differentiating microglia from other glia cell types in the brain. Microglia are the resident macrophages of the brain and spinal cord and play a critical role in maintaining the neural environment and responding to injury. Microglia are challenging to identify as most fluorescent labeling approaches cross-react with other immune cell types, are often insensitive to activation state, and require the use of multiple specialized antibody labels. Furthermore, the use of these extrinsic antibody labels prevents application in in vivo animal models and possible future clinical adaptations such as neurodegenerative pathologies. With the ANN-based NAD(P)H FLIM analysis approach, we found that microglia in cell culture mixed with other glial cells can be identified with more than 0.9 True Positive Rate (TPR). We also extended our approach to identify microglia in fixed brain tissue with a TPR of 0.79. In both cases the False Discovery Rate was around 30%. This method can be further extended to potentially study and better understand microglia’s role in neurodegenerative disease with improved detection accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e963cc09f3307ba752ca0fffa7efed479cd6921f" target='_blank'>Machine Learning Methods for Fluorescence Lifetime Imaging (FLIM) Based Label-Free Detection of Microglia</a></td>
          <td>
            M. A. Sagar, Kevin P. Cheng, Jonathan N. Ouellette, Justin C. Williams, J. Watters, K. Eliceiri
          </td>
          <td>2020-09-03</td>
          <td>Frontiers in Neuroscience</td>
          <td>23</td>
          <td>49</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>