---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-14 09:21:25 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Mitochondrial complex I activity in microglia sustains neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="BACKGROUND
A multitude of high-quality imaging modalities exist that provide structural data at unprecedented levels of detail. Tissue ultrastructure greatly influences the rate of transport of proteins and other molecules that contribute to neurodegeneration. However, our ability to model flow and diffusion processes in the brain lags behind the quality of the neuroimaging data. Here, a new framework will be presented for converting neuroimaging data into computational fluid dynamics (CFD) models in a scalable manner suitable for high performance computing. TEM data from the Allen Institute will illustrate its capabilities.


METHODS
Segmented TEM images were downloaded from the MicronsExplorer public storage cache. Segmented image stack volumes (4 × 4 × 4 µm) were converted into surface meshes using the Lewiner Marching Cubes algorithm. Convection and diffusion along the surface of individual cells was modeled. After each timestep mass was allowed to exchange between neighboring cells, implemented on a high performance computing cluster using the MPI protocol on 40 or more computing cores. Multiple volumes were analyzed simultaneously, with updates of the flux between cells in different volumes occurring at each time step. These steps were performed with new custom CFD code written in C and Julia.


RESULTS
Segmented z-stacks of TEM data from the MICrONS Explorer website were downloaded in 4 × 4 × 4 µm blocks, which typically contain ≈500 distinct cells (Fig. 1A&B). The meshing algorithm leads to perfect matching between adjacent cells (Fig. 1C). The convection-diffusion equations are solved, modeling flow/diffusion of solutes between cells (Fig. 1D&E). Equations are solved for all ≈500 distinct cells in parallel, with concentrations at matching faces re-balanced at each time step (Fig. 2A&B). A manager-worker MPI model allows for solution on an arbitrary number of cores (Fig. 2C). The interfaces between cell surfaces across volumes are also perfectly matched, allowing for adjustment of the flux of mass between volumes at each timestep (Fig. 2D).


CONCLUSIONS
A new modeling framework is described for solution of transport equations on high performance computing clusters for any imaging modality that produces segmented z-stacked images.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/442e21cebdf5c35d5db269b1a675a5bbe438eb87" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            D. Elbert
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="BACKGROUND
Haplotypes are patterns of nucleotides in close proximity along a chromosome that are passed together across time and space. These patterns dictate the specific properties of proteins produced and the regulation of this production. General haplotype inference methods phase all provided genotypes within the region of interest into two haplotypes without regard for the ages or evolutionary impact of each mutation, thus force the inclusion of more recent and/or neutral mutations. Alternatively, BlocBuster (www.blocbuster.org) identifies partial haplotypes that include only highly correlated nucleotides using a unique correlation metric that anticipates heterogeneity of the individuals and an expanded network scaffolding. Given the heterogeneous nature of late-onset Alzheimer disease (AD), we utilize BlocBuster to identify genetic risk patterns.


METHOD
We analyzed WGS_Omni2.5M_20140220 data for chromosome 19 provided in the Alzheimer's Disease Neuroimaging Initiative (ADNI) database(adni.loni.usc.edu). The data were arbitrarily split into 70% Discovery (42 AD, 69 MCI, 87 control) and 30% Validation (18 AD, 29 MCI, 37 control). BlocBuster builds a network model using AD, MCI, and control individuals (without labeling), where nodes represent SNP alleles and edges represent pairwise correlations between alleles, extracts clusters of inter-correlated SNP alleles representing a partial haplotype, then tests the pattern for association with the given phenotype.


RESULT
BlocBuster identified two patterns that are significantly associated with AD in both the Discovery and the independent Validation datasets(Table 1). The first represents a variant of the RUVBL2 gene and the second is a large allelic combination beginning upstream from DIRAS1 and spanning across SLC39A3, SGTA, and THOP1(Table 2).


CONCLUSION
The network model was created blindly using all individuals in the Discovery dataset, yet two of the clusters were significantly associated with AD risk in both the Discovery and Validation data. Interestingly, THOP1 is a zinc-activated oligopeptidase that is capable of cleaving beta-amyloid precursor protein into amyloidogenic peptides. Furthermore, SLC39A3 is predicted to enable zinc transport across the membrane, SGTA is suggested to play a role in aggregate formation in neurodegenerative diseases, and methylation of DIRAS1 is associated with AD risk. These results identify interesting signatures that may pinpoint specific AD biological pathways, motivating further validation efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffaf6272eeb42929eee055d87ec2ca9b125c821c" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            S. Climer
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>14</td>
        </tr>
    
        <tr id="The Alzheimer's Disease Sequencing Project (ADSP) has used whole genome sequencing, computational approaches, and epidemiological and statistical expertise to accelerate understanding of the fundamental mechanisms of Alzheimer disease and related disorders (ADRD). Going forward, the ADSP will continue growing to over 110,000 participants to further dissect the risk for AD. To move towards novel insights, the ADSP is adopting new approaches while leveraging the rich existing data. New technologies such as long-read sequencing will reveal complex structural variants in diverse genetic ancestries and enable discovery of novel variants for further exploration. The ADSP Functional Genomics Consortium will expand their work in the generation and integration of multiomic data to better elucidate underlying disease mechanisms thereby enhancing therapeutic target discovery. Harnessing the ADSP data resource will allow examination of genetic heterogeneity by ancestry, identifying both shared and ancestry-specific genetic variants that modulate AD risk, encompassing variants of all types using a variety of new statistical approaches and integration with large AD-specific multiomics resources such as through xQTL projects. The phenotypic heterogeneity of AD will also be examined through the collection and harmonization of emerging AD fluid and imaging biomarkers and cognitive testing through the efforts of the ADSP Phenotype Harmonization Consortium. The ADSP AI/ML consortium will develop new analysis approaches for many of these tasks and will construct comprehensive risk models for AD by incorporating genetics, endophenotypes, biomarkers, and social determinants of health. Finally, as we enter an era of new Alzheimer's therapeutics, the ADSP is poised to uncover genetic predictors of treatment response and adverse events as these are adopted in the general population, enabling personalized medicine approaches for maximizing benefit and minimizing harm. Through expanding sample diversity, integrating diverse datasets, applying cutting-edge technologies, and forging global collaborative networks, the Alzheimer's Disease Sequencing Project aims to achieve an unprecedented understanding of Alzheimer's disease genetics-ultimately translating genetic insights into life-changing treatments and prevention strategies worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3241006b4767a84665b0befc88fc4189c121ba25" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            William S Bush, Anthony J Griswold
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef48d25a397bb27588186f4c6bb49745be2c1b" target='_blank'>NCBI News, July 2010</a></td>
          <td>
            Peter S. Cooper, Dawn Lipshultz
          </td>
          <td>2010-07-31</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7b7a38cc5b24f63b5d71714bfb46813e06a56f" target='_blank'>Rapid responses</a></td>
          <td>
            Ruth Macklin
          </td>
          <td>2001-05-01</td>
          <td>Archives of Disease in Childhood</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7b7a38cc5b24f63b5d71714bfb46813e06a56f" target='_blank'>Rapid responses</a></td>
          <td>
            Ruth Macklin
          </td>
          <td>2001-05-01</td>
          <td>Archives of Disease in Childhood</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39e4a41252dbf836ccd7c717542f5551024f59d9" target='_blank'>Guides: Alerting Services: Table of Contents, Saved Search, Citation Alerts, & RSS Feeds: Table of Contents Alerts</a></td>
          <td>
            Patty Lynn
          </td>
          <td>2014-04-23</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57c5f7089fb8bb0cf1d7fad1e7a0676e703b8b9b" target='_blank'>Guides: Alerting Services: Table of Contents, Saved Search, Citation Alerts, & RSS Feeds: About Alerts</a></td>
          <td>
            Patty Lynn
          </td>
          <td>2014-04-23</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="This week, The Journal of Neuroscience has taken the next step in reinforcing our commitment to open science by initiating a direct submission process for manuscripts posted on bioRxiv. With this process, preprints can be submitted for peer review at JNeurosci with one click from the bioRxiv site,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70da4cf8cf5b121f08cfe7c6ffe4c430c92039e9" target='_blank'>Direct Submissions from bioRxiv</a></td>
          <td>
            M. Picciotto
          </td>
          <td>2017-01-11</td>
          <td>The Journal of Neuroscience</td>
          <td>1</td>
          <td>83</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>