---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-07 09:16:42 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Human immunity relies on the coordinated responses of many cellular subsets and functional states. Inter-individual variations in cellular composition and communication could thus potentially alter host protection. Here, we explore this hypothesis by applying single-cell RNA-sequencing to examine viral responses among the dendritic cells (DCs) of three elite controllers (ECs) of HIV-1 infection. To overcome the potentially confounding effects of donor-to-donor variability, we present a generally applicable computational framework for identifying reproducible patterns in gene expression across donors who share a unifying classification. Applying it, we discover a highly functional antiviral DC state in ECs whose fractional abundance after in vitro exposure to HIV-1 correlates with higher CD4+ T cell counts and lower HIV-1 viral loads, and that effectively primes polyfunctional T cell responses in vitro. By integrating information from existing genomic databases into our reproducibility-based analysis, we identify and validate select immunomodulators that increase the fractional abundance of this state in primary peripheral blood mononuclear cells from healthy individuals in vitro. Overall, our results demonstrate how single-cell approaches can reveal previously unappreciated, yet important, immune behaviors and empower rational frameworks for modulating systems-level immune responses that may prove therapeutically and prophylactically useful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdaaa8164fd46945764987c98bec307fda098c2" target='_blank'>A Reproducibility-Based Computational Framework Identifies an Inducible, Enhanced Antiviral State in Dendritic Cells from HIV-1 Elite Controllers</a></td>
          <td>
            E. Martín-Gayo, Michael Cole, Kellie E. Kolb, Zhengyu Ouyang, Jacqueline Cronin, S. Kazer, J. Ordovas-Montañes, M. Lichterfeld, B. Walker, N. Yosef, A. Shalek, Xu G. Yu
          </td>
          <td>2018-01-29</td>
          <td>Genome Biology</td>
          <td>44</td>
          <td>159</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>432</td>
          <td>102</td>
        </tr>
    
        <tr id="
 To improve diagnoses and further our understanding of functional defects in subjects with primary immunodeficiency (PI) diseases, we developed a new screen to broadly study the signaling of defective human immune responses to infections. Using time-of-flight mass cytometry (CyTOF) and phospho-specific antibodies, we simultaneously examined the major signaling pathways of all circulating innate and adaptive immune cell types in response to cytokines and TLR agonists to identify signaling responses. We studied healthy controls and developed software tools to generate a systematic map of signaling responses. CyTOF successfully recapitulated established signaling responses, and further, elucidated new pathways that would not have been appreciated with traditional studies. Signaling map of controls was compared to maps generated by analyzing PI subjects with gain-of-function STAT1 and autosomal dominant STAT3 deficiency diseases. CyTOF allowed us to perform unbiased studies to uncover many ex-vivo signaling defects simultaneously to better understand how known genetic etiologies manifest functionally, and exactly which cell subtypes are affected. We identified more glaring defective signaling responses in these subjects aside from the canonical defects, elucidating the true breadth of cell subtypes and pathways affected. Broad, unbiased, functional analysis can complement whole exome sequencing to facilitate identification of novel or rare immune diseases. Comprehensive view of signaling across all immune cell subsets facilitates learning more about abnormal signaling patterns that confer susceptibility to infection or autoimmunity, improving our understanding of immune responses and can lead to new therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2823684aa13e4e2575d6d1da7cd6194c358c3ce5" target='_blank'>Systems Biology Approach to Uncover Signaling Defects in Primary Immunodeficiency Diseases</a></td>
          <td>
            Jeff Choi, R. Fernandez, H. Maecker, M. Butte
          </td>
          <td>2016-05-01</td>
          <td>The Journal of Immunology</td>
          <td>2</td>
          <td>77</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8abba679e2263c0dd6b357ce0b9b74a569cd8b2" target='_blank'>Ten computational challenges in human virome studies</a></td>
          <td>
            Yifan Wu, Yousong Peng
          </td>
          <td>2024-02-23</td>
          <td>Virologica Sinica</td>
          <td>2</td>
          <td>2</td>
        </tr>
    
        <tr id="Summary
In solid-organ transplant recipients, a delicate balance between immunosuppression and immunocompetence must be achieved, which can be difficult to monitor in real-time. Shotgun sequencing of cell-free DNA (cfDNA) has been recently proposed as a new way to indirectly assess immune function in transplant recipients through analysis of the status of the human virome. To facilitate exploration of the utility of the human virome as an indicator of immune status, and to enable rapid, straightforward analyses by clinicians, we developed a fully automated computational pipeline, EzMap, for performing metagenomic analysis of the human virome. EzMap combines a number of tools to clean, filter, and subtract WGS reads by mapping to a reference human assembly. The relative abundance of each virus present is estimated using a maximum likelihood approach that accounts for genome size, and results are presented with interactive visualizations and taxonomy-based summaries that enable rapid insights. The pipeline is automated to run on both workstations and computing clusters for all steps. EzMap automates an otherwise tedious and time-consuming protocol and aims to facilitate rapid and reproducible insights from cfDNA.


Availability and Implementation
EzMap is freely available at https://github.com/dekoning-lab/ezmap.


Contact
jason.dekoning@ucalgary.ca.


Supplementary information
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr id="Mucosal-associated invariant T (MAIT) cells are a group of unconventional T cells that mainly recognize bacterial vitamin B metabolites presented on MHC-related protein 1 (MR1). MAIT cells have been shown to play an important role in controlling bacterial infection and in responding to viral infections. Furthermore, MAIT cells have been implicated in different chronic inflammatory diseases such as inflammatory bowel disease and multiple sclerosis. Despite their involvement in different physiological and pathological processes, a deeper understanding of MAIT cells is still lacking. Arguably, this can be attributed to the difficulty of quantifying and measuring MAIT cells in different biological samples which is commonly done using flow cytometry-based methods and single-cell-based RNA sequencing techniques. These methods mostly require fresh samples which are difficult to obtain, especially from tissues, have low to medium throughput, and are costly and labor-intensive. To address these limitations, we developed sequence-to-MAIT (Seq2MAIT) which is a transformer-based deep neural network capable of identifying MAIT cells in bulk TCR-sequencing datasets, enabling the quantification of MAIT cells from any biological materials where human DNA is available. Benchmarking Seq2MAIT across different test datasets showed an average area-under-the-receiver-operator-curve (AU[ROC]) >0.80. In conclusion, Seq2MAIT is a novel, economical, and scalable method for identifying and quantifying MAIT cells in virtually any biological sample.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7407ca7605f24e11c1fc4335ba413bb4681f30b9" target='_blank'>Seq2MAIT: A Novel Deep Learning Framework for Identifying Mucosal Associated Invariant T (MAIT) Cells</a></td>
          <td>
            H. ElAbd, Rachel Byron, Steven Woodhouse, Brittney Robinett, Joe Sulc, A. Franke, Mitchell Pesesky, Wenyu Zhou, Haiyin Chen-Harris, Bryan Howie, Ruth Taniguchi, Harlan S. Robins
          </td>
          <td>2024-03-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>
    
        <tr id="ABSTRACT The human gut virome is predominantly made up of bacteriophages (phages), viruses that infect bacteria. Metagenomic studies have revealed that phages in the gut are highly individual specific and dynamic. These features make it challenging to perform meaningful cross-study comparisons. While several taxonomy frameworks exist to group phages and improve these comparisons, these strategies provide little insight into the potential effects phages have on their bacterial hosts. Here, we propose the use of predicted phage host families (PHFs) as a functionally relevant, qualitative unit of phage classification to improve these cross-study analyses. We first show that bioinformatic predictions of phage hosts are accurate at the host family level by measuring their concordance to Hi-C sequencing-based predictions in human and mouse fecal samples. Next, using phage host family predictions, we determined that PHFs reduce intra- and interindividual ecological distances compared to viral contigs in a previously published cohort of 10 healthy individuals, while simultaneously improving longitudinal virome stability. Lastly, by reanalyzing a previously published metagenomics data set with >1,000 samples, we determined that PHFs are prevalent across individuals and can aid in the detection of inflammatory bowel disease-specific virome signatures. Overall, our analyses support the use of predicted phage hosts in reducing between-sample distances and providing a biologically relevant framework for making between-sample virome comparisons. IMPORTANCE The human gut virome consists mainly of bacteriophages (phages), which infect bacteria and show high individual specificity and variability, complicating cross-study comparisons. Furthermore, existing taxonomic frameworks offer limited insight into their interactions with bacterial hosts. In this study, we propose using predicted phage host families (PHFs) as a higher-level classification unit to enhance functional cross-study comparisons. We demonstrate that bioinformatic predictions of phage hosts align with Hi-C sequencing results at the host family level in human and mouse fecal samples. We further show that PHFs reduce ecological distances and improve virome stability over time. Additionally, reanalysis of a large metagenomics data set revealed that PHFs are widespread and can help identify disease-specific virome patterns, such as those linked to inflammatory bowel disease. The human gut virome consists mainly of bacteriophages (phages), which infect bacteria and show high individual specificity and variability, complicating cross-study comparisons. Furthermore, existing taxonomic frameworks offer limited insight into their interactions with bacterial hosts. In this study, we propose using predicted phage host families (PHFs) as a higher-level classification unit to enhance functional cross-study comparisons. We demonstrate that bioinformatic predictions of phage hosts align with Hi-C sequencing results at the host family level in human and mouse fecal samples. We further show that PHFs reduce ecological distances and improve virome stability over time. Additionally, reanalysis of a large metagenomics data set revealed that PHFs are widespread and can help identify disease-specific virome patterns, such as those linked to inflammatory bowel disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3fe59fc08f0a002c4b5a3c3980de8529f06eba5" target='_blank'>Improving gut virome comparisons using predicted phage host information</a></td>
          <td>
            Michael Shamash, Anshul Sinha, C. Maurice
          </td>
          <td>2024-10-11</td>
          <td>mSystems</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr id="Defining physiology and methods to measure biological mechanisms is essential. Extensive datasets such as RNA sequencing are used with little analysis of the knowledge gained from the various methodologies. Within this work, we have processed publicly available NCBI RNAseq datasets using a combination of bioinformatics tools for the largest physiological organ, the skin. In many datasets, we identify the quality of the sample, human transcript mapping, the sex of each sample, foreign RNA from bacteria/viruses/protists, and the presence of B/T-cell immune repertoire. Processing 8,274 samples from 132 different experiments for skin samples identifies common flora of skin with elevation of protists (such as Leishmania), bacteria (Staphylococcus, Cutibacterium acnes), and viruses (HSV, HPV) that may be involved in physiological differences. We observed samples with the Heilongjiang tick virus, Human T-cell Leukemia Virus Type I, and Equine Infectious Anemia Virus that likely play pathological roles in physiology. Integrating the various biomarkers identified five ideal datasets for skin pathologies that elucidated a novel correlation between the normal skin flora bacterium Bacillus megaterium with MHC complex regulation and the immune repertoire clonal expansion, particularly in patients with hidradenitis suppurativa. Finally, we show that in multiple independent experiments, biological sex is associated with multiple sex chromosome gene differences, highlighting the importance of future work in studying sex differences in skin. Data integrations and multidimensional data mapping are critical for physiological omics advancements, and this work highlights the exciting ability to apply these tools to skin physiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21d7405dc4848bad01933fa8e058f89b35b57024" target='_blank'>From Castaways to Discoveries: Unveiling Treasures in Skin RNAseq Using a Novel Multidimensional Data Processing Workflow Including Infection-Host Dynamics.</a></td>
          <td>
            Disha Bhargava, Alec Labadie, Rebecca L Hanson-Rios-Stutz, Austin Goodyke, Ella M Moses, Akansha S Das, Sophie Vanderweele, Janelle V Lemon, Taylor W Cook, David Pearson, Joseph M Redinger, Adam J Caulfield, Rosemary Olivero, Kate Foster, Kurt Ashack, S. Rajasekaran, Caleb P. Bupp, Timothy J. Triche, Connie M Krawczyk, Dave Chesla, Matthew D Sims, Nicholas Hartog, Jeremy W Prokop
          </td>
          <td>2025-03-12</td>
          <td>Physiological genomics</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr id="Understanding the link between the human gut virome and diseases has garnered significant interest in the research community. Extracting virus-related information from metagenomic sequencing data is crucial for unravelling virus composition, host interactions, and disease associations. However, current metagenomic analysis workflows for viral genomes vary in effectiveness, posing challenges for researchers seeking the most up-to-date tools. To address this, we present ViromeFlowX, a user-friendly Nextflow workflow that automates viral genome assembly, identification, classification, and annotation. This streamlined workflow integrates cutting-edge tools for processing raw sequencing data for taxonomic annotation and functional analysis. Application to a dataset of 200 metagenomic samples yielded high-quality viral genomes. ViromeFlowX enables efficient mining of viral genomic data, offering a valuable resource to investigate the gut virome’s role in virus-host interactions and virus-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ea369c597f3fa29579967e6b4bff2cb63bdf5c" target='_blank'>ViromeFlowX: a Comprehensive Nextflow-based Automated Workflow for Mining Viral Genomes from Metagenomic Sequencing Data</a></td>
          <td>
            Xiaokai Wang, Zhimin Ding, Ying Yang, Lifeng Liang, Yingshuai Sun, Chaojian Hou, Yuning Zheng, Yan Xia, Lixin Dong
          </td>
          <td>2024-02-01</td>
          <td>Microbial Genomics</td>
          <td>1</td>
          <td>9</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>