---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-25 09:15:05 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8abba679e2263c0dd6b357ce0b9b74a569cd8b2" target='_blank'>Ten computational challenges in human virome studies</a></td>
          <td>
            Yifan Wu, Yousong Peng
          </td>
          <td>2024-02-23</td>
          <td>Virologica Sinica</td>
          <td>3</td>
          <td>3</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr id="ABSTRACT The human gut virome is predominantly made up of bacteriophages (phages), viruses that infect bacteria. Metagenomic studies have revealed that phages in the gut are highly individual specific and dynamic. These features make it challenging to perform meaningful cross-study comparisons. While several taxonomy frameworks exist to group phages and improve these comparisons, these strategies provide little insight into the potential effects phages have on their bacterial hosts. Here, we propose the use of predicted phage host families (PHFs) as a functionally relevant, qualitative unit of phage classification to improve these cross-study analyses. We first show that bioinformatic predictions of phage hosts are accurate at the host family level by measuring their concordance to Hi-C sequencing-based predictions in human and mouse fecal samples. Next, using phage host family predictions, we determined that PHFs reduce intra- and interindividual ecological distances compared to viral contigs in a previously published cohort of 10 healthy individuals, while simultaneously improving longitudinal virome stability. Lastly, by reanalyzing a previously published metagenomics data set with >1,000 samples, we determined that PHFs are prevalent across individuals and can aid in the detection of inflammatory bowel disease-specific virome signatures. Overall, our analyses support the use of predicted phage hosts in reducing between-sample distances and providing a biologically relevant framework for making between-sample virome comparisons. IMPORTANCE The human gut virome consists mainly of bacteriophages (phages), which infect bacteria and show high individual specificity and variability, complicating cross-study comparisons. Furthermore, existing taxonomic frameworks offer limited insight into their interactions with bacterial hosts. In this study, we propose using predicted phage host families (PHFs) as a higher-level classification unit to enhance functional cross-study comparisons. We demonstrate that bioinformatic predictions of phage hosts align with Hi-C sequencing results at the host family level in human and mouse fecal samples. We further show that PHFs reduce ecological distances and improve virome stability over time. Additionally, reanalysis of a large metagenomics data set revealed that PHFs are widespread and can help identify disease-specific virome patterns, such as those linked to inflammatory bowel disease. The human gut virome consists mainly of bacteriophages (phages), which infect bacteria and show high individual specificity and variability, complicating cross-study comparisons. Furthermore, existing taxonomic frameworks offer limited insight into their interactions with bacterial hosts. In this study, we propose using predicted phage host families (PHFs) as a higher-level classification unit to enhance functional cross-study comparisons. We demonstrate that bioinformatic predictions of phage hosts align with Hi-C sequencing results at the host family level in human and mouse fecal samples. We further show that PHFs reduce ecological distances and improve virome stability over time. Additionally, reanalysis of a large metagenomics data set revealed that PHFs are widespread and can help identify disease-specific virome patterns, such as those linked to inflammatory bowel disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3fe59fc08f0a002c4b5a3c3980de8529f06eba5" target='_blank'>Improving gut virome comparisons using predicted phage host information</a></td>
          <td>
            Michael Shamash, Anshul Sinha, C. Maurice
          </td>
          <td>2024-10-11</td>
          <td>mSystems</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr id="Understanding the link between the human gut virome and diseases has garnered significant interest in the research community. Extracting virus-related information from metagenomic sequencing data is crucial for unravelling virus composition, host interactions, and disease associations. However, current metagenomic analysis workflows for viral genomes vary in effectiveness, posing challenges for researchers seeking the most up-to-date tools. To address this, we present ViromeFlowX, a user-friendly Nextflow workflow that automates viral genome assembly, identification, classification, and annotation. This streamlined workflow integrates cutting-edge tools for processing raw sequencing data for taxonomic annotation and functional analysis. Application to a dataset of 200 metagenomic samples yielded high-quality viral genomes. ViromeFlowX enables efficient mining of viral genomic data, offering a valuable resource to investigate the gut virome’s role in virus-host interactions and virus-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ea369c597f3fa29579967e6b4bff2cb63bdf5c" target='_blank'>ViromeFlowX: a Comprehensive Nextflow-based Automated Workflow for Mining Viral Genomes from Metagenomic Sequencing Data</a></td>
          <td>
            Xiaokai Wang, Zhimin Ding, Ying Yang, Lifeng Liang, Yingshuai Sun, Chaojian Hou, Yuning Zheng, Yan Xia, Lixin Dong
          </td>
          <td>2024-02-01</td>
          <td>Microbial Genomics</td>
          <td>1</td>
          <td>9</td>
        </tr>
    
        <tr id="The evolutionary arms race between host restriction factors and viral antagonists provides crucial insights into immune system evolution and viral adaptation. This study investigates the structural and evolutionary dynamics of the double-domain restriction factors A3F and A3G and their viral inhibitor, Vif, across diverse primate species. By constructing 3D structural homology models and integrating ancestral sequence reconstruction (ASR), we identified patterns of sequence diversity, structural conservation, and functional adaptation. Inactive CD1 (Catalytic Domain 1) domains displayed greater sequence diversity and more positive surface charges than active CD2 domains, aiding nucleotide chain binding and intersegmental transfer. Despite variability, the CD2 DNA-binding grooves remained structurally consistent with conserved residues maintaining critical functions. A3F and A3G diverged in loop 7’ interaction strategies, utilising distinct molecular interactions to facilitate their roles. Vif exhibited charge variation linked to host species, reflecting its coevolution with A3 proteins. These findings illuminate how structural adaptations and charge dynamics enable both restriction factors and their viral antagonists to adapt to selective pressures. Our results emphasize the importance of studying structural evolution in host–virus interactions, with implications for understanding immune defense mechanisms, zoonotic risks, and viral evolution. This work establishes a foundation for further exploration of restriction factor diversity and coevolution across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d3b8f863a1b52ab95b465a0ea4758895d223fee" target='_blank'>Coevolution of Lentiviral Vif with Host A3F and A3G: Insights from Computational Modelling and Ancestral Sequence Reconstruction</a></td>
          <td>
            David Nicolas Giuseppe Huebert, Atefeh Ghorbani, Shaw Yick Brian Lam, M. Larijani
          </td>
          <td>2025-03-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr id="Monkeypox has been a neglected, zoonotic tropical disease for over 50 years. Since the 2022 global outbreak, hundreds of human clinical samples have been subjected to next-generation sequencing (NGS) worldwide with raw data deposited in public repositories. However, sequence analysis for in-depth investigation of viral evolution remains hindered by the lack of a curated, whole genome Monkeypox virus (MPXV) database (DB) and efficient bioinformatics pipelines. To address this, we developed a customized MPXV DB for integration with “ready-to-use” workflows in the CLC Microbial Genomics Module for whole genomic and metagenomic analysis. After database construction (218 MPXV genomes), whole genome alignment, pairwise comparison, and evolutionary analysis of all genomes were analyzed to autogenerate tabular outputs and visual displays (collective runtime: 16 min). The clinical utility of the MPXV DB was demonstrated by using a Chimpanzee fecal, hybrid-capture NGS dataset (publicly available) for metagenomic, phylogenomic, and viral/host integration analysis. The clinically relevant MPXV DB embedded in CLC workflows proved to be a rapid method of sequence analysis useful for phylogenomic exploration and a wide range of applications in translational science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282a7b3fdf017ea4a95eeead5216dccb42ea0c9b" target='_blank'>A Customized Monkeypox Virus Genomic Database (MPXV DB v1.0) for Rapid Sequence Analysis and Phylogenomic Discoveries in CLC Microbial Genomics</a></td>
          <td>
            J. Shen-Gunther, Hong Cai, Yufeng Wang
          </td>
          <td>2022-12-22</td>
          <td>Viruses</td>
          <td>7</td>
          <td>22</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05ae3c9f0f4159c245dcca9a5fc2262129307daf" target='_blank'>Computational analysis of HIV-1 evolution and epidemiology</a></td>
          <td>
            Sergei L Kosakovsky Pond
          </td>
          <td>2011-11-12</td>
          <td>2011 IEEE International Conference on Bioinformatics and Biomedicine Workshops (BIBMW)</td>
          <td>2</td>
          <td>39</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>