---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-15 09:15:14 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="An increasing number of studies have demonstrated the exacerbation of disease progression of both COVID-19 and HIV when coexisting. However, the molecular mechanisms underlying the interplay between these two viruses are still poorly understood. In this study, we utilized a comprehensive analysis of single-cell transcriptomics to identify and characterize peripheral blood cell subsets in COVID-19 and HIV-infected patients. Our findings revealed that COVID-19 and HIV exhibit a partially similar cellular composition and cell cycle distribution. Additionally, we identified a common pathogenesis in B-cell subsets of COVID-19 and HIV patients, which showed abnormal activation states of the unfolded protein response (UPR) pathway. Based on B-cell signature genes and UPR-related genes, we developed a machine learning diagnostic model that can accurately diagnose both COVID-19 and HIV infections. Our model was validated using a large number of bulk transcriptome data sets and showed good clinical efficacy. Our study provides molecular insights into the single-cell level interplay between SARS-CoV-2 and HIV infections, suggesting a possible common disease mechanism that warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bacba4a2642c5c3b59b43552fa3355f4df068315" target='_blank'>Single-Cell Omics Analysis Reveals Immunological Dysregulation in COVID-19 and HIV: Identifying a Shared Abnormality of B Cell Activation via the Unfolded Protein Response and Diagnostic Biomarkers Using Machine Learning Algorithms.</a></td>
          <td>
            Feng Li, Wei Zhao, Hong Liu, Yandie Niu, Jiahao Ma
          </td>
          <td>2025-08-01</td>
          <td>Journal of medical virology</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr id="Human immunity relies on the coordinated responses of many cellular subsets and functional states. Inter-individual variations in cellular composition and communication could thus potentially alter host protection. Here, we explore this hypothesis by applying single-cell RNA-sequencing to examine viral responses among the dendritic cells (DCs) of three elite controllers (ECs) of HIV-1 infection. To overcome the potentially confounding effects of donor-to-donor variability, we present a generally applicable computational framework for identifying reproducible patterns in gene expression across donors who share a unifying classification. Applying it, we discover a highly functional antiviral DC state in ECs whose fractional abundance after in vitro exposure to HIV-1 correlates with higher CD4+ T cell counts and lower HIV-1 viral loads, and that effectively primes polyfunctional T cell responses in vitro. By integrating information from existing genomic databases into our reproducibility-based analysis, we identify and validate select immunomodulators that increase the fractional abundance of this state in primary peripheral blood mononuclear cells from healthy individuals in vitro. Overall, our results demonstrate how single-cell approaches can reveal previously unappreciated, yet important, immune behaviors and empower rational frameworks for modulating systems-level immune responses that may prove therapeutically and prophylactically useful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdaaa8164fd46945764987c98bec307fda098c2" target='_blank'>A Reproducibility-Based Computational Framework Identifies an Inducible, Enhanced Antiviral State in Dendritic Cells from HIV-1 Elite Controllers</a></td>
          <td>
            E. Martín-Gayo, Michael Cole, Kellie E. Kolb, Zhengyu Ouyang, Jacqueline Cronin, S. Kazer, J. Ordovas-Montañes, M. Lichterfeld, B. Walker, N. Yosef, A. Shalek, Xu G. Yu
          </td>
          <td>2018-01-29</td>
          <td>Genome Biology</td>
          <td>44</td>
          <td>159</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>434</td>
          <td>103</td>
        </tr>
    
        <tr id="Predicting individual vaccine responses remains a significant challenge due to the complexity and variability of immune processes. To address this gap, we developed immunaut, an open-source, data-driven framework implemented as an R package specifically designed for all systems vaccinologists seeking to analyze and predict immunological outcomes across diverse vaccination settings. Leveraging one of the most comprehensive live attenuated influenza vaccine (LAIV) datasets to date - 244 Gambian children enrolled in a phase 4 immunogenicity study - immunaut integrates humoral, mucosal, cellular, transcriptomic, and microbiological parameters collected before and after vaccination, providing an unprecedentedly holistic view of LAIV-induced immunity. Through advanced dimensionality reduction, clustering, and predictive modeling, immunaut identifies distinct immunophenotypic responder profiles and their underlying baseline determinants. In this study, immunaut delineated three immunophenotypes: (1) CD8 T-cell responders, marked by strong baseline mucosal immunity and extensive prior influenza virus exposure that boosts memory CD8 T-cell responses, without generating influenza virus-specific antibody responses; (2) Mucosal responders, characterized by pre-existing systemic influenza A virus immunity (specifically to H3N2) and stable epithelial integrity, leading to potent mucosal IgA expansions and subsequent seroconversion to influenza B virus; and (3) Systemic, broad influenza A virus responders, who start with relatively naive immunity and leverage greater initial viral replication to drive broad systemic antibody responses against multiple influenza A virus variants beyond those included in the LAIV vaccine. By integrating pathway-level analysis, model-derived contribution scores, and hierarchical decision rules, immunaut elucidates how distinct immunological landscapes shape each response trajectory and how key baseline features, including pre-existing immunity, mucosal preparedness, and cellular support, dictate vaccine outcomes. Collectively, these findings emphasize the power of integrative, predictive frameworks to advance precision vaccinology, and highlight immunaut as a versatile, community-available resource for optimizing immunization strategies across diverse populations and vaccine platforms. One-Sentence Summary Using one of the most comprehensive LAIV datasets compiled to date, immunaut, an integrative machine learning framework, identifies distinct immunophenotypic responder groups shaped by baseline immune landscapes, advancing precision vaccinology and guiding more effective, personalized immunization strategies. Graphical abstract Immunaut, an automated framework for mapping and predicting vaccine response immunotypes. Step 1 outlines the identification of vaccine response outcomes using pre- and post-vaccination data integration across immune features, including antibodies, flu-specific T-cells, and immunophenotyping at mucosal and systemic sites. Clustering methods define the vaccine response landscape, stability, and validation through t-SNE-based visualization. Step 2 leverages an automated machine learning modeling approach, to enhance the accuracy and interpretability of vaccine response predictions, enabling stratification and targeted intervention strategies for personalized vaccine immunogenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dfad70bdcf6c0501623eb1305faf8c365faa1a5" target='_blank'>Integrative Mapping of Pre-existing Immune Landscapes for Vaccine Response Prediction</a></td>
          <td>
            Stephanie Hao, Ivan Tomic, B. Lindsey, Y. Jagne, Katja Hoschler, Adam Meijer, Juan Manuel Carreño Quiroz, Philip Meade, Kaori Sano, Chikondi Peno, Andre G. Costa-Martins, Debby Bogaert, B. Kampmann, H. Nakaya, F. Krammer, T. D. de Silva, Adriana Tomic
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>108</td>
        </tr>
    
        <tr id="With the increasing application of artificial intelligence (AI) in medical research, studies on the human immunodeficiency virus type 1(HIV-1) and acquired immunodeficiency syndrome (AIDS) have become more in-depth. Integrating AI with technologies like single-cell sequencing enables precise biomarker identification and improved therapeutic targeting. This review aims to explore the advancements in AI technologies and their applications across various facets of HIV research, including viral mechanisms, diagnostic innovations, therapeutic strategies, and prevention efforts. Despite challenges like data limitations and model interpretability, AI holds significant potential in advancing HIV-1 management and contributing to global health goals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3723412fe43b510911ce44ce7719b8fe3d629" target='_blank'>AI applications in HIV research: advances and future directions</a></td>
          <td>
            Ruyi Jin, Li Zhang
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Microbiology</td>
          <td>4</td>
          <td>1</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8abba679e2263c0dd6b357ce0b9b74a569cd8b2" target='_blank'>Ten computational challenges in human virome studies</a></td>
          <td>
            Yifan Wu, Yousong Peng
          </td>
          <td>2024-02-23</td>
          <td>Virologica Sinica</td>
          <td>3</td>
          <td>3</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr id="Numerous HIV related outcomes can be determined on the viral genome, for example, resistance associated mutations, population transmission dynamics, viral heritability traits, or time since infection. Viral sequences of people with HIV (PWH) are therefore essential for therapeutic and research purposes. While in the first three decades of the HIV pandemic viral genomes were mainly sequenced using Sanger sequencing, the last decade has seen a shift towards next-generation sequencing (NGS) as the preferred method. NGS can achieve near full length genome sequence coverage and simultaneously, it accurately encapsulates the within-host diversity by characterizing HIV subpopulations. NGS opens new avenues for HIV research, but it also presents challenges concerning data management and analysis. We therefore set up the Swiss HIV Cohort Study Viral NGS Database (SHCND) to address key issues in the handling of NGS data including high loads of raw- and processed NGS data, data storage solutions, downstream application of sophisticated bioinformatic tools, high-performance computing resources, and reproducibility. The database is nested within the Swiss HIV Cohort Study (SHCS) and the Zurich Primary HIV Infection Cohort Study (ZPHI), which together enrolled 21,876 PWH since 1988 and include a biobank dating back to the early nineties. Since its initiation in 2018, the SHCND accumulated NGS sequences (plasma and proviral origin) of 5,178 unique PWH. We here describe the design, set-up, and use of this NGS database. Overall, the SHCND has contributed to several research projects on HIV pathogenesis, treatment, drug resistance, and molecular epidemiology, and has thereby become a central part of HIV-genomics research in Switzerland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1ae2695568c7b9933f404942071b7dfee9c31bf" target='_blank'>Addressing data management and analysis challenges in viral genomics: The Swiss HIV cohort study viral next generation sequencing database</a></td>
          <td>
            Marius Zeeb, Paul Frischknecht, S. Balakrishna, Lisa Jörimann, Jasmin Tschumi, Levente Zsichla, Sandra E. Chaudron, Bashkim Jaha, Kathrin Neumann, Christine Leemann, M. Huber, K. Leuzinger, H. Günthard, K. Metzner, R. Kouyos
          </td>
          <td>2025-04-01</td>
          <td>PLOS Digital Health</td>
          <td>2</td>
          <td>95</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>