---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-05 09:14:07 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>425</td>
          <td>102</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2823684aa13e4e2575d6d1da7cd6194c358c3ce5" target='_blank'>Systems Biology Approach to Uncover Signaling Defects in Primary Immunodeficiency Diseases</a></td>
          <td>
            Jeff Choi, R. Fernandez, H. Maecker, M. Butte
          </td>
          <td>2016-05-01</td>
          <td>The Journal of Immunology</td>
          <td>2</td>
          <td>77</td>
        </tr>
    
        <tr id="We have developed a full genome virus detection process that combines sensitive nucleic acid preparation optimised for virus identification in fecal material with Illumina MiSeq sequencing and a novel post-sequencing virus identification algorithm. Enriched viral nucleic acid was converted to double-stranded DNA and subjected to Illumina MiSeq sequencing. The resulting short reads were processed with a novel iterative Python algorithm SLIM for the identification of sequences with homology to known viruses. De novo assembly was then used to generate full viral genomes. The sensitivity of this process was demonstrated with a set of fecal samples from HIV-1 infected patients. A quantitative assessment of the mammalian, plant, and bacterial virus content of this compartment was generated and the deep sequencing data were sufficient to assembly 12 complete viral genomes from 6 virus families. The method detected high levels of enteropathic viruses that are normally controlled in healthy adults, but may be involved in the pathogenesis of HIV-1 infection and will provide a powerful tool for virus detection and for analyzing changes in the fecal virome associated with HIV-1 progression and pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab98d1b125aa0704e63adef426b27abd32e935f0" target='_blank'>Full Genome Virus Detection in Fecal Samples Using Sensitive Nucleic Acid Preparation, Deep Sequencing, and a Novel Iterative Sequence Classification Algorithm</a></td>
          <td>
            M. Cotten, B. O. Oude Munnink, M. Canuti, M. Deijs, S. Watson, P. Kellam, L. van der Hoek
          </td>
          <td>2014-04-02</td>
          <td>PLoS ONE</td>
          <td>71</td>
          <td>77</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21d7405dc4848bad01933fa8e058f89b35b57024" target='_blank'>From Castaways to Discoveries: Unveiling Treasures in Skin RNAseq Using a Novel Multidimensional Data Processing Workflow Including Infection-Host Dynamics.</a></td>
          <td>
            Disha Bhargava, Alec Labadie, Rebecca L Hanson-Rios-Stutz, Austin Goodyke, Ella M Moses, Akansha S Das, Sophie Vanderweele, Janelle V Lemon, Taylor W Cook, David Pearson, Joseph M Redinger, Adam J Caulfield, Rosemary Olivero, Kate Foster, Kurt Ashack, S. Rajasekaran, Caleb P. Bupp, Timothy J. Triche, Connie M Krawczyk, Dave Chesla, Matthew D Sims, Nicholas Hartog, Jeremy W Prokop
          </td>
          <td>2025-03-12</td>
          <td>Physiological genomics</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr id="Understanding the link between the human gut virome and diseases has garnered significant interest in the research community. Extracting virus-related information from metagenomic sequencing data is crucial for unravelling virus composition, host interactions, and disease associations. However, current metagenomic analysis workflows for viral genomes vary in effectiveness, posing challenges for researchers seeking the most up-to-date tools. To address this, we present ViromeFlowX, a user-friendly Nextflow workflow that automates viral genome assembly, identification, classification, and annotation. This streamlined workflow integrates cutting-edge tools for processing raw sequencing data for taxonomic annotation and functional analysis. Application to a dataset of 200 metagenomic samples yielded high-quality viral genomes. ViromeFlowX enables efficient mining of viral genomic data, offering a valuable resource to investigate the gut virome’s role in virus-host interactions and virus-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ea369c597f3fa29579967e6b4bff2cb63bdf5c" target='_blank'>ViromeFlowX: a Comprehensive Nextflow-based Automated Workflow for Mining Viral Genomes from Metagenomic Sequencing Data</a></td>
          <td>
            Xiaokai Wang, Zhimin Ding, Ying Yang, Lifeng Liang, Yingshuai Sun, Chaojian Hou, Yuning Zheng, Yan Xia, Lixin Dong
          </td>
          <td>2024-02-01</td>
          <td>Microbial Genomics</td>
          <td>1</td>
          <td>9</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a952e012e1800bbd1317b1a754da406bbe7cda" target='_blank'>Systems analysis of human innate immunity in COVID-19</a></td>
          <td>
            Sophie Müller, J. Schultze
          </td>
          <td>2023-05-01</td>
          <td>Seminars in Immunology</td>
          <td>4</td>
          <td>94</td>
        </tr>
    
        <tr id="Crohn's disease (CD) is a chronic relapsing–remitting inflammatory disorder of the gastrointestinal tract that is characterized by altered innate and adaptive immune function. Although massively parallel sequencing studies of the T cell receptor repertoire identified oligoclonal expansion of unique clones, much less is known about the B cell receptor (BCR) repertoire in CD. Here, we present a novel BCR repertoire sequencing data set from ileal biopsies from pediatric patients with CD and controls, and identify CD-specific somatic hypermutation (SHM) patterns, revealed by a machine learning (ML) algorithm trained on BCR repertoire sequences. Moreover, ML classification of a different data set from blood samples of adults with CD versus controls identified that V gene usage, clusters, or mutation frequencies yielded excellent results in classifying the disease (F1 > 90%). In summary, we show that an ML algorithm enables the classification of CD based on unique BCR repertoire features with high accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b02ad45f1c8bd8f0a708c7bb9c92eef47822c8e" target='_blank'>A somatic hypermutation–based machine learning model stratifies individuals with Crohn's disease and controls</a></td>
          <td>
            Modi Safra, L. Werner, P. Polak, Ayelet Peres, Naomi Salamon, M. Schvimer, B. Weiss, I. Barshack, D. Shouval, Gur Yaari
          </td>
          <td>2022-12-16</td>
          <td>Genome Research</td>
          <td>9</td>
          <td>57</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32015c79bd160189ddb13fa99f034c432b94c59" target='_blank'>Probing infectious disease by single-cell RNA sequencing: Progresses and perspectives</a></td>
          <td>
            Geyang Luo, Q. Gao, Shuye Zhang, Bo Yan
          </td>
          <td>2020-10-21</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>41</td>
          <td>43</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>