---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-26 09:27:23 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Human immunity relies on the coordinated responses of many cellular subsets and functional states. Inter-individual variations in cellular composition and communication could thus potentially alter host protection. Here, we explore this hypothesis by applying single-cell RNA-sequencing to examine viral responses among the dendritic cells (DCs) of three elite controllers (ECs) of HIV-1 infection. To overcome the potentially confounding effects of donor-to-donor variability, we present a generally applicable computational framework for identifying reproducible patterns in gene expression across donors who share a unifying classification. Applying it, we discover a highly functional antiviral DC state in ECs whose fractional abundance after in vitro exposure to HIV-1 correlates with higher CD4+ T cell counts and lower HIV-1 viral loads, and that effectively primes polyfunctional T cell responses in vitro. By integrating information from existing genomic databases into our reproducibility-based analysis, we identify and validate select immunomodulators that increase the fractional abundance of this state in primary peripheral blood mononuclear cells from healthy individuals in vitro. Overall, our results demonstrate how single-cell approaches can reveal previously unappreciated, yet important, immune behaviors and empower rational frameworks for modulating systems-level immune responses that may prove therapeutically and prophylactically useful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdaaa8164fd46945764987c98bec307fda098c2" target='_blank'>A Reproducibility-Based Computational Framework Identifies an Inducible, Enhanced Antiviral State in Dendritic Cells from HIV-1 Elite Controllers</a></td>
          <td>
            E. Martín-Gayo, Michael Cole, Kellie E. Kolb, Zhengyu Ouyang, Jacqueline Cronin, S. Kazer, J. Ordovas-Montañes, M. Lichterfeld, B. Walker, N. Yosef, A. Shalek, Xu G. Yu
          </td>
          <td>2018-01-29</td>
          <td>Genome Biology</td>
          <td>46</td>
          <td>158</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>443</td>
          <td>105</td>
        </tr>
    
        <tr id="We and others have reported evidence of T cell exhaustion in children with perinatal HIV with increased expression of inhibitory receptors PD-1, CD160, and TIM-3, but there is limited data on the virologic functional consequences of this immune exhaustion. We address this by using an immune database from Kenyan children with perinatal HIV and unexposed controls. We computationally integrate T cell profiles of differentiation, activation and exhaustion in an agent-based model (ABM) to predict how T cell exhaustion impacts viral control following HIV exposure in vitro. Our ABM includes macrophages, CD4 and CD8 T cells, cytokines, and HIV. Model mechanisms include viral dynamics, macrophage activation, T cell activation and proliferation, cytotoxic T cell killing, and cytokine/HIV diffusion and degradation. Participants are grouped by HIV plasma viremia and by age, less than 5 years or 5-18 years. Our findings indicate that cells from virally active participants, who have the highest levels of exhaustion, have lower predicted viral concentrations and infected cells compared to other participant groups during new infection. However, this coincides with higher cell death, suggesting that short-term viral control is associated with excessive inflammation, which could be detrimental long-term. Cells from virally suppressed participants older than 5 years can maintain lower viral concentrations while limiting cell death, reflecting a more sustainable short-term immune response. In virally suppressed children younger than 5 years, immune response patterns strongly resemble the age-matched healthy control group, suggesting early viral suppression may preserve antiviral immune responses. Our model predicts unique patterns of cell death for each participant group, with CD8 T cell death being dominant in virally active groups and CD4 T cell and macrophage death being dominant in healthy and virally suppressed groups. Finally, exhausted CD8 T cells are predicted to contribute significantly to CD8 T cell killing, proliferation, and activation in the virally active group, indicating partially functional CD8 T cells can still contribute to short-term viral control. Our analysis functionally integrates participant-specific immunophenotypic data to allow quantification of the extent, mechanisms, and impact of immune dysfunction in perinatal HIV and could inform pediatric HIV remission and cure strategies. Author Summary Cytotoxic CD8+ T cells are vital for our ability to kill infected cells. However, chronic stimulation of CD8+ T cells can result in T cell exhaustion, leading to cellular dysfunction and reduced ability to fight infections. Here we ask: What impact does CD8+ T cell exhaustion have on the ability to control HIV replication over the short term? We answer this using a computational model, anchored in experimental data from children with perinatal HIV. We use participant-specific data on dozens of immune markers that characterize the state of their immune system. We carefully map these markers to known immune cell functions and use our model to predict how immune cells with these participant-specific markers would respond to HIV infection. Our results suggest that earlier viral suppression can lead to better immune function in children with perinatal HIV. Our findings also suggest that exhausted CD8+ T cells could still contribute to fighting HIV infection, but only if there are enough of them to compensate for their reduced functionality. Our work uses computational models to put valuable participant-specific data into context, allowing us to predict short-term infection outcomes and better understand immune function in children with perinatal HIV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feef85b43caeefacd242e0fed705966cf8ef6533" target='_blank'>Partially-functional exhausted CD8+ T cells can contribute to short-term viral suppression: a computational prediction for children with perinatal HIV</a></td>
          <td>
            Alexis Hoerter, Alexa Petrucciani, Fatma Marshed, Mussa Mwamzuka, Aabid A. Ahmed, Alka Khaitan, E. Pienaar
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b72639d6e59b797a501444b04af3436df03cac" target='_blank'>Intracellular ISG-virus interactions determine viral infection severity and persistence.</a></td>
          <td>
            A. Bouchnita, V. Volpert
          </td>
          <td>2025-09-12</td>
          <td>Journal of theoretical biology</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8abba679e2263c0dd6b357ce0b9b74a569cd8b2" target='_blank'>Ten computational challenges in human virome studies</a></td>
          <td>
            Yifan Wu, Yousong Peng
          </td>
          <td>2024-02-23</td>
          <td>Virologica Sinica</td>
          <td>6</td>
          <td>4</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr id="Defining physiology and methods to measure biological mechanisms is essential. Extensive datasets such as RNA sequencing are used with little analysis of the knowledge gained from the various methodologies. Within this work, we have processed publicly available NCBI RNAseq datasets using a combination of bioinformatics tools for the largest physiological organ, the skin. In many datasets, we identify the quality of the sample, human transcript mapping, the sex of each sample, foreign RNA from bacteria/viruses/protists, and the presence of B/T-cell immune repertoire. Processing 8,274 samples from 132 different experiments for skin samples identifies common flora of skin with elevation of protists (such as Leishmania), bacteria (Staphylococcus, Cutibacterium acnes), and viruses (HSV, HPV) that may be involved in physiological differences. We observed samples with the Heilongjiang tick virus, Human T-cell Leukemia Virus Type I, and Equine Infectious Anemia Virus that likely play pathological roles in physiology. Integrating the various biomarkers identified five ideal datasets for skin pathologies that elucidated a novel correlation between the normal skin flora bacterium Bacillus megaterium with MHC complex regulation and the immune repertoire clonal expansion, particularly in patients with hidradenitis suppurativa. Finally, we show that in multiple independent experiments, biological sex is associated with multiple sex chromosome gene differences, highlighting the importance of future work in studying sex differences in skin. Data integrations and multidimensional data mapping are critical for physiological omics advancements, and this work highlights the exciting ability to apply these tools to skin physiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21d7405dc4848bad01933fa8e058f89b35b57024" target='_blank'>From Castaways to Discoveries: Unveiling Treasures in Skin RNAseq Using a Novel Multidimensional Data Processing Workflow Including Infection-Host Dynamics.</a></td>
          <td>
            Disha Bhargava, Alec Labadie, Rebecca L Hanson-Rios-Stutz, Austin Goodyke, Ella M Moses, Akansha S. Das, Sophie Vanderweele, Janelle V Lemon, Taylor W Cook, David Pearson, Joseph M Redinger, Adam J Caulfield, Rosemary Olivero, Kate Foster, Kurt Ashack, S. Rajasekaran, Caleb P. Bupp, Timothy J. Triche, Connie M Krawczyk, Dave Chesla, Matthew D Sims, Nicholas Hartog, Jeremy W. Prokop
          </td>
          <td>2025-03-12</td>
          <td>Physiological genomics</td>
          <td>1</td>
          <td>19</td>
        </tr>
    
        <tr id="Understanding the link between the human gut virome and diseases has garnered significant interest in the research community. Extracting virus-related information from metagenomic sequencing data is crucial for unravelling virus composition, host interactions, and disease associations. However, current metagenomic analysis workflows for viral genomes vary in effectiveness, posing challenges for researchers seeking the most up-to-date tools. To address this, we present ViromeFlowX, a user-friendly Nextflow workflow that automates viral genome assembly, identification, classification, and annotation. This streamlined workflow integrates cutting-edge tools for processing raw sequencing data for taxonomic annotation and functional analysis. Application to a dataset of 200 metagenomic samples yielded high-quality viral genomes. ViromeFlowX enables efficient mining of viral genomic data, offering a valuable resource to investigate the gut virome’s role in virus-host interactions and virus-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ea369c597f3fa29579967e6b4bff2cb63bdf5c" target='_blank'>ViromeFlowX: a Comprehensive Nextflow-based Automated Workflow for Mining Viral Genomes from Metagenomic Sequencing Data</a></td>
          <td>
            Xiaokai Wang, Zhimin Ding, Ying Yang, Lifeng Liang, Yingshuai Sun, Chaojian Hou, Yuning Zheng, Yan Xia, Lixin Dong
          </td>
          <td>2024-02-01</td>
          <td>Microbial Genomics</td>
          <td>3</td>
          <td>9</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>