---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-12 09:25:05 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Human immunity relies on the coordinated responses of many cellular subsets and functional states. Inter-individual variations in cellular composition and communication could thus potentially alter host protection. Here, we explore this hypothesis by applying single-cell RNA-sequencing to examine viral responses among the dendritic cells (DCs) of three elite controllers (ECs) of HIV-1 infection. To overcome the potentially confounding effects of donor-to-donor variability, we present a generally applicable computational framework for identifying reproducible patterns in gene expression across donors who share a unifying classification. Applying it, we discover a highly functional antiviral DC state in ECs whose fractional abundance after in vitro exposure to HIV-1 correlates with higher CD4+ T cell counts and lower HIV-1 viral loads, and that effectively primes polyfunctional T cell responses in vitro. By integrating information from existing genomic databases into our reproducibility-based analysis, we identify and validate select immunomodulators that increase the fractional abundance of this state in primary peripheral blood mononuclear cells from healthy individuals in vitro. Overall, our results demonstrate how single-cell approaches can reveal previously unappreciated, yet important, immune behaviors and empower rational frameworks for modulating systems-level immune responses that may prove therapeutically and prophylactically useful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdaaa8164fd46945764987c98bec307fda098c2" target='_blank'>A Reproducibility-Based Computational Framework Identifies an Inducible, Enhanced Antiviral State in Dendritic Cells from HIV-1 Elite Controllers</a></td>
          <td>
            E. Martín-Gayo, Michael Cole, Kellie E. Kolb, Zhengyu Ouyang, Jacqueline Cronin, S. Kazer, J. Ordovas-Montañes, M. Lichterfeld, B. Walker, N. Yosef, A. Shalek, Xu G. Yu
          </td>
          <td>2018-01-29</td>
          <td>Genome Biology</td>
          <td>46</td>
          <td>160</td>
        </tr>
    
        <tr id="We and others have reported evidence of T cell exhaustion in children with perinatal HIV with increased expression of inhibitory receptors PD-1, CD160, and TIM-3, but there is limited data on the virologic functional consequences of this immune exhaustion. We address this by using an immune database from Kenyan children with perinatal HIV and unexposed controls. We computationally integrate T cell profiles of differentiation, activation and exhaustion in an agent-based model (ABM) to predict how T cell exhaustion impacts viral control following HIV exposure in vitro. Our ABM includes macrophages, CD4 and CD8 T cells, cytokines, and HIV. Model mechanisms include viral dynamics, macrophage activation, T cell activation and proliferation, cytotoxic T cell killing, and cytokine/HIV diffusion and degradation. Participants are grouped by HIV plasma viremia and by age, less than 5 years or 5-18 years. Our findings indicate that cells from virally active participants, who have the highest levels of exhaustion, have lower predicted viral concentrations and infected cells compared to other participant groups during new infection. However, this coincides with higher cell death, suggesting that short-term viral control is associated with excessive inflammation, which could be detrimental long-term. Cells from virally suppressed participants older than 5 years can maintain lower viral concentrations while limiting cell death, reflecting a more sustainable short-term immune response. In virally suppressed children younger than 5 years, immune response patterns strongly resemble the age-matched healthy control group, suggesting early viral suppression may preserve antiviral immune responses. Our model predicts unique patterns of cell death for each participant group, with CD8 T cell death being dominant in virally active groups and CD4 T cell and macrophage death being dominant in healthy and virally suppressed groups. Finally, exhausted CD8 T cells are predicted to contribute significantly to CD8 T cell killing, proliferation, and activation in the virally active group, indicating partially functional CD8 T cells can still contribute to short-term viral control. Our analysis functionally integrates participant-specific immunophenotypic data to allow quantification of the extent, mechanisms, and impact of immune dysfunction in perinatal HIV and could inform pediatric HIV remission and cure strategies. Author Summary Cytotoxic CD8+ T cells are vital for our ability to kill infected cells. However, chronic stimulation of CD8+ T cells can result in T cell exhaustion, leading to cellular dysfunction and reduced ability to fight infections. Here we ask: What impact does CD8+ T cell exhaustion have on the ability to control HIV replication over the short term? We answer this using a computational model, anchored in experimental data from children with perinatal HIV. We use participant-specific data on dozens of immune markers that characterize the state of their immune system. We carefully map these markers to known immune cell functions and use our model to predict how immune cells with these participant-specific markers would respond to HIV infection. Our results suggest that earlier viral suppression can lead to better immune function in children with perinatal HIV. Our findings also suggest that exhausted CD8+ T cells could still contribute to fighting HIV infection, but only if there are enough of them to compensate for their reduced functionality. Our work uses computational models to put valuable participant-specific data into context, allowing us to predict short-term infection outcomes and better understand immune function in children with perinatal HIV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feef85b43caeefacd242e0fed705966cf8ef6533" target='_blank'>Partially-functional exhausted CD8+ T cells can contribute to short-term viral suppression: a computational prediction for children with perinatal HIV</a></td>
          <td>
            Alexis Hoerter, Alexa Petrucciani, Fatma Marshed, Mussa Mwamzuka, Aabid A. Ahmed, Alka Khaitan, E. Pienaar
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b72639d6e59b797a501444b04af3436df03cac" target='_blank'>Intracellular ISG-virus interactions determine viral infection severity and persistence.</a></td>
          <td>
            A. Bouchnita, V. Volpert
          </td>
          <td>2025-09-12</td>
          <td>Journal of theoretical biology</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr id="
 To improve diagnoses and further our understanding of functional defects in subjects with primary immunodeficiency (PI) diseases, we developed a new screen to broadly study the signaling of defective human immune responses to infections. Using time-of-flight mass cytometry (CyTOF) and phospho-specific antibodies, we simultaneously examined the major signaling pathways of all circulating innate and adaptive immune cell types in response to cytokines and TLR agonists to identify signaling responses. We studied healthy controls and developed software tools to generate a systematic map of signaling responses. CyTOF successfully recapitulated established signaling responses, and further, elucidated new pathways that would not have been appreciated with traditional studies. Signaling map of controls was compared to maps generated by analyzing PI subjects with gain-of-function STAT1 and autosomal dominant STAT3 deficiency diseases. CyTOF allowed us to perform unbiased studies to uncover many ex-vivo signaling defects simultaneously to better understand how known genetic etiologies manifest functionally, and exactly which cell subtypes are affected. We identified more glaring defective signaling responses in these subjects aside from the canonical defects, elucidating the true breadth of cell subtypes and pathways affected. Broad, unbiased, functional analysis can complement whole exome sequencing to facilitate identification of novel or rare immune diseases. Comprehensive view of signaling across all immune cell subsets facilitates learning more about abnormal signaling patterns that confer susceptibility to infection or autoimmunity, improving our understanding of immune responses and can lead to new therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2823684aa13e4e2575d6d1da7cd6194c358c3ce5" target='_blank'>Systems Biology Approach to Uncover Signaling Defects in Primary Immunodeficiency Diseases</a></td>
          <td>
            Jeff Choi, R. Fernandez, H. Maecker, M. Butte
          </td>
          <td>2016-05-01</td>
          <td>The Journal of Immunology</td>
          <td>2</td>
          <td>77</td>
        </tr>
    
        <tr id="With the increasing application of artificial intelligence (AI) in medical research, studies on the human immunodeficiency virus type 1(HIV-1) and acquired immunodeficiency syndrome (AIDS) have become more in-depth. Integrating AI with technologies like single-cell sequencing enables precise biomarker identification and improved therapeutic targeting. This review aims to explore the advancements in AI technologies and their applications across various facets of HIV research, including viral mechanisms, diagnostic innovations, therapeutic strategies, and prevention efforts. Despite challenges like data limitations and model interpretability, AI holds significant potential in advancing HIV-1 management and contributing to global health goals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3723412fe43b510911ce44ce7719b8fe3d629" target='_blank'>AI applications in HIV research: advances and future directions</a></td>
          <td>
            Ruyi Jin, Li Zhang
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Microbiology</td>
          <td>7</td>
          <td>1</td>
        </tr>
    
        <tr id="Numerous HIV related outcomes can be determined on the viral genome, for example, resistance associated mutations, population transmission dynamics, viral heritability traits, or time since infection. Viral sequences of people with HIV (PWH) are therefore essential for therapeutic and research purposes. While in the first three decades of the HIV pandemic viral genomes were mainly sequenced using Sanger sequencing, the last decade has seen a shift towards next-generation sequencing (NGS) as the preferred method. NGS can achieve near full length genome sequence coverage and simultaneously, it accurately encapsulates the within-host diversity by characterizing HIV subpopulations. NGS opens new avenues for HIV research, but it also presents challenges concerning data management and analysis. We therefore set up the Swiss HIV Cohort Study Viral NGS Database (SHCND) to address key issues in the handling of NGS data including high loads of raw- and processed NGS data, data storage solutions, downstream application of sophisticated bioinformatic tools, high-performance computing resources, and reproducibility. The database is nested within the Swiss HIV Cohort Study (SHCS) and the Zurich Primary HIV Infection Cohort Study (ZPHI), which together enrolled 21,876 PWH since 1988 and include a biobank dating back to the early nineties. Since its initiation in 2018, the SHCND accumulated NGS sequences (plasma and proviral origin) of 5,178 unique PWH. We here describe the design, set-up, and use of this NGS database. Overall, the SHCND has contributed to several research projects on HIV pathogenesis, treatment, drug resistance, and molecular epidemiology, and has thereby become a central part of HIV-genomics research in Switzerland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1ae2695568c7b9933f404942071b7dfee9c31bf" target='_blank'>Addressing data management and analysis challenges in viral genomics: The Swiss HIV cohort study viral next generation sequencing database</a></td>
          <td>
            Marius Zeeb, Paul Frischknecht, S. Balakrishna, Lisa Jörimann, Jasmin Tschumi, Levente Zsichla, Sandra E. Chaudron, Bashkim Jaha, Kathrin Neumann, Christine Leemann, M. Huber, K. Leuzinger, H. Günthard, K. Metzner, R. Kouyos
          </td>
          <td>2025-04-01</td>
          <td>PLOS Digital Health</td>
          <td>3</td>
          <td>97</td>
        </tr>
    
        <tr id="[This corrects the article DOI: 10.1371/journal.pcbi.1003973.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed729e4a2bb0086f320dea2c6e9874b4ebd93f2" target='_blank'>Correction: Comprehensive Sieve Analysis of Breakthrough HIV-1 Sequences in the RV144 Vaccine Efficacy Trial</a></td>
          <td>
            P. Edlefsen, M. Rolland, T. Hertz, S. Tovanabutra, A. Gartland, Allan C. deCamp, Craig A. Magaret, H. Ahmed, R. Gottardo, M. Juraska, Connor McCoy, B. Larsen, E. Sanders-Buell, Chris Carrico, Sergey Menis, G. Kijak, M. Bose, Sequencing Team, M. Arroyo, Robert J O 'connell, S. Nitayaphan, P. Pitisuttithum, J. Kaewkungwal, S. Rerks-Ngarm, M. Robb, Tatsiana Kirys, I. Georgiev, P. Kwong, K. Scheffler, Sergei L. Kosakovsky Pond, J. Carlson, N. Michael, W. Schief, J. Mullins, Jerome H. Kim, P. Gilbert
          </td>
          <td>2016-01-01</td>
          <td>PLoS Computational Biology</td>
          <td>1</td>
          <td>120</td>
        </tr>
    
        <tr id="HIV continues to be a major global health concern, impacting millions.  While antiretroviral therapy (ART) has dramatically improved outcomes, transforming HIV into a manageable chronic condition rather than a death sentence, a cure remains elusive. This mini-review examines the intricacies of HIV infection and explores novel curative strategies.  Key challenges include the persistence of latent HIV reservoirs, where the virus lies dormant and undetectable and the emergence of drug resistance due to HIV's high mutation rate.  Promising avenues of research include using Artificial Intelligence (AI) and Machine Learning (ML) to improve drug discovery and personalize treatment.  Nanoparticles are being developed to deliver drugs directly to infected cells, improving effectiveness and minimizing side effects.  Synthetic biology approaches, such as CAR-T and CAR-NK cells, aim to engineer immune cells to target and eliminate infected cells.  Gut microbiome modulation is also being investigated as a way to boost immune response and reduce viral reservoirs.  RNA interference (RNAi) uses siRNAs to suppress viral gene expression.  Finally, therapeutic exosomes offer a new way to deliver antiviral agents to infected cells. These cutting-edge strategies offer hope for a functional or sterilizing cure.  Further research and clinical trials are crucial to optimize these technologies and translate them into clinical practice, ultimately aiming to eliminate HIV and end the global epidemic.
                 
Peer Review History: 
Received 5 December 2024;   Reviewed 11 January 2025; Accepted 20 February; Available online 15 March 2025
Academic Editor: Dr. Emmanuel O. Olorunsola, Department of Pharmaceutics & Pharmaceutical Technology, University of Uyo, Nigeria, olorunsolaeo@yahoo.com">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d31fb1d23ff8bcbb9fc115e77b0694264f41bab" target='_blank'>A BRIEF OVERVIEW OF HIV TREATMENT STRATEGIES: A FOCUSED LOOK AT CHALLENGES AND OPPORTUNITIES</a></td>
          <td>
            Mostafa Essam Eissa
          </td>
          <td>2025-03-15</td>
          <td>Universal Journal of Pharmaceutical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>