---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-03 09:14:04 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Predicting individual vaccine responses remains a significant challenge due to the complexity and variability of immune processes. To address this gap, we developed immunaut, an open-source, data-driven framework implemented as an R package specifically designed for all systems vaccinologists seeking to analyze and predict immunological outcomes across diverse vaccination settings. Leveraging one of the most comprehensive live attenuated influenza vaccine (LAIV) datasets to date - 244 Gambian children enrolled in a phase 4 immunogenicity study - immunaut integrates humoral, mucosal, cellular, transcriptomic, and microbiological parameters collected before and after vaccination, providing an unprecedentedly holistic view of LAIV-induced immunity. Through advanced dimensionality reduction, clustering, and predictive modeling, immunaut identifies distinct immunophenotypic responder profiles and their underlying baseline determinants. In this study, immunaut delineated three immunophenotypes: (1) CD8 T-cell responders, marked by strong baseline mucosal immunity and extensive prior influenza virus exposure that boosts memory CD8 T-cell responses, without generating influenza virus-specific antibody responses; (2) Mucosal responders, characterized by pre-existing systemic influenza A virus immunity (specifically to H3N2) and stable epithelial integrity, leading to potent mucosal IgA expansions and subsequent seroconversion to influenza B virus; and (3) Systemic, broad influenza A virus responders, who start with relatively naive immunity and leverage greater initial viral replication to drive broad systemic antibody responses against multiple influenza A virus variants beyond those included in the LAIV vaccine. By integrating pathway-level analysis, model-derived contribution scores, and hierarchical decision rules, immunaut elucidates how distinct immunological landscapes shape each response trajectory and how key baseline features, including pre-existing immunity, mucosal preparedness, and cellular support, dictate vaccine outcomes. Collectively, these findings emphasize the power of integrative, predictive frameworks to advance precision vaccinology, and highlight immunaut as a versatile, community-available resource for optimizing immunization strategies across diverse populations and vaccine platforms. One-Sentence Summary Using one of the most comprehensive LAIV datasets compiled to date, immunaut, an integrative machine learning framework, identifies distinct immunophenotypic responder groups shaped by baseline immune landscapes, advancing precision vaccinology and guiding more effective, personalized immunization strategies. Graphical abstract Immunaut, an automated framework for mapping and predicting vaccine response immunotypes. Step 1 outlines the identification of vaccine response outcomes using pre- and post-vaccination data integration across immune features, including antibodies, flu-specific T-cells, and immunophenotyping at mucosal and systemic sites. Clustering methods define the vaccine response landscape, stability, and validation through t-SNE-based visualization. Step 2 leverages an automated machine learning modeling approach, to enhance the accuracy and interpretability of vaccine response predictions, enabling stratification and targeted intervention strategies for personalized vaccine immunogenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dfad70bdcf6c0501623eb1305faf8c365faa1a5" target='_blank'>Integrative Mapping of Pre-existing Immune Landscapes for Vaccine Response Prediction</a></td>
          <td>
            Stephanie Hao, Ivan Tomic, B. Lindsey, Y. Jagne, Katja Hoschler, Adam Meijer, Juan Manuel Carreño Quiroz, Philip Meade, Kaori Sano, Chikondi Peno, Andre G. Costa-Martins, Debby Bogaert, B. Kampmann, H. Nakaya, F. Krammer, Thushan de Silva, Adriana Tomic
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20d2a886a2819555f1b29519193323bf6535394" target='_blank'>The Immune Signatures data resource, a compendium of systems vaccinology datasets</a></td>
          <td>
            J. Diray-Arce, H. Miller, E. Henrich, B. Gerritsen, Matthew P. Mulè, S. Fourati, Jeremy P. Gygi, T. Hagan, Lewis E. Tomalin, D. Rychkov, Dmitri Kazmin, D. G. Chawla, Hailong Meng, P. Dunn, John Campbell, , Alison Deckhut-Augustine, R. Gottardo, Elias K. Haddad, D. Hafler, Eva Harris, Donna Farber, Ofer Levy, J. McElrath, R. Montgomery, Bjoern Peters, Adeeb Rahman, Elaine F. Reed, Nadine Rouphael, A. Fernández-Sesma, Alessandro Sette, Ken Stuart, A. Togias, John S. Tsang, M. Sarwal, John S. Tsang, Ofer Levy, B. Pulendran, R. Sékaly, A. Floratos, R. Gottardo, S. Kleinstein, M. Suárez-Fariñas
          </td>
          <td>2022-10-20</td>
          <td>Scientific Data</td>
          <td>17</td>
          <td>100</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a952e012e1800bbd1317b1a754da406bbe7cda" target='_blank'>Systems analysis of human innate immunity in COVID-19</a></td>
          <td>
            Sophie Müller, J. Schultze
          </td>
          <td>2023-05-01</td>
          <td>Seminars in Immunology</td>
          <td>4</td>
          <td>94</td>
        </tr>
    
        <tr id="Individuals with systemic symptoms long after COVID-19 has cleared represent approximately ~10% of all COVID-19 infected individuals. Here we present a bioinformatics approach to predict and model the phases of COVID so that effective treatment strategies can be devised and monitored. We investigated 144 individuals including normal individuals and patients spanning the COVID-19 disease continuum. We collected plasma and isolated PBMCs from 29 normal individuals, 26 individuals with mild-moderate COVID-19, 25 individuals with severe COVID-19, and 64 individuals with Chronic COVID-19 symptoms. Immune subset profiling and a 14-plex cytokine panel were run on all patients. Data was analyzed using machine learning methods to predict and distinguish the groups from each other.Using a multi-class deep neural network classifier to better fit our prediction model, we recapitulated a 100% precision, 100% recall and F1 score of 1 on the test set. Moreover, a first score specific for the chronic COVID-19 patients was defined as S1 = (IFN-γ + IL-2)/ CCL4-MIP-1β. Second, a score specific for the severe COVID-19 patients was defined as S2 = (10*IL-10 + IL-6) - (IL-2 + IL-8). Severe cases are characterized by excessive inflammation and dysregulated T cell activation, recruitment, and counteracting activities. While chronic patients are characterized by a profile able to induce the activation of effector T cells with pro-inflammatory properties and the capacity of generating an effective immune response to eliminate the virus but without the proper recruitment signals to attract activated T cells. Summary Immunologic Modeling of Severity and Chronicity of COVID-19">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4012e0de3b3e4a093b5e764e7fb3cf18b48b6f21" target='_blank'>Immune-Based Prediction of COVID-19 Severity and Chronicity Decoded Using Machine Learning</a></td>
          <td>
            B. Patterson, J. Guevara-Coto, Ram Yogendra, E. B. Francisco, Emily Long, Amruta Pise, Hallison Rodrigues, P. Parikh, Javier Mora, R. Mora-Rodríguez
          </td>
          <td>2020-12-17</td>
          <td>Frontiers in Immunology</td>
          <td>130</td>
          <td>42</td>
        </tr>
    
        <tr id="Simple Summary It is known that COVID-19 causes dynamic changes in the immune system. At different stages of the course of COVID-19, the immune cells may exhibit different patterns, which have not been fully uncovered. In this study, a machine-learning-based method was designed to deeply analyze the scRNA-seq data of three types of immune cells from patients with COVID-19, including B cells, T cells, and myeloid cells. Four levels of COVID-19 severity/outcome were involved for each cell type. As a result, several essential genes were obtained, and some of them could be confirmed to be related to SARS-CoV-2 infection. Abstract As COVID-19 develops, dynamic changes occur in the patient’s immune system. Changes in molecular levels in different immune cells can reflect the course of COVID-19. This study aims to uncover the molecular characteristics of different immune cell subpopulations at different stages of COVID-19. We designed a machine learning workflow to analyze scRNA-seq data of three immune cell types (B, T, and myeloid cells) in four levels of COVID-19 severity/outcome. The datasets for three cell types included 403,700 B-cell, 634,595 T-cell, and 346,547 myeloid cell samples. Each cell subtype was divided into four groups, control, convalescence, progression mild/moderate, and progression severe/critical, and each immune cell contained 27,943 gene features. A feature analysis procedure was applied to the data of each cell type. Irrelevant features were first excluded according to their relevance to the target variable measured by mutual information. Then, four ranking algorithms (last absolute shrinkage and selection operator, light gradient boosting machine, Monte Carlo feature selection, and max-relevance and min-redundancy) were adopted to analyze the remaining features, resulting in four feature lists. These lists were fed into the incremental feature selection, incorporating three classification algorithms (decision tree, k-nearest neighbor, and random forest) to extract key gene features and construct classifiers with superior performance. The results confirmed that genes such as PFN1, RPS26, and FTH1 played important roles in SARS-CoV-2 infection. These findings provide a useful reference for the understanding of the ongoing effect of COVID-19 development on the immune system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dde4a9a1829d30b2f8ec8592b82f892c43656bc" target='_blank'>Identification of Gene Markers Associated with COVID-19 Severity and Recovery in Different Immune Cell Subtypes</a></td>
          <td>
            Ji Ren, Qian Gao, Xiao-Chao Zhou, Lei Chen, Wei Guo, Kai-Yan Feng, Lin Lu, Tao Huang, Yunpeng Cai
          </td>
          <td>2023-07-01</td>
          <td>Biology</td>
          <td>4</td>
          <td>41</td>
        </tr>
    
        <tr id="The Next-Gen Global Health 6.0 initiative offers an integrative model employing genomics, nano-immunotherapies, and artificial intelligence (AI) to address the escalating complexity of global health issues, particularly the convergence of infectious and chronic diseases. This framework advances precision medicine by integrating real-time genomic surveillance with AI algorithms, enabling timely prediction and response to outbreaks, as well as tailored therapeutic approaches. Nano-immunotherapies play a critical role in modulating immune responses with high specificity, especially in chronic infections and diseases resistant to conventional treatments. Through these synergistic technologies, the Next-Gen Global Health 6.0 approach aims to transcend traditional healthcare boundaries, offering scalable, data-driven interventions that are adaptable to varying resource levels worldwide. Emphasizing accessibility and equity, this framework highlights the necessity for innovative health policies and interdisciplinary collaboration to optimize deployment in underserved regions, ultimately contributing to sustainable, resilient healthcare systems prepared for evolving global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90bd1c7266bdf43344b615ce022812acfc7bb57" target='_blank'>Next-Gen Global Health 6.0: Integrative Genomics, Nano-Immunotherapies, and AI for Infectious and Chronic Disease Convergence</a></td>
          <td>
            Dito Anurogo
          </td>
          <td>2025-01-30</td>
          <td>International Journal of Health and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790e3a8ad8718788e35b3a0fdd34cb22338162ef" target='_blank'>A guide to systems-level immunomics</a></td>
          <td>
            L. Bonaguro, J. Schulte-Schrepping, T. Ulas, A. Aschenbrenner, M. Beyer, J. Schultze
          </td>
          <td>2022-09-22</td>
          <td>Nature Immunology</td>
          <td>46</td>
          <td>94</td>
        </tr>
    
        <tr id="
 Continued advances in single cell gene detection sensitivity and the ability to analyze data quickly with biological context are critical for discoveries in therapeutic research. The second version of the Chromium Single Cell Immune Profiling Solution by 10x Genomics enables highly sensitive detection of gene expression, full-length paired T-cell alpha- and beta-chain and immunoglobulin sequences, T-cell antigen specificity, and cell surface protein expression from the same single cells, allowing a comprehensive view of the immune response at the cellular level. The new workflow provides a 45% increase in the number of genes detected per cell and up to a 25% increase in the cells detected with paired full-length V(D)J receptor sequences in melanoma tumor derived cells. Using a new version of Cell Ranger (v5.0), clonotypes were grouped from >30,000 cells, increasing the power to detect small clonotype expansions with fewer total cells. In addition, the new software enabled comparison of pre- and post-influenza vaccination B-cell receptor sequences from a single donor, identifying post-vaccination specific clonotypes. These technological and informatics advancements enhance a researcher’s ability to perform a broad characterization of immune cell populations at unprecedented throughput and resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/156b256b1e23e17af3b5bb92d9ae493b6da8676b" target='_blank'>Technological advancements in multiomic single cell immune profiling and analysis</a></td>
          <td>
            Ariel E. Royall, David Sukovich, Bruce A. Adams, Javelin C. Chi, Daniel S. Reyes, Alaina Puleo, Sreenath M. Krishnan, D. Jaffe, Wyatt J. McDonnell, Denise Cheung, Samuel Marrs, Tina Chen, Niranjan J Srinavas, Natasha J Carli, Luz Montesclaros, J. Lau, S. Taylor
          </td>
          <td>2021-05-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>46</td>
        </tr>
    
        <tr id="Background Single cell RNA sequencing provides unprecedented opportunity to simultaneously explore the transcriptomic and immune receptor diversity of T and B cells. However, there are limited tools available that simultaneously analyse large multi-omics datasets integrated with metadata such as patient and clinical information. Results We developed VDJView, which permits the simultaneous or independent analysis and visualisation of gene expression, immune receptors, and clinical metadata of both T and B cells. This tool is implemented as an easy-to-use R shiny web-application, which integrates numerous gene expression and TCR analysis tools, and accepts data from plate-based sorted or high-throughput single cell platforms. We utilised VDJView to analyse several 10X scRNA-seq datasets, including a recent dataset of 150,000 CD8+ T cells with available gene expression, TCR sequences, quantification of 15 surface proteins, and 44 antigen specificities (across viruses, cancer, and self-antigens). We performed quality control, filtering of tetramer non-specific cells, clustering, random sampling and hypothesis testing to discover antigen specific gene signatures which were associated with immune cell differentiation states and clonal expansion across the pathogen specific T cells. We also analysed 563 single cells (plate-based sorted) obtained from 11 subjects, revealing clonally expanded T and B cells across primary cancer tissues and metastatic lymph-node. These immune cells clustered with distinct gene signatures according to the breast cancer molecular subtype. VDJView has been tested in lab meetings and peer-to-peer discussions, showing effective data generation and discussion without the need to consult bioinformaticians. Conclusions VDJView enables researchers without profound bioinformatics skills to analyse immune scRNA-seq data, integrating and visualising this with clonality and metadata profiles, thus accelerating the process of hypothesis testing, data interpretation and discovery of cellular heterogeneity. VDJView is freely available at https://bitbucket.org/kirbyvisp/vdjview.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d4f7c7f23908f09d99ea4f349d0fc07d56ee4fa" target='_blank'>Exploring and analysing immune single cell multi-omics data with VDJView</a></td>
          <td>
            J. Samir, Simone Rizzetto, Money Gupta, F. Luciani
          </td>
          <td>2019-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>
    
        <tr id="Over the last several years, next-generation sequencing and its recent push toward single-cell resolution have transformed the landscape of immunology research by revealing novel complexities about...">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b09b8492de717806723f5cff099cd6de390199" target='_blank'>Data Integration for Immunology</a></td>
          <td>
            S. Pineda, D. Bunis, I. Kosti, M. Sirota
          </td>
          <td>2020-07-20</td>
          <td></td>
          <td>5</td>
          <td>39</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>