---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-16 09:43:47 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="ABSTRACT The human gastrointestinal tract is home to a diverse community of microorganisms from all domains of life, collectively referred to as the gut microbiome. While gut bacteria have been studied extensively in relation to human host health and physiology, other constituents remain underexplored. This includes the gut virome, the collection of bacteriophages, eukaryotic viruses, and other mobile genetic elements present in the intestine. Like gut bacteria, the gut virome has been causatively linked to human health and disease. However, the gut virome is substantially more difficult to characterize, given its high diversity and complexity, as well as multiple challenges related to in vitro cultivation and in silico detection and annotation. In this mini-review, we describe various methodologies for examining the gut virome using both culture-dependent and culture-independent tools. We highlight in vitro and in vivo approaches to cultivate viruses and characterize viral-bacterial host dynamics, as well as high-throughput screens to interrogate these relationships. We also outline a general workflow for identifying and characterizing uncultivated viral genomes from fecal metagenomes, along with several key considerations throughout the process. More broadly, we aim to highlight the opportunities to synergize and streamline wet- and dry-lab techniques to robustly and comprehensively interrogate the human gut virome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258fd70f50f39ad9a4b78956201edf400391bf60" target='_blank'>Tools and approaches to study the human gut virome: from the bench to bioinformatics</a></td>
          <td>
            Haley Anne Hallowell, Justin Malogan, Jotham Suez
          </td>
          <td>2026-02-04</td>
          <td>mSystems</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr id="Human immunity relies on the coordinated responses of many cellular subsets and functional states. Inter-individual variations in cellular composition and communication could thus potentially alter host protection. Here, we explore this hypothesis by applying single-cell RNA-sequencing to examine viral responses among the dendritic cells (DCs) of three elite controllers (ECs) of HIV-1 infection. To overcome the potentially confounding effects of donor-to-donor variability, we present a generally applicable computational framework for identifying reproducible patterns in gene expression across donors who share a unifying classification. Applying it, we discover a highly functional antiviral DC state in ECs whose fractional abundance after in vitro exposure to HIV-1 correlates with higher CD4+ T cell counts and lower HIV-1 viral loads, and that effectively primes polyfunctional T cell responses in vitro. By integrating information from existing genomic databases into our reproducibility-based analysis, we identify and validate select immunomodulators that increase the fractional abundance of this state in primary peripheral blood mononuclear cells from healthy individuals in vitro. Overall, our results demonstrate how single-cell approaches can reveal previously unappreciated, yet important, immune behaviors and empower rational frameworks for modulating systems-level immune responses that may prove therapeutically and prophylactically useful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdaaa8164fd46945764987c98bec307fda098c2" target='_blank'>A Reproducibility-Based Computational Framework Identifies an Inducible, Enhanced Antiviral State in Dendritic Cells from HIV-1 Elite Controllers</a></td>
          <td>
            E. Martín-Gayo, Michael Cole, Kellie E. Kolb, Zhengyu Ouyang, Jacqueline Cronin, S. Kazer, J. Ordovas-Montañes, M. Lichterfeld, B. Walker, N. Yosef, A. Shalek, Xu G. Yu
          </td>
          <td>2018-01-29</td>
          <td>Genome Biology</td>
          <td>47</td>
          <td>158</td>
        </tr>
    
        <tr id="
 
 Viruses are a constant threat to global health as highlighted by the current COVID-19 pandemic. Currently, lack of data underlying how the human host interacts with viruses, including the SARS-CoV-2 virus, limits effective therapeutic intervention. We introduce Viral-Track, a computational method that globally scans unmapped scRNA-seq data for the presence of viral RNA, enabling transcriptional cell sorting of infected versus bystander cells. We demonstrate the sensitivity and specificity of Viral-Track to systematically detect viruses from multiple models of infection, including hepatitis B virus in an unsupervised manner. Applying Viral-Track to Bronchoalveloar-Lavage samples from severe and mild COVID-19 patients reveals a dramatic impact of the virus on the immune system of severe patients compared to mild cases. Viral-Track detects an unexpected co-infection of the human MetaPneumoVirus, present mainly in monocytes perturbed in type-I IFN-signaling. Viral-Track provides a robust technology for dissecting the mechanisms of viral-infection and pathology.
 
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>445</td>
          <td>106</td>
        </tr>
    
        <tr id="We and others have reported evidence of T cell exhaustion in children with perinatal HIV with increased expression of inhibitory receptors PD-1, CD160, and TIM-3, but there is limited data on the virologic functional consequences of this immune exhaustion. We address this by using an immune database from Kenyan children with perinatal HIV and unexposed controls. We computationally integrate T cell profiles of differentiation, activation and exhaustion in an agent-based model (ABM) to predict how T cell exhaustion impacts viral control following HIV exposure in vitro. Our ABM includes macrophages, CD4 and CD8 T cells, cytokines, and HIV. Model mechanisms include viral dynamics, macrophage activation, T cell activation and proliferation, cytotoxic T cell killing, and cytokine/HIV diffusion and degradation. Participants are grouped by HIV plasma viremia and by age, less than 5 years or 5-18 years. Our findings indicate that cells from virally active participants, who have the highest levels of exhaustion, have lower predicted viral concentrations and infected cells compared to other participant groups during new infection. However, this coincides with higher cell death, suggesting that short-term viral control is associated with excessive inflammation, which could be detrimental long-term. Cells from virally suppressed participants older than 5 years can maintain lower viral concentrations while limiting cell death, reflecting a more sustainable short-term immune response. In virally suppressed children younger than 5 years, immune response patterns strongly resemble the age-matched healthy control group, suggesting early viral suppression may preserve antiviral immune responses. Our model predicts unique patterns of cell death for each participant group, with CD8 T cell death being dominant in virally active groups and CD4 T cell and macrophage death being dominant in healthy and virally suppressed groups. Finally, exhausted CD8 T cells are predicted to contribute significantly to CD8 T cell killing, proliferation, and activation in the virally active group, indicating partially functional CD8 T cells can still contribute to short-term viral control. Our analysis functionally integrates participant-specific immunophenotypic data to allow quantification of the extent, mechanisms, and impact of immune dysfunction in perinatal HIV and could inform pediatric HIV remission and cure strategies. Author Summary Cytotoxic CD8+ T cells are vital for our ability to kill infected cells. However, chronic stimulation of CD8+ T cells can result in T cell exhaustion, leading to cellular dysfunction and reduced ability to fight infections. Here we ask: What impact does CD8+ T cell exhaustion have on the ability to control HIV replication over the short term? We answer this using a computational model, anchored in experimental data from children with perinatal HIV. We use participant-specific data on dozens of immune markers that characterize the state of their immune system. We carefully map these markers to known immune cell functions and use our model to predict how immune cells with these participant-specific markers would respond to HIV infection. Our results suggest that earlier viral suppression can lead to better immune function in children with perinatal HIV. Our findings also suggest that exhausted CD8+ T cells could still contribute to fighting HIV infection, but only if there are enough of them to compensate for their reduced functionality. Our work uses computational models to put valuable participant-specific data into context, allowing us to predict short-term infection outcomes and better understand immune function in children with perinatal HIV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feef85b43caeefacd242e0fed705966cf8ef6533" target='_blank'>Partially-functional exhausted CD8+ T cells can contribute to short-term viral suppression: a computational prediction for children with perinatal HIV</a></td>
          <td>
            Alexis Hoerter, Alexa Petrucciani, Fatma Marshed, Mussa Mwamzuka, Aabid A. Ahmed, Alka Khaitan, E. Pienaar
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b72639d6e59b797a501444b04af3436df03cac" target='_blank'>Intracellular ISG-virus interactions determine viral infection severity and persistence.</a></td>
          <td>
            A. Bouchnita, V. Volpert
          </td>
          <td>2025-09-12</td>
          <td>Journal of theoretical biology</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr id="We have developed a full genome virus detection process that combines sensitive nucleic acid preparation optimised for virus identification in fecal material with Illumina MiSeq sequencing and a novel post-sequencing virus identification algorithm. Enriched viral nucleic acid was converted to double-stranded DNA and subjected to Illumina MiSeq sequencing. The resulting short reads were processed with a novel iterative Python algorithm SLIM for the identification of sequences with homology to known viruses. De novo assembly was then used to generate full viral genomes. The sensitivity of this process was demonstrated with a set of fecal samples from HIV-1 infected patients. A quantitative assessment of the mammalian, plant, and bacterial virus content of this compartment was generated and the deep sequencing data were sufficient to assembly 12 complete viral genomes from 6 virus families. The method detected high levels of enteropathic viruses that are normally controlled in healthy adults, but may be involved in the pathogenesis of HIV-1 infection and will provide a powerful tool for virus detection and for analyzing changes in the fecal virome associated with HIV-1 progression and pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab98d1b125aa0704e63adef426b27abd32e935f0" target='_blank'>Full Genome Virus Detection in Fecal Samples Using Sensitive Nucleic Acid Preparation, Deep Sequencing, and a Novel Iterative Sequence Classification Algorithm</a></td>
          <td>
            M. Cotten, B. O. Oude Munnink, M. Canuti, M. Deijs, S. Watson, P. Kellam, L. van der Hoek
          </td>
          <td>2014-04-02</td>
          <td>PLoS ONE</td>
          <td>73</td>
          <td>78</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8abba679e2263c0dd6b357ce0b9b74a569cd8b2" target='_blank'>Ten computational challenges in human virome studies</a></td>
          <td>
            Yifan Wu, Yousong Peng
          </td>
          <td>2024-02-23</td>
          <td>Virologica Sinica</td>
          <td>7</td>
          <td>4</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>