---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-17 09:14:09 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdaaa8164fd46945764987c98bec307fda098c2" target='_blank'>A Reproducibility-Based Computational Framework Identifies an Inducible, Enhanced Antiviral State in Dendritic Cells from HIV-1 Elite Controllers</a></td>
          <td>
            E. Martín-Gayo, Michael Cole, Kellie E. Kolb, Zhengyu Ouyang, Jacqueline Cronin, S. Kazer, J. Ordovas-Montañes, M. Lichterfeld, B. Walker, N. Yosef, A. Shalek, Xu G. Yu
          </td>
          <td>2018-01-29</td>
          <td>Genome Biology</td>
          <td>43</td>
          <td>159</td>
        </tr>
    
        <tr id="Predicting individual vaccine responses remains a significant challenge due to the complexity and variability of immune processes. To address this gap, we developed immunaut, an open-source, data-driven framework implemented as an R package specifically designed for all systems vaccinologists seeking to analyze and predict immunological outcomes across diverse vaccination settings. Leveraging one of the most comprehensive live attenuated influenza vaccine (LAIV) datasets to date - 244 Gambian children enrolled in a phase 4 immunogenicity study - immunaut integrates humoral, mucosal, cellular, transcriptomic, and microbiological parameters collected before and after vaccination, providing an unprecedentedly holistic view of LAIV-induced immunity. Through advanced dimensionality reduction, clustering, and predictive modeling, immunaut identifies distinct immunophenotypic responder profiles and their underlying baseline determinants. In this study, immunaut delineated three immunophenotypes: (1) CD8 T-cell responders, marked by strong baseline mucosal immunity and extensive prior influenza virus exposure that boosts memory CD8 T-cell responses, without generating influenza virus-specific antibody responses; (2) Mucosal responders, characterized by pre-existing systemic influenza A virus immunity (specifically to H3N2) and stable epithelial integrity, leading to potent mucosal IgA expansions and subsequent seroconversion to influenza B virus; and (3) Systemic, broad influenza A virus responders, who start with relatively naive immunity and leverage greater initial viral replication to drive broad systemic antibody responses against multiple influenza A virus variants beyond those included in the LAIV vaccine. By integrating pathway-level analysis, model-derived contribution scores, and hierarchical decision rules, immunaut elucidates how distinct immunological landscapes shape each response trajectory and how key baseline features, including pre-existing immunity, mucosal preparedness, and cellular support, dictate vaccine outcomes. Collectively, these findings emphasize the power of integrative, predictive frameworks to advance precision vaccinology, and highlight immunaut as a versatile, community-available resource for optimizing immunization strategies across diverse populations and vaccine platforms. One-Sentence Summary Using one of the most comprehensive LAIV datasets compiled to date, immunaut, an integrative machine learning framework, identifies distinct immunophenotypic responder groups shaped by baseline immune landscapes, advancing precision vaccinology and guiding more effective, personalized immunization strategies. Graphical abstract Immunaut, an automated framework for mapping and predicting vaccine response immunotypes. Step 1 outlines the identification of vaccine response outcomes using pre- and post-vaccination data integration across immune features, including antibodies, flu-specific T-cells, and immunophenotyping at mucosal and systemic sites. Clustering methods define the vaccine response landscape, stability, and validation through t-SNE-based visualization. Step 2 leverages an automated machine learning modeling approach, to enhance the accuracy and interpretability of vaccine response predictions, enabling stratification and targeted intervention strategies for personalized vaccine immunogenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dfad70bdcf6c0501623eb1305faf8c365faa1a5" target='_blank'>Integrative Mapping of Pre-existing Immune Landscapes for Vaccine Response Prediction</a></td>
          <td>
            Stephanie Hao, Ivan Tomic, B. Lindsey, Y. Jagne, Katja Hoschler, Adam Meijer, Juan Manuel Carreño Quiroz, Philip Meade, Kaori Sano, Chikondi Peno, Andre G. Costa-Martins, Debby Bogaert, B. Kampmann, H. Nakaya, F. Krammer, Thushan de Silva, Adriana Tomic
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>105</td>
        </tr>
    
        <tr id="
 To improve diagnoses and further our understanding of functional defects in subjects with primary immunodeficiency (PI) diseases, we developed a new screen to broadly study the signaling of defective human immune responses to infections. Using time-of-flight mass cytometry (CyTOF) and phospho-specific antibodies, we simultaneously examined the major signaling pathways of all circulating innate and adaptive immune cell types in response to cytokines and TLR agonists to identify signaling responses. We studied healthy controls and developed software tools to generate a systematic map of signaling responses. CyTOF successfully recapitulated established signaling responses, and further, elucidated new pathways that would not have been appreciated with traditional studies. Signaling map of controls was compared to maps generated by analyzing PI subjects with gain-of-function STAT1 and autosomal dominant STAT3 deficiency diseases. CyTOF allowed us to perform unbiased studies to uncover many ex-vivo signaling defects simultaneously to better understand how known genetic etiologies manifest functionally, and exactly which cell subtypes are affected. We identified more glaring defective signaling responses in these subjects aside from the canonical defects, elucidating the true breadth of cell subtypes and pathways affected. Broad, unbiased, functional analysis can complement whole exome sequencing to facilitate identification of novel or rare immune diseases. Comprehensive view of signaling across all immune cell subsets facilitates learning more about abnormal signaling patterns that confer susceptibility to infection or autoimmunity, improving our understanding of immune responses and can lead to new therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2823684aa13e4e2575d6d1da7cd6194c358c3ce5" target='_blank'>Systems Biology Approach to Uncover Signaling Defects in Primary Immunodeficiency Diseases</a></td>
          <td>
            Jeff Choi, R. Fernandez, H. Maecker, M. Butte
          </td>
          <td>2016-05-01</td>
          <td>The Journal of Immunology</td>
          <td>2</td>
          <td>77</td>
        </tr>
    
        <tr id="Mucosal-associated invariant T (MAIT) cells are a group of unconventional T cells that mainly recognize bacterial vitamin B metabolites presented on MHC-related protein 1 (MR1). MAIT cells have been shown to play an important role in controlling bacterial infection and in responding to viral infections. Furthermore, MAIT cells have been implicated in different chronic inflammatory diseases such as inflammatory bowel disease and multiple sclerosis. Despite their involvement in different physiological and pathological processes, a deeper understanding of MAIT cells is still lacking. Arguably, this can be attributed to the difficulty of quantifying and measuring MAIT cells in different biological samples which is commonly done using flow cytometry-based methods and single-cell-based RNA sequencing techniques. These methods mostly require fresh samples which are difficult to obtain, especially from tissues, have low to medium throughput, and are costly and labor-intensive. To address these limitations, we developed sequence-to-MAIT (Seq2MAIT) which is a transformer-based deep neural network capable of identifying MAIT cells in bulk TCR-sequencing datasets, enabling the quantification of MAIT cells from any biological materials where human DNA is available. Benchmarking Seq2MAIT across different test datasets showed an average area-under-the-receiver-operator-curve (AU[ROC]) >0.80. In conclusion, Seq2MAIT is a novel, economical, and scalable method for identifying and quantifying MAIT cells in virtually any biological sample.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7407ca7605f24e11c1fc4335ba413bb4681f30b9" target='_blank'>Seq2MAIT: A Novel Deep Learning Framework for Identifying Mucosal Associated Invariant T (MAIT) Cells</a></td>
          <td>
            H. ElAbd, Rachel Byron, Steven Woodhouse, Brittney Robinett, Joe Sulc, A. Franke, Mitchell Pesesky, Wenyu Zhou, Haiyin Chen-Harris, Bryan Howie, Ruth Taniguchi, Harlan S. Robins
          </td>
          <td>2024-03-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20d2a886a2819555f1b29519193323bf6535394" target='_blank'>The Immune Signatures data resource, a compendium of systems vaccinology datasets</a></td>
          <td>
            J. Diray-Arce, H. Miller, E. Henrich, B. Gerritsen, Matthew P. Mulè, S. Fourati, Jeremy P. Gygi, T. Hagan, Lewis E. Tomalin, D. Rychkov, Dmitri Kazmin, D. G. Chawla, Hailong Meng, P. Dunn, John Campbell, , Alison Deckhut-Augustine, R. Gottardo, Elias K. Haddad, D. Hafler, Eva Harris, Donna Farber, Ofer Levy, J. McElrath, R. Montgomery, Bjoern Peters, Adeeb Rahman, Elaine F. Reed, Nadine Rouphael, A. Fernández-Sesma, Alessandro Sette, Ken Stuart, A. Togias, John S. Tsang, M. Sarwal, John S. Tsang, Ofer Levy, B. Pulendran, R. Sékaly, A. Floratos, R. Gottardo, S. Kleinstein, M. Suárez-Fariñas
          </td>
          <td>2022-10-20</td>
          <td>Scientific Data</td>
          <td>18</td>
          <td>100</td>
        </tr>
    
        <tr id="Hundreds of immune cell types work in coordination to maintain tissue homeostasis. Upon infection, dramatic changes occur with the localization, migration, and proliferation of the immune cells to first alert the body of the danger, confine it to limit spreading, and finally extinguish the threat and bring the tissue back to homeostasis. Since current technologies can follow the dynamics of only a limited number of cell types, we have yet to grasp the full complexity of global in vivo cell dynamics in normal developmental processes and disease. Here, we devise a computational method, digital cell quantification (DCQ), which combines genome‐wide gene expression data with an immune cell compendium to infer in vivo changes in the quantities of 213 immune cell subpopulations. DCQ was applied to study global immune cell dynamics in mice lungs at ten time points during 7 days of flu infection. We find dramatic changes in quantities of 70 immune cell types, including various innate, adaptive, and progenitor immune cells. We focus on the previously unreported dynamics of four immune dendritic cell subtypes and suggest a specific role for CD103+ CD11b− DCs in early stages of disease and CD8+ pDC in late stages of flu infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77a0a3591292c89f8a23e67e05e4e9abcd805fa" target='_blank'>Digital cell quantification identifies global immune cell dynamics during influenza infection</a></td>
          <td>
            Z. Altboum, Yael Steuerman, Eyal David, Z. Barnett-Itzhaki, Liran Valadarsky, Hadas Keren-Shaul, Tal Meningher, E. Mendelson, M. Mandelboim, Irit Gat-Viks, I. Amit
          </td>
          <td>2014-02-01</td>
          <td>Molecular Systems Biology</td>
          <td>105</td>
          <td>101</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a952e012e1800bbd1317b1a754da406bbe7cda" target='_blank'>Systems analysis of human innate immunity in COVID-19</a></td>
          <td>
            Sophie Müller, J. Schultze
          </td>
          <td>2023-05-01</td>
          <td>Seminars in Immunology</td>
          <td>4</td>
          <td>94</td>
        </tr>
    
        <tr id="Hidradenitis suppurativa (HS) is a chronic inflammatory skin condition characterized by painful nodules, abscesses, and scarring, predominantly affecting intertriginous regions and it is often underdiagnosed. This study aimed to utilize single cell RNA and cell-surface protein sequencing (CITE-Seq) to delineate the immune composition of circulating cells in Hidradenitis suppurativa (HS) peripheral blood compared to healthy controls. CITE-Seq was used to analyze the gene and protein expression profiles of peripheral blood mononuclear cells (PBMCs) from 9 HS and 29 healthy controls. The study identified significant differences cell composition between HS patients and healthy controls, including increased proportions of CD14+ and CD16+ monocytes, cDC2, plasmablasts, and proliferating CD4+ T cells in HS patients. Differential expression analysis revealed upregulation of inflammatory markers such as TNF, IL1B, and NF-κB in monocytes, as well as chemokines and cell adhesion molecules involved in immune cell recruitment and tissue infiltration. Pathway enrichment analysis highlighted the involvement of IL-17, IL-26 and TNF signaling pathways in HS pathogenesis. Machine learning identified key markers for diagnostics and therapeutic development. The findings also support the potential for machine learning models to aid in the diagnosis of HS based on immune cell markers. These insights may inform future therapeutic strategies targeting specific immune pathways in HS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6380ba17ee3a7e4d4ad44277aa5f9a5dc959cc2" target='_blank'>Cellular indexing of transcriptomes and epitopes (CITE-Seq) in hidradenitis suppurativa identifies dysregulated cell types in peripheral blood and facilitates diagnosis via machine learning</a></td>
          <td>
            Wilson Liao, Sugandh Kumar, Faye Orcales, Bobby Shih, Xiaohui Fang, C. Yin, Ashley Yates, P. Dimitrion, Isaac Neuhaus, Chandler E Johnson, Indra Adrianto, A. Wiala, I. Hamzavi, Li Zhou, Haley Naik, Christian Posch, Q-S Mi
          </td>
          <td>2024-09-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>57</td>
        </tr>
    
        <tr id="Individuals with systemic symptoms long after COVID-19 has cleared represent approximately ~10% of all COVID-19 infected individuals. Here we present a bioinformatics approach to predict and model the phases of COVID so that effective treatment strategies can be devised and monitored. We investigated 144 individuals including normal individuals and patients spanning the COVID-19 disease continuum. We collected plasma and isolated PBMCs from 29 normal individuals, 26 individuals with mild-moderate COVID-19, 25 individuals with severe COVID-19, and 64 individuals with Chronic COVID-19 symptoms. Immune subset profiling and a 14-plex cytokine panel were run on all patients. Data was analyzed using machine learning methods to predict and distinguish the groups from each other.Using a multi-class deep neural network classifier to better fit our prediction model, we recapitulated a 100% precision, 100% recall and F1 score of 1 on the test set. Moreover, a first score specific for the chronic COVID-19 patients was defined as S1 = (IFN-γ + IL-2)/ CCL4-MIP-1β. Second, a score specific for the severe COVID-19 patients was defined as S2 = (10*IL-10 + IL-6) - (IL-2 + IL-8). Severe cases are characterized by excessive inflammation and dysregulated T cell activation, recruitment, and counteracting activities. While chronic patients are characterized by a profile able to induce the activation of effector T cells with pro-inflammatory properties and the capacity of generating an effective immune response to eliminate the virus but without the proper recruitment signals to attract activated T cells. Summary Immunologic Modeling of Severity and Chronicity of COVID-19">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4012e0de3b3e4a093b5e764e7fb3cf18b48b6f21" target='_blank'>Immune-Based Prediction of COVID-19 Severity and Chronicity Decoded Using Machine Learning</a></td>
          <td>
            B. Patterson, J. Guevara-Coto, Ram Yogendra, E. B. Francisco, Emily Long, Amruta Pise, Hallison Rodrigues, P. Parikh, Javier Mora, R. Mora-Rodríguez
          </td>
          <td>2020-12-17</td>
          <td>Frontiers in Immunology</td>
          <td>130</td>
          <td>42</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>