---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-28 09:21:17 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Defining physiology and methods to measure biological mechanisms is essential. Extensive datasets such as RNA sequencing are used with little analysis of the knowledge gained from the various methodologies. Within this work, we have processed publicly available NCBI RNAseq datasets using a combination of bioinformatics tools for the largest physiological organ, the skin. In many datasets, we identify the quality of the sample, human transcript mapping, the sex of each sample, foreign RNA from bacteria/viruses/protists, and the presence of B/T-cell immune repertoire. Processing 8,274 samples from 132 different experiments for skin samples identifies common flora of skin with elevation of protists (such as Leishmania), bacteria (Staphylococcus, Cutibacterium acnes), and viruses (HSV, HPV) that may be involved in physiological differences. We observed samples with the Heilongjiang tick virus, Human T-cell Leukemia Virus Type I, and Equine Infectious Anemia Virus that likely play pathological roles in physiology. Integrating the various biomarkers identified five ideal datasets for skin pathologies that elucidated a novel correlation between the normal skin flora bacterium Bacillus megaterium with MHC complex regulation and the immune repertoire clonal expansion, particularly in patients with hidradenitis suppurativa. Finally, we show that in multiple independent experiments, biological sex is associated with multiple sex chromosome gene differences, highlighting the importance of future work in studying sex differences in skin. Data integrations and multidimensional data mapping are critical for physiological omics advancements, and this work highlights the exciting ability to apply these tools to skin physiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21d7405dc4848bad01933fa8e058f89b35b57024" target='_blank'>From Castaways to Discoveries: Unveiling Treasures in Skin RNAseq Using a Novel Multidimensional Data Processing Workflow Including Infection-Host Dynamics.</a></td>
          <td>
            Disha Bhargava, Alec Labadie, Rebecca L Hanson-Rios-Stutz, Austin Goodyke, Ella M Moses, Akansha S Das, Sophie Vanderweele, Janelle V Lemon, Taylor W Cook, David Pearson, Joseph M Redinger, Adam J Caulfield, Rosemary Olivero, Kate Foster, Kurt Ashack, S. Rajasekaran, Caleb P. Bupp, Timothy J. Triche, Connie M Krawczyk, Dave Chesla, Matthew D Sims, Nicholas Hartog, Jeremy W Prokop
          </td>
          <td>2025-03-12</td>
          <td>Physiological genomics</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a952e012e1800bbd1317b1a754da406bbe7cda" target='_blank'>Systems analysis of human innate immunity in COVID-19</a></td>
          <td>
            Sophie Müller, J. Schultze
          </td>
          <td>2023-05-01</td>
          <td>Seminars in Immunology</td>
          <td>5</td>
          <td>94</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32015c79bd160189ddb13fa99f034c432b94c59" target='_blank'>Probing infectious disease by single-cell RNA sequencing: Progresses and perspectives</a></td>
          <td>
            Geyang Luo, Q. Gao, Shuye Zhang, Bo Yan
          </td>
          <td>2020-10-21</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>42</td>
          <td>44</td>
        </tr>
    
        <tr id="Hidradenitis suppurativa (HS) is a chronic inflammatory skin condition characterized by painful nodules, abscesses, and scarring, predominantly affecting intertriginous regions and it is often underdiagnosed. This study aimed to utilize single cell RNA and cell-surface protein sequencing (CITE-Seq) to delineate the immune composition of circulating cells in Hidradenitis suppurativa (HS) peripheral blood compared to healthy controls. CITE-Seq was used to analyze the gene and protein expression profiles of peripheral blood mononuclear cells (PBMCs) from 9 HS and 29 healthy controls. The study identified significant differences cell composition between HS patients and healthy controls, including increased proportions of CD14+ and CD16+ monocytes, cDC2, plasmablasts, and proliferating CD4+ T cells in HS patients. Differential expression analysis revealed upregulation of inflammatory markers such as TNF, IL1B, and NF-κB in monocytes, as well as chemokines and cell adhesion molecules involved in immune cell recruitment and tissue infiltration. Pathway enrichment analysis highlighted the involvement of IL-17, IL-26 and TNF signaling pathways in HS pathogenesis. Machine learning identified key markers for diagnostics and therapeutic development. The findings also support the potential for machine learning models to aid in the diagnosis of HS based on immune cell markers. These insights may inform future therapeutic strategies targeting specific immune pathways in HS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6380ba17ee3a7e4d4ad44277aa5f9a5dc959cc2" target='_blank'>Cellular indexing of transcriptomes and epitopes (CITE-Seq) in hidradenitis suppurativa identifies dysregulated cell types in peripheral blood and facilitates diagnosis via machine learning</a></td>
          <td>
            Wilson Liao, Sugandh Kumar, Faye Orcales, Bobby Shih, Xiaohui Fang, C. Yin, Ashley Yates, P. Dimitrion, Isaac Neuhaus, Chandler E Johnson, Indra Adrianto, A. Wiala, I. Hamzavi, Li Zhou, Haley Naik, Christian Posch, Q-S Mi
          </td>
          <td>2024-09-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>58</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e096ebd7370fb9e33e17eeb039e8d87258842d" target='_blank'>Systems Virology</a></td>
          <td>
            M. J. Korth, G. L. Law
          </td>
          <td>2016-02-12</td>
          <td>Viral Pathogenesis</td>
          <td>0</td>
          <td>37</td>
        </tr>
    
        <tr id="Simple Summary It is known that COVID-19 causes dynamic changes in the immune system. At different stages of the course of COVID-19, the immune cells may exhibit different patterns, which have not been fully uncovered. In this study, a machine-learning-based method was designed to deeply analyze the scRNA-seq data of three types of immune cells from patients with COVID-19, including B cells, T cells, and myeloid cells. Four levels of COVID-19 severity/outcome were involved for each cell type. As a result, several essential genes were obtained, and some of them could be confirmed to be related to SARS-CoV-2 infection. Abstract As COVID-19 develops, dynamic changes occur in the patient’s immune system. Changes in molecular levels in different immune cells can reflect the course of COVID-19. This study aims to uncover the molecular characteristics of different immune cell subpopulations at different stages of COVID-19. We designed a machine learning workflow to analyze scRNA-seq data of three immune cell types (B, T, and myeloid cells) in four levels of COVID-19 severity/outcome. The datasets for three cell types included 403,700 B-cell, 634,595 T-cell, and 346,547 myeloid cell samples. Each cell subtype was divided into four groups, control, convalescence, progression mild/moderate, and progression severe/critical, and each immune cell contained 27,943 gene features. A feature analysis procedure was applied to the data of each cell type. Irrelevant features were first excluded according to their relevance to the target variable measured by mutual information. Then, four ranking algorithms (last absolute shrinkage and selection operator, light gradient boosting machine, Monte Carlo feature selection, and max-relevance and min-redundancy) were adopted to analyze the remaining features, resulting in four feature lists. These lists were fed into the incremental feature selection, incorporating three classification algorithms (decision tree, k-nearest neighbor, and random forest) to extract key gene features and construct classifiers with superior performance. The results confirmed that genes such as PFN1, RPS26, and FTH1 played important roles in SARS-CoV-2 infection. These findings provide a useful reference for the understanding of the ongoing effect of COVID-19 development on the immune system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dde4a9a1829d30b2f8ec8592b82f892c43656bc" target='_blank'>Identification of Gene Markers Associated with COVID-19 Severity and Recovery in Different Immune Cell Subtypes</a></td>
          <td>
            Ji Ren, Qian Gao, Xiao-Chao Zhou, Lei Chen, Wei Guo, Kai-Yan Feng, Lin Lu, Tao Huang, Yunpeng Cai
          </td>
          <td>2023-07-01</td>
          <td>Biology</td>
          <td>6</td>
          <td>42</td>
        </tr>
    
        <tr id="The Next-Gen Global Health 6.0 initiative offers an integrative model employing genomics, nano-immunotherapies, and artificial intelligence (AI) to address the escalating complexity of global health issues, particularly the convergence of infectious and chronic diseases. This framework advances precision medicine by integrating real-time genomic surveillance with AI algorithms, enabling timely prediction and response to outbreaks, as well as tailored therapeutic approaches. Nano-immunotherapies play a critical role in modulating immune responses with high specificity, especially in chronic infections and diseases resistant to conventional treatments. Through these synergistic technologies, the Next-Gen Global Health 6.0 approach aims to transcend traditional healthcare boundaries, offering scalable, data-driven interventions that are adaptable to varying resource levels worldwide. Emphasizing accessibility and equity, this framework highlights the necessity for innovative health policies and interdisciplinary collaboration to optimize deployment in underserved regions, ultimately contributing to sustainable, resilient healthcare systems prepared for evolving global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90bd1c7266bdf43344b615ce022812acfc7bb57" target='_blank'>Next-Gen Global Health 6.0: Integrative Genomics, Nano-Immunotherapies, and AI for Infectious and Chronic Disease Convergence</a></td>
          <td>
            Dito Anurogo
          </td>
          <td>2025-01-30</td>
          <td>International Journal of Health and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr id="This article discusses the application of single-cell sequencing technology in COVID-19 research. Single-cell sequencing technology can help scientists gain deeper insights into the mechanisms of COVID-19 virus infection and immune responses, and provide new ideas and targets for the treatment of COVID-19. The future development directions of single-cell sequencing technology may include data processing and analysis, technical improvements, and multi-modal fusion. By further optimizing and improving single-cell sequencing technology, we can better understand the molecular mechanisms of virus infection and immune responses, and provide more scientific and effective methods for the prevention and treatment of viral diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0688e4f9fca777c4d5d82d5fe46baabb8a89b96b" target='_blank'>Applications and Future Directions of Single-Cell Sequencing Technology in COVID-19 Research</a></td>
          <td>
            
          </td>
          <td>2023-08-08</td>
          <td>International Internal Medicine Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr id="Defective interfering (DI) genomes, or defective viral genomes (DVGs), are truncated viral genomes generated during replication of most viruses, including live viral vaccines. Among these, “panhandle” or copy-back (cb) and “hairpin” or snap-back (sb) DI genomes are generated during RNA virus replication. 5′ cb/sb DI genomes are highly relevant for viral pathogenesis since they harbor immunostimulatory properties that increase virus recognition by the innate immune system of the host. We have developed DI-tector, a user-friendly and freely available program that identifies and characterizes cb/sb genomes from next-generation sequencing (NGS) data. DI-tector confirmed the presence of 5′ cb genomes in cells infected with measles virus (MV). DI-tector also identified a novel 5′ cb genome, as well as a variety of 3′ cb/sb genomes whose existence had not previously been detected by conventional approaches in MV-infected cells. The presence of these novel cb/sb genomes was confirmed by RT-qPCR and RT-PCR, validating the ability of DI-tector to reveal the landscape of DI genome population in infected cell samples. Performance assessment using different experimental and simulated data sets revealed the robust specificity and sensitivity of DI-tector. We propose DI-tector as a universal tool for the unbiased detection of DI viral genomes, including 5′ cb/sb DI genomes, in NGS data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279c7499df0958bce2b69a6249cab9193c939d25" target='_blank'>DI-tector: defective interfering viral genomes’ detector for next-generation sequencing data</a></td>
          <td>
            Guillaume Beauclair, M. Mura, C. Combredet, F. Tangy, N. Jouvenet, A. Komarova
          </td>
          <td>2018-07-16</td>
          <td>RNA</td>
          <td>36</td>
          <td>48</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>