---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-01 09:20:22 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>439</td>
          <td>104</td>
        </tr>
    
        <tr id="We and others have reported evidence of T cell exhaustion in children with perinatal HIV with increased expression of inhibitory receptors PD-1, CD160, and TIM-3, but there is limited data on the virologic functional consequences of this immune exhaustion. We address this by using an immune database from Kenyan children with perinatal HIV and unexposed controls. We computationally integrate T cell profiles of differentiation, activation and exhaustion in an agent-based model (ABM) to predict how T cell exhaustion impacts viral control following HIV exposure in vitro. Our ABM includes macrophages, CD4 and CD8 T cells, cytokines, and HIV. Model mechanisms include viral dynamics, macrophage activation, T cell activation and proliferation, cytotoxic T cell killing, and cytokine/HIV diffusion and degradation. Participants are grouped by HIV plasma viremia and by age, less than 5 years or 5-18 years. Our findings indicate that cells from virally active participants, who have the highest levels of exhaustion, have lower predicted viral concentrations and infected cells compared to other participant groups during new infection. However, this coincides with higher cell death, suggesting that short-term viral control is associated with excessive inflammation, which could be detrimental long-term. Cells from virally suppressed participants older than 5 years can maintain lower viral concentrations while limiting cell death, reflecting a more sustainable short-term immune response. In virally suppressed children younger than 5 years, immune response patterns strongly resemble the age-matched healthy control group, suggesting early viral suppression may preserve antiviral immune responses. Our model predicts unique patterns of cell death for each participant group, with CD8 T cell death being dominant in virally active groups and CD4 T cell and macrophage death being dominant in healthy and virally suppressed groups. Finally, exhausted CD8 T cells are predicted to contribute significantly to CD8 T cell killing, proliferation, and activation in the virally active group, indicating partially functional CD8 T cells can still contribute to short-term viral control. Our analysis functionally integrates participant-specific immunophenotypic data to allow quantification of the extent, mechanisms, and impact of immune dysfunction in perinatal HIV and could inform pediatric HIV remission and cure strategies. Author Summary Cytotoxic CD8+ T cells are vital for our ability to kill infected cells. However, chronic stimulation of CD8+ T cells can result in T cell exhaustion, leading to cellular dysfunction and reduced ability to fight infections. Here we ask: What impact does CD8+ T cell exhaustion have on the ability to control HIV replication over the short term? We answer this using a computational model, anchored in experimental data from children with perinatal HIV. We use participant-specific data on dozens of immune markers that characterize the state of their immune system. We carefully map these markers to known immune cell functions and use our model to predict how immune cells with these participant-specific markers would respond to HIV infection. Our results suggest that earlier viral suppression can lead to better immune function in children with perinatal HIV. Our findings also suggest that exhausted CD8+ T cells could still contribute to fighting HIV infection, but only if there are enough of them to compensate for their reduced functionality. Our work uses computational models to put valuable participant-specific data into context, allowing us to predict short-term infection outcomes and better understand immune function in children with perinatal HIV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feef85b43caeefacd242e0fed705966cf8ef6533" target='_blank'>Partially-functional exhausted CD8+ T cells can contribute to short-term viral suppression: a computational prediction for children with perinatal HIV</a></td>
          <td>
            Alexis Hoerter, Alexa Petrucciani, Fatma Marshed, Mussa Mwamzuka, Aabid A. Ahmed, Alka Khaitan, E. Pienaar
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr id="Viral infections impose a substantial threat to human health, characterized by a wide range of pathogens, clinical manifestations, and complex immune responses. Single-cell multi-omics technologies have revolutionized the study of antiviral immunity by resolving cellular heterogeneity, transcriptional reprogramming, and clonal dynamics. However, no resource has yet comprehensively integrated such datasets in the context of viral infections. Here, we present scMOVIR, a single-cell multi-omics database for human viral infections and immune responses. The database systematically integrates transcriptomic, proteomic, and immune receptor repertoire profiles at single-cell resolution, compiling 8532 human samples across 114 viral species, subtypes, and strains. These datasets span 59 well-defined disease states, including acute and chronic infections, virus-associated malignancies, and immune-related disorders, and extend across >30 tissue types. In addition, these datasets incorporate vaccination cohorts, therapeutic interventions, and time-course models of infection. To ensure consistency and interoperability, all datasets undergo standardized preprocessing, including batch correction and unified cell-type annotation, with metadata harmonized using authoritative ontologies. scMOVIR provides user-friendly interfaces for dataset exploration and interactive visualization of cellular dynamics and molecular profiles, offering a high-resolution reference for investigating virus-host immune interactions and supporting antiviral research. The database is freely accessible atÂ https://pgx.zju.edu.cn/scmovir.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130c5b82f9725db7d1697918120f36ba3e49fd02" target='_blank'>scMOVIR: a single-cell multi-omics database for human viral infections and immune responses.</a></td>
          <td>
            Xue Zhang, Shounan Yang, Xiaobin Xu, Yitao Lin, Huaicheng Sun, Bangyu Zhu, Wenyi Zhao, Binbin Zhou, Yan Lou, Xinyu Wang, Shuqing Chen, Qiaojun He, Feng Zhu, Zhan Zhou
          </td>
          <td>2025-11-13</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b72639d6e59b797a501444b04af3436df03cac" target='_blank'>Intracellular ISG-virus interactions determine viral infection severity and persistence.</a></td>
          <td>
            A. Bouchnita, V. Volpert
          </td>
          <td>2025-09-12</td>
          <td>Journal of theoretical biology</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr id="With the increasing application of artificial intelligence (AI) in medical research, studies on the human immunodeficiency virus type 1(HIV-1) and acquired immunodeficiency syndrome (AIDS) have become more in-depth. Integrating AI with technologies like single-cell sequencing enables precise biomarker identification and improved therapeutic targeting. This review aims to explore the advancements in AI technologies and their applications across various facets of HIV research, including viral mechanisms, diagnostic innovations, therapeutic strategies, and prevention efforts. Despite challenges like data limitations and model interpretability, AI holds significant potential in advancing HIV-1 management and contributing to global health goals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3723412fe43b510911ce44ce7719b8fe3d629" target='_blank'>AI applications in HIV research: advances and future directions</a></td>
          <td>
            Ruyi Jin, Li Zhang
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Microbiology</td>
          <td>5</td>
          <td>1</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8abba679e2263c0dd6b357ce0b9b74a569cd8b2" target='_blank'>Ten computational challenges in human virome studies</a></td>
          <td>
            Yifan Wu, Yousong Peng
          </td>
          <td>2024-02-23</td>
          <td>Virologica Sinica</td>
          <td>5</td>
          <td>3</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr id="[This corrects the article DOI: 10.1371/journal.pcbi.1003973.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed729e4a2bb0086f320dea2c6e9874b4ebd93f2" target='_blank'>Correction: Comprehensive Sieve Analysis of Breakthrough HIV-1 Sequences in the RV144 Vaccine Efficacy Trial</a></td>
          <td>
            P. Edlefsen, M. Rolland, T. Hertz, S. Tovanabutra, A. Gartland, Allan C. deCamp, Craig A. Magaret, H. Ahmed, R. Gottardo, M. Juraska, Connor McCoy, B. Larsen, E. Sanders-Buell, Chris Carrico, Sergey Menis, G. Kijak, M. Bose, Sequencing Team, M. Arroyo, Robert J O 'connell, S. Nitayaphan, P. Pitisuttithum, J. Kaewkungwal, S. Rerks-Ngarm, M. Robb, Tatsiana Kirys, I. Georgiev, P. Kwong, K. Scheffler, Sergei L. Kosakovsky Pond, J. Carlson, N. Michael, W. Schief, J. Mullins, Jerome H. Kim, P. Gilbert
          </td>
          <td>2016-01-01</td>
          <td>PLoS Computational Biology</td>
          <td>1</td>
          <td>120</td>
        </tr>
    
        <tr id="ABSTRACT The human gut virome is predominantly made up of bacteriophages (phages), viruses that infect bacteria. Metagenomic studies have revealed that phages in the gut are highly individual specific and dynamic. These features make it challenging to perform meaningful cross-study comparisons. While several taxonomy frameworks exist to group phages and improve these comparisons, these strategies provide little insight into the potential effects phages have on their bacterial hosts. Here, we propose the use of predicted phage host families (PHFs) as a functionally relevant, qualitative unit of phage classification to improve these cross-study analyses. We first show that bioinformatic predictions of phage hosts are accurate at the host family level by measuring their concordance to Hi-C sequencing-based predictions in human and mouse fecal samples. Next, using phage host family predictions, we determined that PHFs reduce intra- and interindividual ecological distances compared to viral contigs in a previously published cohort of 10 healthy individuals, while simultaneously improving longitudinal virome stability. Lastly, by reanalyzing a previously published metagenomics data set with >1,000 samples, we determined that PHFs are prevalent across individuals and can aid in the detection of inflammatory bowel disease-specific virome signatures. Overall, our analyses support the use of predicted phage hosts in reducing between-sample distances and providing a biologically relevant framework for making between-sample virome comparisons. IMPORTANCE The human gut virome consists mainly of bacteriophages (phages), which infect bacteria and show high individual specificity and variability, complicating cross-study comparisons. Furthermore, existing taxonomic frameworks offer limited insight into their interactions with bacterial hosts. In this study, we propose using predicted phage host families (PHFs) as a higher-level classification unit to enhance functional cross-study comparisons. We demonstrate that bioinformatic predictions of phage hosts align with Hi-C sequencing results at the host family level in human and mouse fecal samples. We further show that PHFs reduce ecological distances and improve virome stability over time. Additionally, reanalysis of a large metagenomics data set revealed that PHFs are widespread and can help identify disease-specific virome patterns, such as those linked to inflammatory bowel disease. The human gut virome consists mainly of bacteriophages (phages), which infect bacteria and show high individual specificity and variability, complicating cross-study comparisons. Furthermore, existing taxonomic frameworks offer limited insight into their interactions with bacterial hosts. In this study, we propose using predicted phage host families (PHFs) as a higher-level classification unit to enhance functional cross-study comparisons. We demonstrate that bioinformatic predictions of phage hosts align with Hi-C sequencing results at the host family level in human and mouse fecal samples. We further show that PHFs reduce ecological distances and improve virome stability over time. Additionally, reanalysis of a large metagenomics data set revealed that PHFs are widespread and can help identify disease-specific virome patterns, such as those linked to inflammatory bowel disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3fe59fc08f0a002c4b5a3c3980de8529f06eba5" target='_blank'>Improving gut virome comparisons using predicted phage host information</a></td>
          <td>
            Michael Shamash, Anshul Sinha, C. Maurice
          </td>
          <td>2024-10-11</td>
          <td>mSystems</td>
          <td>1</td>
          <td>20</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>