---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-24 09:14:16 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdaaa8164fd46945764987c98bec307fda098c2" target='_blank'>A Reproducibility-Based Computational Framework Identifies an Inducible, Enhanced Antiviral State in Dendritic Cells from HIV-1 Elite Controllers</a></td>
          <td>
            E. Martín-Gayo, Michael Cole, Kellie E. Kolb, Zhengyu Ouyang, Jacqueline Cronin, S. Kazer, J. Ordovas-Montañes, M. Lichterfeld, B. Walker, N. Yosef, A. Shalek, Xu G. Yu
          </td>
          <td>2018-01-29</td>
          <td>Genome Biology</td>
          <td>43</td>
          <td>159</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>422</td>
          <td>101</td>
        </tr>
    
        <tr id="Predicting individual vaccine responses remains a significant challenge due to the complexity and variability of immune processes. To address this gap, we developed immunaut, an open-source, data-driven framework implemented as an R package specifically designed for all systems vaccinologists seeking to analyze and predict immunological outcomes across diverse vaccination settings. Leveraging one of the most comprehensive live attenuated influenza vaccine (LAIV) datasets to date - 244 Gambian children enrolled in a phase 4 immunogenicity study - immunaut integrates humoral, mucosal, cellular, transcriptomic, and microbiological parameters collected before and after vaccination, providing an unprecedentedly holistic view of LAIV-induced immunity. Through advanced dimensionality reduction, clustering, and predictive modeling, immunaut identifies distinct immunophenotypic responder profiles and their underlying baseline determinants. In this study, immunaut delineated three immunophenotypes: (1) CD8 T-cell responders, marked by strong baseline mucosal immunity and extensive prior influenza virus exposure that boosts memory CD8 T-cell responses, without generating influenza virus-specific antibody responses; (2) Mucosal responders, characterized by pre-existing systemic influenza A virus immunity (specifically to H3N2) and stable epithelial integrity, leading to potent mucosal IgA expansions and subsequent seroconversion to influenza B virus; and (3) Systemic, broad influenza A virus responders, who start with relatively naive immunity and leverage greater initial viral replication to drive broad systemic antibody responses against multiple influenza A virus variants beyond those included in the LAIV vaccine. By integrating pathway-level analysis, model-derived contribution scores, and hierarchical decision rules, immunaut elucidates how distinct immunological landscapes shape each response trajectory and how key baseline features, including pre-existing immunity, mucosal preparedness, and cellular support, dictate vaccine outcomes. Collectively, these findings emphasize the power of integrative, predictive frameworks to advance precision vaccinology, and highlight immunaut as a versatile, community-available resource for optimizing immunization strategies across diverse populations and vaccine platforms. One-Sentence Summary Using one of the most comprehensive LAIV datasets compiled to date, immunaut, an integrative machine learning framework, identifies distinct immunophenotypic responder groups shaped by baseline immune landscapes, advancing precision vaccinology and guiding more effective, personalized immunization strategies. Graphical abstract Immunaut, an automated framework for mapping and predicting vaccine response immunotypes. Step 1 outlines the identification of vaccine response outcomes using pre- and post-vaccination data integration across immune features, including antibodies, flu-specific T-cells, and immunophenotyping at mucosal and systemic sites. Clustering methods define the vaccine response landscape, stability, and validation through t-SNE-based visualization. Step 2 leverages an automated machine learning modeling approach, to enhance the accuracy and interpretability of vaccine response predictions, enabling stratification and targeted intervention strategies for personalized vaccine immunogenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dfad70bdcf6c0501623eb1305faf8c365faa1a5" target='_blank'>Integrative Mapping of Pre-existing Immune Landscapes for Vaccine Response Prediction</a></td>
          <td>
            Stephanie Hao, Ivan Tomic, B. Lindsey, Y. Jagne, Katja Hoschler, Adam Meijer, Juan Manuel Carreño Quiroz, Philip Meade, Kaori Sano, Chikondi Peno, Andre G. Costa-Martins, Debby Bogaert, B. Kampmann, H. Nakaya, F. Krammer, Thushan de Silva, Adriana Tomic
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>105</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2823684aa13e4e2575d6d1da7cd6194c358c3ce5" target='_blank'>Systems Biology Approach to Uncover Signaling Defects in Primary Immunodeficiency Diseases</a></td>
          <td>
            Jeff Choi, R. Fernandez, H. Maecker, M. Butte
          </td>
          <td>2016-05-01</td>
          <td>The Journal of Immunology</td>
          <td>2</td>
          <td>77</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8abba679e2263c0dd6b357ce0b9b74a569cd8b2" target='_blank'>Ten computational challenges in human virome studies</a></td>
          <td>
            Yifan Wu, Yousong Peng
          </td>
          <td>2024-02-23</td>
          <td>Virologica Sinica</td>
          <td>2</td>
          <td>2</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr id="Mucosal-associated invariant T (MAIT) cells are a group of unconventional T cells that mainly recognize bacterial vitamin B metabolites presented on MHC-related protein 1 (MR1). MAIT cells have been shown to play an important role in controlling bacterial infection and in responding to viral infections. Furthermore, MAIT cells have been implicated in different chronic inflammatory diseases such as inflammatory bowel disease and multiple sclerosis. Despite their involvement in different physiological and pathological processes, a deeper understanding of MAIT cells is still lacking. Arguably, this can be attributed to the difficulty of quantifying and measuring MAIT cells in different biological samples which is commonly done using flow cytometry-based methods and single-cell-based RNA sequencing techniques. These methods mostly require fresh samples which are difficult to obtain, especially from tissues, have low to medium throughput, and are costly and labor-intensive. To address these limitations, we developed sequence-to-MAIT (Seq2MAIT) which is a transformer-based deep neural network capable of identifying MAIT cells in bulk TCR-sequencing datasets, enabling the quantification of MAIT cells from any biological materials where human DNA is available. Benchmarking Seq2MAIT across different test datasets showed an average area-under-the-receiver-operator-curve (AU[ROC]) >0.80. In conclusion, Seq2MAIT is a novel, economical, and scalable method for identifying and quantifying MAIT cells in virtually any biological sample.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7407ca7605f24e11c1fc4335ba413bb4681f30b9" target='_blank'>Seq2MAIT: A Novel Deep Learning Framework for Identifying Mucosal Associated Invariant T (MAIT) Cells</a></td>
          <td>
            H. ElAbd, Rachel Byron, Steven Woodhouse, Brittney Robinett, Joe Sulc, A. Franke, Mitchell Pesesky, Wenyu Zhou, Haiyin Chen-Harris, Bryan Howie, Ruth Taniguchi, Harlan S. Robins
          </td>
          <td>2024-03-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20d2a886a2819555f1b29519193323bf6535394" target='_blank'>The Immune Signatures data resource, a compendium of systems vaccinology datasets</a></td>
          <td>
            J. Diray-Arce, H. Miller, E. Henrich, B. Gerritsen, Matthew P. Mulè, S. Fourati, Jeremy P. Gygi, T. Hagan, Lewis E. Tomalin, D. Rychkov, Dmitri Kazmin, D. G. Chawla, Hailong Meng, P. Dunn, John Campbell, , Alison Deckhut-Augustine, R. Gottardo, Elias K. Haddad, D. Hafler, Eva Harris, Donna Farber, Ofer Levy, J. McElrath, R. Montgomery, Bjoern Peters, Adeeb Rahman, Elaine F. Reed, Nadine Rouphael, A. Fernández-Sesma, Alessandro Sette, Ken Stuart, A. Togias, John S. Tsang, M. Sarwal, John S. Tsang, Ofer Levy, B. Pulendran, R. Sékaly, A. Floratos, R. Gottardo, S. Kleinstein, M. Suárez-Fariñas
          </td>
          <td>2022-10-20</td>
          <td>Scientific Data</td>
          <td>18</td>
          <td>100</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21d7405dc4848bad01933fa8e058f89b35b57024" target='_blank'>From Castaways to Discoveries: Unveiling Treasures in Skin RNAseq Using a Novel Multidimensional Data Processing Workflow Including Infection-Host Dynamics.</a></td>
          <td>
            Disha Bhargava, Alec Labadie, Rebecca L Hanson-Rios-Stutz, Austin Goodyke, Ella M Moses, Akansha S Das, Sophie Vanderweele, Janelle V Lemon, Taylor W Cook, David Pearson, Joseph M Redinger, Adam J Caulfield, Rosemary Olivero, Kate Foster, Kurt Ashack, S. Rajasekaran, Caleb P. Bupp, Timothy J. Triche, Connie M Krawczyk, Dave Chesla, Matthew D Sims, Nicholas Hartog, Jeremy W Prokop
          </td>
          <td>2025-03-12</td>
          <td>Physiological genomics</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>