---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-31 09:13:56 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Gastrointestinal germinal center B cell depletion and reduction in IgA+ plasma cells in HIV-1 infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>422</td>
          <td>102</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2823684aa13e4e2575d6d1da7cd6194c358c3ce5" target='_blank'>Systems Biology Approach to Uncover Signaling Defects in Primary Immunodeficiency Diseases</a></td>
          <td>
            Jeff Choi, R. Fernandez, H. Maecker, M. Butte
          </td>
          <td>2016-05-01</td>
          <td>The Journal of Immunology</td>
          <td>2</td>
          <td>77</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21d7405dc4848bad01933fa8e058f89b35b57024" target='_blank'>From Castaways to Discoveries: Unveiling Treasures in Skin RNAseq Using a Novel Multidimensional Data Processing Workflow Including Infection-Host Dynamics.</a></td>
          <td>
            Disha Bhargava, Alec Labadie, Rebecca L Hanson-Rios-Stutz, Austin Goodyke, Ella M Moses, Akansha S Das, Sophie Vanderweele, Janelle V Lemon, Taylor W Cook, David Pearson, Joseph M Redinger, Adam J Caulfield, Rosemary Olivero, Kate Foster, Kurt Ashack, S. Rajasekaran, Caleb P. Bupp, Timothy J. Triche, Connie M Krawczyk, Dave Chesla, Matthew D Sims, Nicholas Hartog, Jeremy W Prokop
          </td>
          <td>2025-03-12</td>
          <td>Physiological genomics</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr id="Hundreds of immune cell types work in coordination to maintain tissue homeostasis. Upon infection, dramatic changes occur with the localization, migration, and proliferation of the immune cells to first alert the body of the danger, confine it to limit spreading, and finally extinguish the threat and bring the tissue back to homeostasis. Since current technologies can follow the dynamics of only a limited number of cell types, we have yet to grasp the full complexity of global in vivo cell dynamics in normal developmental processes and disease. Here, we devise a computational method, digital cell quantification (DCQ), which combines genome‐wide gene expression data with an immune cell compendium to infer in vivo changes in the quantities of 213 immune cell subpopulations. DCQ was applied to study global immune cell dynamics in mice lungs at ten time points during 7 days of flu infection. We find dramatic changes in quantities of 70 immune cell types, including various innate, adaptive, and progenitor immune cells. We focus on the previously unreported dynamics of four immune dendritic cell subtypes and suggest a specific role for CD103+ CD11b− DCs in early stages of disease and CD8+ pDC in late stages of flu infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77a0a3591292c89f8a23e67e05e4e9abcd805fa" target='_blank'>Digital cell quantification identifies global immune cell dynamics during influenza infection</a></td>
          <td>
            Z. Altboum, Yael Steuerman, Eyal David, Z. Barnett-Itzhaki, Liran Valadarsky, Hadas Keren-Shaul, Tal Meningher, E. Mendelson, M. Mandelboim, Irit Gat-Viks, I. Amit
          </td>
          <td>2014-02-01</td>
          <td>Molecular Systems Biology</td>
          <td>105</td>
          <td>102</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a952e012e1800bbd1317b1a754da406bbe7cda" target='_blank'>Systems analysis of human innate immunity in COVID-19</a></td>
          <td>
            Sophie Müller, J. Schultze
          </td>
          <td>2023-05-01</td>
          <td>Seminars in Immunology</td>
          <td>4</td>
          <td>94</td>
        </tr>
    
        <tr id="The evolutionary arms race between host restriction factors and viral antagonists provides crucial insights into immune system evolution and viral adaptation. This study investigates the structural and evolutionary dynamics of the double-domain restriction factors A3F and A3G and their viral inhibitor, Vif, across diverse primate species. By constructing 3D structural homology models and integrating ancestral sequence reconstruction (ASR), we identified patterns of sequence diversity, structural conservation, and functional adaptation. Inactive CD1 (Catalytic Domain 1) domains displayed greater sequence diversity and more positive surface charges than active CD2 domains, aiding nucleotide chain binding and intersegmental transfer. Despite variability, the CD2 DNA-binding grooves remained structurally consistent with conserved residues maintaining critical functions. A3F and A3G diverged in loop 7’ interaction strategies, utilising distinct molecular interactions to facilitate their roles. Vif exhibited charge variation linked to host species, reflecting its coevolution with A3 proteins. These findings illuminate how structural adaptations and charge dynamics enable both restriction factors and their viral antagonists to adapt to selective pressures. Our results emphasize the importance of studying structural evolution in host–virus interactions, with implications for understanding immune defense mechanisms, zoonotic risks, and viral evolution. This work establishes a foundation for further exploration of restriction factor diversity and coevolution across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d3b8f863a1b52ab95b465a0ea4758895d223fee" target='_blank'>Coevolution of Lentiviral Vif with Host A3F and A3G: Insights from Computational Modelling and Ancestral Sequence Reconstruction</a></td>
          <td>
            David Nicolas Giuseppe Huebert, Atefeh Ghorbani, Shaw Yick Brian Lam, M. Larijani
          </td>
          <td>2025-03-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr id="Hidradenitis suppurativa (HS) is a chronic inflammatory skin condition characterized by painful nodules, abscesses, and scarring, predominantly affecting intertriginous regions and it is often underdiagnosed. This study aimed to utilize single cell RNA and cell-surface protein sequencing (CITE-Seq) to delineate the immune composition of circulating cells in Hidradenitis suppurativa (HS) peripheral blood compared to healthy controls. CITE-Seq was used to analyze the gene and protein expression profiles of peripheral blood mononuclear cells (PBMCs) from 9 HS and 29 healthy controls. The study identified significant differences cell composition between HS patients and healthy controls, including increased proportions of CD14+ and CD16+ monocytes, cDC2, plasmablasts, and proliferating CD4+ T cells in HS patients. Differential expression analysis revealed upregulation of inflammatory markers such as TNF, IL1B, and NF-κB in monocytes, as well as chemokines and cell adhesion molecules involved in immune cell recruitment and tissue infiltration. Pathway enrichment analysis highlighted the involvement of IL-17, IL-26 and TNF signaling pathways in HS pathogenesis. Machine learning identified key markers for diagnostics and therapeutic development. The findings also support the potential for machine learning models to aid in the diagnosis of HS based on immune cell markers. These insights may inform future therapeutic strategies targeting specific immune pathways in HS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6380ba17ee3a7e4d4ad44277aa5f9a5dc959cc2" target='_blank'>Cellular indexing of transcriptomes and epitopes (CITE-Seq) in hidradenitis suppurativa identifies dysregulated cell types in peripheral blood and facilitates diagnosis via machine learning</a></td>
          <td>
            Wilson Liao, Sugandh Kumar, Faye Orcales, Bobby Shih, Xiaohui Fang, C. Yin, Ashley Yates, P. Dimitrion, Isaac Neuhaus, Chandler E Johnson, Indra Adrianto, A. Wiala, I. Hamzavi, Li Zhou, Haley Naik, Christian Posch, Q-S Mi
          </td>
          <td>2024-09-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>57</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>