---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-12 09:25:02 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Bispecific T cell engager therapy for refractory rheumatoid arthritis</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Chapter 31 gives an overview of rheumatology, the study of rheumatic conditions which can involve the joints, soft tissues, and bones and also comprises connective tissue disorders, vasculitides, and a number of autoimmune conditions. The chapter outlines patterns of rheumatological disease (e.g. mono-, oligo-, and polyarthropathies), symmetrical versus asymmetrical presentations of disease, and explains how to try and differentiate between inflammatory and non-inflammatory disorders. The chapter provides information on common drug treatments, including dangerous side effects from improperly prescribed methotrexate. Advice is given on history taking and detailed hand examination, with clinical signs of rheumatoid arthritis: Swan neck deformity, Boutonniere’s deformity, Z deformity of thumb, Bouchard’s nodes (proximal interphalangeal joint), Heberden’s nodes (distal interphalangeal joint, first metacarpophalangeal joint), finger ulnar deviation (metacarpophalangeal joint), and wrist radial deviation and subluxation. The most common rheumatological disorders are discussed, including rheumatoid arthritis, spondyloarthropathy, ankylosing spondylitis, psoriatic arthritis, gout, and osteoporosis (including the ‘MESSAGE’ mnemonic for osteoporosis risk factors). The Chapel Hill classification for vasculitides is also covered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0385480d077c026fdc5b1fbf0fc2482d77ffe69d" target='_blank'>Rheumatology</a></td>
          <td>
            Paul Bentley, Ben Lovell
          </td>
          <td>2019-01-23</td>
          <td>Memorizing Medicine</td>
          <td>216</td>
          <td>2</td>
        </tr>
    
        <tr id="Chapter 31 gives an overview of rheumatology, the study of rheumatic conditions which can involve the joints, soft tissues, and bones and also comprises connective tissue disorders, vasculitides, and a number of autoimmune conditions. The chapter outlines patterns of rheumatological disease (e.g. mono-, oligo-, and polyarthropathies), symmetrical versus asymmetrical presentations of disease, and explains how to try and differentiate between inflammatory and non-inflammatory disorders. The chapter provides information on common drug treatments, including dangerous side effects from improperly prescribed methotrexate. Advice is given on history taking and detailed hand examination, with clinical signs of rheumatoid arthritis: Swan neck deformity, Boutonniere’s deformity, Z deformity of thumb, Bouchard’s nodes (proximal interphalangeal joint), Heberden’s nodes (distal interphalangeal joint, first metacarpophalangeal joint), finger ulnar deviation (metacarpophalangeal joint), and wrist radial deviation and subluxation. The most common rheumatological disorders are discussed, including rheumatoid arthritis, spondyloarthropathy, ankylosing spondylitis, psoriatic arthritis, gout, and osteoporosis (including the ‘MESSAGE’ mnemonic for osteoporosis risk factors). The Chapel Hill classification for vasculitides is also covered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0385480d077c026fdc5b1fbf0fc2482d77ffe69d" target='_blank'>Rheumatology</a></td>
          <td>
            Paul Bentley, Ben Lovell
          </td>
          <td>2019-01-23</td>
          <td>Memorizing Medicine</td>
          <td>216</td>
          <td>2</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a53e4f5ccd4166c9cfd1b548ed301889dc9292" target='_blank'>Seeking the ultimate challenge</a></td>
          <td>
            P. Brodin
          </td>
          <td>2019-05-24</td>
          <td>Nature Medicine</td>
          <td>0</td>
          <td>54</td>
        </tr>
    
        <tr id="Ankylosing spondylitis (AS) is an immune-mediated inflammatory disorder that primarily affects the axial skeleton, especially the sacroiliac joints and spine. This results in chronic back pain and, in extreme cases, ankylosis of the spine. Despite its debilitating effects, the pathogenesis of AS remains to be further elucidated. This study used single cell CITE-seq technology to analyze peripheral blood mononuclear cells (PBMCs) in AS and in healthy controls. We identified a number of molecular features associated with AS. CD52 was found to be overexpressed in both RNA and surface protein expression across several cell types in patients with AS. CD16+ monocytes overexpressed TNFSF10 and IL-18Rα in AS, while CD8+ TEM cells and natural killer cells overexpressed genes linked with cytotoxicity, including GZMH, GZMB, and NKG7. Tregs underexpressed CD39 in AS, suggesting reduced functionality. We identified an overrepresented NK cell subset in AS that overexpressed CD16, CD161, and CD38, as well as cytotoxic genes and pathways. Finally, we developed machine learning models derived from CITE-seq data for the classification of AS and achieved an Area Under the Receiver Operating Characteristic (AUROC) curve of > 0.95. In summary, CITE-seq identification of AS-associated genes and surface proteins in specific cell subsets informs our understanding of pathogenesis and potential new therapeutic targets, while providing new approaches for diagnosis via machine learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df7233a9fd7894e3c6a13ed8c284c660104293a" target='_blank'>Single Cell Transcriptome and Surface Epitope Analysis of Ankylosing Spondylitis Facilitates Disease Classification by Machine Learning</a></td>
          <td>
            S. Alber, Sugandh Kumar, Jared Liu, Zhi-ming Huang, D. Páez, Julie J. Hong, H. Chang, T. Bhutani, L. Gensler, W. Liao
          </td>
          <td>2022-05-13</td>
          <td>Frontiers in Immunology</td>
          <td>19</td>
          <td>50</td>
        </tr>
    
        <tr id="Trained Immunity is the nonspecific (pathogen agnostic) memory of innate immune cells, characterized by altered responses upon secondary stimulation. This review provides a RoadMap for the discovery and development of therapeutics targeting Trained Immunity, aimed at researchers with strong scientific backgrounds but limited drug development experience. The article outlines five drug development domains – epigenetic, metabolic, differentiation, inflammatory, and memory changes – that guide the identification of molecular targets, model selection, and biomarker development for the discovery and development of Trained Immunity-based therapeutics. It emphasizes the application of preclinical models and artificial intelligence in target discovery and compound screening. Additionally, the review addresses challenges in translating preclinical Trained Immunity findings to clinical trials and highlights relevant disease indications and ongoing clinical trials. This review integrates scientific findings with development strategy and thereby aims to bridge the gap between discovery and clinical application, advancing the field of Trained Immunity-based therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e32e161fe88188b7ab3aacb49680bec8bd2cf08" target='_blank'>Trained Immunity: RoadMap for drug discovery and development</a></td>
          <td>
            Jelmer H. van Puffelen, Callum Campbell, Irene Gander-Meisterernst, Johanna Holldack, Pauline T Lukey
          </td>
          <td>2025-12-23</td>
          <td>eLife</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr id="COPYRIGHT © 2023 Sun, Zhang, Li, Xie, Luo, Wang, Chen, Wang, Jiang and Yang. This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. TYPE Opinion PUBLISHED 31 January 2023 DOI 10.3389/fimmu.2023.1075419">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55297e1d6a1eb3e5ad28e222e6484ea9889ddb40" target='_blank'>Integration: Gospel for immune bioinformatician on epitope-based therapy</a></td>
          <td>
            Baozeng Sun, Junqi Zhang, Zhikui Li, Mingyang Xie, Cheng Luo, Yongkai Wang, Longyu Chen, Yue-yun Wang, Dongbo Jiang, Kun Yang
          </td>
          <td>2023-01-31</td>
          <td>Frontiers in Immunology</td>
          <td>4</td>
          <td>15</td>
        </tr>
    
        <tr id="Mucosal-associated invariant T (MAIT) cells are a group of unconventional T cells that mainly recognize bacterial vitamin B metabolites presented on MHC-related protein 1 (MR1). MAIT cells have been shown to play an important role in controlling bacterial infection and in responding to viral infections. Furthermore, MAIT cells have been implicated in different chronic inflammatory diseases such as inflammatory bowel disease and multiple sclerosis. Despite their involvement in different physiological and pathological processes, a deeper understanding of MAIT cells is still lacking. Arguably, this can be attributed to the difficulty of quantifying and measuring MAIT cells in different biological samples which is commonly done using flow cytometry-based methods and single-cell-based RNA sequencing techniques. These methods mostly require fresh samples which are difficult to obtain, especially from tissues, have low to medium throughput, and are costly and labor-intensive. To address these limitations, we developed sequence-to-MAIT (Seq2MAIT) which is a transformer-based deep neural network capable of identifying MAIT cells in bulk TCR-sequencing datasets, enabling the quantification of MAIT cells from any biological materials where human DNA is available. Benchmarking Seq2MAIT across different test datasets showed an average area-under-the-receiver-operator-curve (AU[ROC]) >0.80. In conclusion, Seq2MAIT is a novel, economical, and scalable method for identifying and quantifying MAIT cells in virtually any biological sample.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7407ca7605f24e11c1fc4335ba413bb4681f30b9" target='_blank'>Seq2MAIT: A Novel Deep Learning Framework for Identifying Mucosal Associated Invariant T (MAIT) Cells</a></td>
          <td>
            H. ElAbd, Rachel Byron, Steven Woodhouse, Brittney Robinett, Joe Sulc, A. Franke, Mitchell Pesesky, Wenyu Zhou, Haiyin Chen-Harris, Bryan Howie, Ruth Taniguchi, Harlan S. Robins
          </td>
          <td>2024-03-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>
    
        <tr id="Because of their diverse clinical manifestations and intricate pathophysiology, autoimmune diseases which are defined by the immune system wrongly attacking healthy tissues present serious difficulties. Artificial intelligence (AI) has shown revolutionary promise in this field, especially in improving diagnostic precision, facilitating tailored treatment plans, and offering real-time illness tracking. This paper highlights AI's role in assessing various datasets pertaining to immune function and pathology while critically examining the present applications of AI in the therapy of autoimmune diseases. In order to find new biomarkers and enable early and accurate detection of various disorders, advanced AI approaches such as machine learning and deep learning have proven essential. AI-powered predictive models have demonstrated promise in predicting periods of remission and disease flares, allowing for prompt and focused treatment modifications. Furthermore, by accelerating the identification of promising therapeutic candidates and lowering related costs, AI is transforming drug discovery and repurposing. However, issues including data heterogeneity, algorithmic transparency, and patient confidence in AI-driven suggestions limit AI's full potential in the treatment of autoimmune diseases. This paper highlights the need for ethical frameworks and interdisciplinary collaboration by examining these limits and suggesting solutions. This paper shows AI's potential to transform the diagnosis, treatment, and management of autoimmune disorders by combining recent developments and future applications. This will pave the way for a future where healthcare solutions are proactive, accurate, and individualized.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f07f41b711d86b6529aa515d8cfa8d3baf41b6ee" target='_blank'>AI in autoimmune diseases: Transforming diagnosis and treatment</a></td>
          <td>
            D. Mane, Abhijeet Nagnath Deshmukh, Rohit Hanumant Ambare, Akash Ankush Solankar, Chaitanya Sanjaykumar Madane
          </td>
          <td>2025-01-15</td>
          <td>Journal of Pharmaceutical and Biological Sciences</td>
          <td>2</td>
          <td>1</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>