---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-14 09:21:13 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Bispecific T cell engager therapy for refractory rheumatoid arthritis.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a53e4f5ccd4166c9cfd1b548ed301889dc9292" target='_blank'>Seeking the ultimate challenge</a></td>
          <td>
            P. Brodin
          </td>
          <td>2019-05-24</td>
          <td>Nature Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>
    
        <tr id="COPYRIGHT © 2023 Sun, Zhang, Li, Xie, Luo, Wang, Chen, Wang, Jiang and Yang. This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. TYPE Opinion PUBLISHED 31 January 2023 DOI 10.3389/fimmu.2023.1075419">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55297e1d6a1eb3e5ad28e222e6484ea9889ddb40" target='_blank'>Integration: Gospel for immune bioinformatician on epitope-based therapy</a></td>
          <td>
            Baozeng Sun, Junqi Zhang, Zhikui Li, Mingyang Xie, Cheng Luo, Yongkai Wang, Longyu Chen, Yue-yun Wang, Dongbo Jiang, Kun Yang
          </td>
          <td>2023-01-31</td>
          <td>Frontiers in Immunology</td>
          <td>4</td>
          <td>14</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a922ce01d2fadcad1bf7cd4bbf91cb8effadd3d4" target='_blank'>Recent advancement in targeted therapy and role of emerging technologies to treat cancer</a></td>
          <td>
            Shrikant Barot, Henis J. Patel, Anjali Yadav, Igor Ban
          </td>
          <td>2023-10-07</td>
          <td>Medical Oncology</td>
          <td>7</td>
          <td>4</td>
        </tr>
    
        <tr id="The pathogenesis of COVID-19 is complicated by immune dysfunction. The impact of immune-based therapy in COVID-19 patients has been well documented, with some notable studies on the use of anti-cytokine medicines. However, the complexity of disease phenotypes, patient heterogeneity and the varying quality of evidence from immunotherapy studies provide problems in clinical decision-making. This review seeks to aid therapeutic decision-making by giving an overview of the immunological responses against COVID-19 disease that may contribute to the severity of the disease. We have extensively discussed theranostic methods for COVID-19 detection. With advancements in technology, bioinformatics has taken studies to a higher level. The paper also discusses the application of bioinformatics and machine learning tools for the diagnosis, vaccine design and drug repurposing against SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd4caabbb311a44234678a44df1c14dec251114" target='_blank'>Probing the Immune System Dynamics of the COVID-19 Disease for Vaccine Designing and Drug Repurposing Using Bioinformatics Tools</a></td>
          <td>
            Deepshikha Yadav, S. Agarwal, Pranav Pancham, Divya Jindal, V. Agarwal, Premshankar Kumar Dubey, S. Jha, Shalini Mani, Rachana, A. Dey, N. Jha, K. Kesari, Manisha Singh
          </td>
          <td>2022-04-28</td>
          <td>Immuno</td>
          <td>3</td>
          <td>48</td>
        </tr>
    
        <tr id="Since the first introduction of Artificial Intelligence (AI) in 1956, its aim was to tackle problems with human intelligence, but with greater speed and accuracy. AI has been inspired a lot by natu...">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51351e802ae63deeeda3a8c35429fcf87289ea58" target='_blank'>Artificial intelligence and immunotherapy</a></td>
          <td>
            Parnian Jabbari, N. Rezaei
          </td>
          <td>2019-06-03</td>
          <td>Expert Review of Clinical Immunology</td>
          <td>25</td>
          <td>94</td>
        </tr>
    
        <tr id="ABSTRACT Introduction The complex pathophysiology of autoimmune diseases (AIDs) is being progressively deciphered, providing evidence for a multiplicity of pro-inflammatory pathways underlying heterogeneous clinical phenotypes and disease evolution. Areas covered Treatment strategies involving drug combinations are emerging as a preferred option to achieve remission in a vast majority of patients affected by systemic AIDs. The design of appropriate drug combinations can benefit from AID modeling following a comprehensive multi-omics molecular profiling of patients combined with Artificial Intelligence (AI)-powered computational analyses. Such disease models support patient stratification in homogeneous subgroups, shed light on dysregulated pro-inflammatory pathways and yield hypotheses regarding potential therapeutic targets and candidate biomarkers to stratify and monitor patients during treatment. AID models inform the rational design of combination therapies interfering with independent pro-inflammatory pathways related to either one of five prominent immune compartments contributing to the pathophysiology of AIDs, i.e. pro-inflammatory signals originating from tissues, innate immune mechanisms, T lymphocyte activation, autoantibodies and B cell activation, as well as soluble mediators involved in immune cross-talk. Expert opinion The optimal management of AIDs in the future will rely upon rationally designed combination therapies, as a modality of a model-based Computational Precision Medicine taking into account the patients’ biological and clinical specificities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fd9bbcc347751dd764acc51b29fd54e388f6bce" target='_blank'>Model-based computational precision medicine to develop combination therapies for autoimmune diseases</a></td>
          <td>
            Emiko Desvaux, A. Aussy, Sandra Hubert, F. Keime-Guibert, A. Blesius, Perrine Soret, M. Guedj, J. Pers, L. Laigle, P. Moingeon
          </td>
          <td>2021-11-29</td>
          <td>Expert Review of Clinical Immunology</td>
          <td>10</td>
          <td>54</td>
        </tr>
    
        <tr id="T cell function is defined by both T cell receptors (TCR) and T cell gene expression (GEX). Although single-cell technology enables the simultaneous capture of TCR and GEX information, the lack of a reference atlas and computational tools hinders our ability to uncover the fundamental TCR usage rules and to efficiently characterize disease-associated TCRs (dTCR). Here, through the collection of million-scale single-cell GEX-TCR reference atlas comprising 20 diverse disease conditions, we revealed the intrinsic features of TCR-MHC (Major Histocompatibility Complex) restriction in CD4/CD8 lineages. We observed the higher coherence for TCRα/β chains in memory T cells, and detected widely-existing public TCRα/β pairs across individuals. Building upon the reference atlas, we introduced TCR- DeepInsight, a deep-learning framework featuring a disease specificity scoring system that enables the characterization of dTCR clusters with similar GEX-TCR. Our study provides a valuable tool for researchers to analyze single-cell GEX-TCR data and identify dTCRs comprehensively and robustly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720a30e266a268f6dea6e074254ea8f53e318b66" target='_blank'>Disease associated human TCR characterization by deep-learning framework TCR-DeepInsight</a></td>
          <td>
            Ziwei Xue, Lize Wu, Ruonan Tian, Zuozhu Liu, Yadan Bai, Di Sun, Yixin Guo, Pengwei Chen, Yu Zhao, Bing He, Lie Wang, Jianhua Yao, Linrong Lu, Wanlu Liu
          </td>
          <td>2023-05-25</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>28</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>