---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-09 09:54:49 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Bispecific T cell engager therapy for refractory rheumatoid arthritis</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a53e4f5ccd4166c9cfd1b548ed301889dc9292" target='_blank'>Seeking the ultimate challenge</a></td>
          <td>
            P. Brodin
          </td>
          <td>2019-05-24</td>
          <td>Nature Medicine</td>
          <td>0</td>
          <td>55</td>
        </tr>
    
        <tr id="Trained Immunity is the nonspecific (pathogen agnostic) memory of innate immune cells, characterized by altered responses upon secondary stimulation. This review provides a RoadMap for the discovery and development of therapeutics targeting Trained Immunity, aimed at researchers with strong scientific backgrounds but limited drug development experience. The article outlines five drug development domains – epigenetic, metabolic, differentiation, inflammatory, and memory changes – that guide the identification of molecular targets, model selection, and biomarker development for the discovery and development of Trained Immunity-based therapeutics. It emphasizes the application of preclinical models and artificial intelligence in target discovery and compound screening. Additionally, the review addresses challenges in translating preclinical Trained Immunity findings to clinical trials and highlights relevant disease indications and ongoing clinical trials. This review integrates scientific findings with development strategy and thereby aims to bridge the gap between discovery and clinical application, advancing the field of Trained Immunity-based therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e32e161fe88188b7ab3aacb49680bec8bd2cf08" target='_blank'>Trained Immunity: RoadMap for drug discovery and development</a></td>
          <td>
            Jelmer H. van Puffelen, Callum Campbell, Irene Gander-Meisterernst, Johanna Holldack, Pauline T Lukey
          </td>
          <td>2025-12-23</td>
          <td>eLife</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr id="The pathogenesis of COVID-19 is complicated by immune dysfunction. The impact of immune-based therapy in COVID-19 patients has been well documented, with some notable studies on the use of anti-cytokine medicines. However, the complexity of disease phenotypes, patient heterogeneity and the varying quality of evidence from immunotherapy studies provide problems in clinical decision-making. This review seeks to aid therapeutic decision-making by giving an overview of the immunological responses against COVID-19 disease that may contribute to the severity of the disease. We have extensively discussed theranostic methods for COVID-19 detection. With advancements in technology, bioinformatics has taken studies to a higher level. The paper also discusses the application of bioinformatics and machine learning tools for the diagnosis, vaccine design and drug repurposing against SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd4caabbb311a44234678a44df1c14dec251114" target='_blank'>Probing the Immune System Dynamics of the COVID-19 Disease for Vaccine Designing and Drug Repurposing Using Bioinformatics Tools</a></td>
          <td>
            Deepshikha Yadav, S. Agarwal, Pranav Pancham, Divya Jindal, V. Agarwal, Premshankar Kumar Dubey, S. Jha, Shalini Mani, Rachana, A. Dey, N. Jha, K. Kesari, Manisha Singh
          </td>
          <td>2022-04-28</td>
          <td>Immuno</td>
          <td>5</td>
          <td>54</td>
        </tr>
    
        <tr id="The innate immune response is vital for the success of prophylactic vaccines and immunotherapies. Control of signaling in innate immune pathways can improve prophylactic vaccines by inhibiting unfavorable systemic inflammation and immunotherapies by enhancing immune stimulation. In this work, we developed a machine learning-enabled active learning pipeline to guide in vitro experimental screening and discovery of small molecule immunomodulators that improve immune responses by altering the signaling activity of innate immune responses stimulated by traditional pattern recognition receptor agonists. Molecules were tested by in vitro high throughput screening (HTS) where we measured modulation of the nuclear factor κ-light-chain-enhancer of activated B-cells (NF-κB) and the interferon regulatory factors (IRF) pathways. These data were used to train data-driven predictive models linking molecular structure to modulation of the NF-κB and IRF responses using deep representational learning, Gaussian process regression, and Bayesian optimization. By interleaving successive rounds of model training and in vitro HTS, we performed an active learning-guided traversal of a 139,998 molecule library. After sampling only ∼2% of the library, we discovered viable molecules with unprecedented immunomodulatory capacity, including those capable of suppressing NF-κB activity by up to 15-fold, elevating NF-κB activity by up to 5-fold, and elevating IRF activity by up to 6-fold. We extracted chemical design rules identifying particular chemical fragments as principal drivers of specific immunomodulation behaviors. We validated the immunomodulatory effect of a subset of our top candidates by measuring cytokine release profiles. Of these, one molecule induced a 3-fold enhancement in IFN-β production when delivered with a cyclic di-nucleotide stimulator of interferon genes (STING) agonist. In sum, our machine learning-enabled screening approach presents an efficient immunomodulator discovery pipeline that has furnished a library of novel small molecules with a strong capacity to enhance or suppress innate immune signaling pathways to shape and improve prophylactic vaccination and immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/856fac1fabe015a9be15d183af164343b060e203" target='_blank'>Data-driven discovery of innate immunomodulators via machine learning-guided high throughput screening</a></td>
          <td>
            Yifeng Tang, Jeremiah Y Kim, C. Ip, A. Bahmani, Qing Chen, M. Rosenberger, Aaron P. Esser‐Kahn, Andrew L. Ferguson
          </td>
          <td>2023-06-28</td>
          <td>Chemical Science</td>
          <td>1</td>
          <td>31</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb300eaef4f93e109a72eac66a78161de807d50" target='_blank'>Can autoimmune diseases be cured? Scientists see hope at last</a></td>
          <td>
            Cassandra Willyard
          </td>
          <td>2024-01-01</td>
          <td>Nature</td>
          <td>9</td>
          <td>20</td>
        </tr>
    
        <tr id="During the COVID‐19 pandemic, methylcobalamin (MCB), an active form of Vitamin B12 (VB12), showed therapeutic potential in mitigating the cytokine storm associated with SARS‐CoV‐2 infection. While MCB's significant anti‐inflammatory properties are confirmed, it is also observed that its treatment may impair macrophage‐mediated innate immune responses. Comprehensive RNA‐seq, ATAC‐seq, and CUT&Tag analyses revealed that MCB reduces inflammation and weakens innate immunity by limiting chromatin accessibility at NF‐κB and EGR1 binding sites, leading to decreased IFNB1 production and enhanced viral immune evasion. To address this challenge, a deep learning model, VDLIN (Vitamin B12‐derived Deep Learning for Innate Immunity), is developed to identify compounds capable of both suppressing inflammation and boosting innate immunity. As anticipated, VDLIN identified a novel compound, “Co7,” which retains MCB's strong anti‐inflammatory effects while also enhancing immune activation via the TLR4 signaling pathway. Co7 thus emerges as a promising therapeutic candidate, offering advantages over MCB by balancing anti‐inflammatory and immune‐stimulatory functions. Taken together, this study sheds light on the intricate interplay between chromatin dynamics and immune regulation, presenting new opportunities for therapeutic interventions in inflammatory diseases and SARS‐CoV‐2 infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54c2da8de104ff7e1d3c8e32703e5f9348febbe0" target='_blank'>VDLIN: A Deep Learning‐Based Platform for Methylcobalamin‐Inspired Immunomodulatory Compound Screening</a></td>
          <td>
            Xuefei Guo, Yang Zhao, Xianle Rong, Chenxi Niu, Shiyu Hu, Qiang Liu, Fuping You
          </td>
          <td>2025-10-27</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr id="The germinal center response or reaction (GCR) is a hallmark event of adaptive humoral immunity. Unfolding in the B cell follicles of the secondary lymphoid organs, a GC culminates in the production of high-affinity antibody-secreting plasma cells along with memory B cells. By interacting with follicular dendritic cells (FDC) and T follicular helper (Tfh) cells, GC B cells exhibit complex spatiotemporal dynamics. Driving the B cell dynamics are the intracellular signal transduction and gene regulatory network that responds to cell surface signaling molecules, cytokines, and chemokines. As our knowledge of the GC continues to expand in depth and in scope, mathematical modeling has become an important tool to help disentangle the intricacy of the GCR and inform novel mechanistic and clinical insights. While the GC has been modeled at different granularities, a multiscale spatial simulation framework – integrating molecular, cellular, and tissue-level responses – is still rare. Here, we report our recent progress toward this end with a hybrid stochastic GC framework developed on the Cellular Potts Model-based CompuCell3D platform. Tellurium is used to simulate the B cell intracellular molecular network comprising NF-κB, FOXO1, MYC, AP4, CXCR4, and BLIMP1 that responds to B cell receptor (BCR) and CD40-mediated signaling. The molecular outputs of the network drive the spatiotemporal behaviors of B cells, including cyclic migration between the dark zone (DZ) and light zone (LZ) via chemotaxis; clonal proliferative bursts, somatic hypermutation, and DNA damage-induced apoptosis in the DZ; and positive selection, apoptosis via a death timer, and emergence of plasma cells in the LZ. Our simulations are able to recapitulate key molecular, cellular, and morphological GC events, including B cell population growth, affinity maturation, and clonal dominance. This novel modeling framework provides an open-source, customizable, and multiscale virtual GC simulation platform that enables qualitative and quantitative in silico investigations of a range of mechanistic and applied research questions on the adaptive humoral immune response in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58adf6fd74277d15e51d7edabf9ddc1d145ba2db" target='_blank'>A multiscale spatial modeling framework for the germinal center response</a></td>
          <td>
            Derek P. Mu, Christopher D. Scharer, Norbert E. Kaminski, Qiang Zhang
          </td>
          <td>2024-05-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>
    
        <tr id="In the last ten years, many advances have been made in the treatment and diagnosis of immune-mediated diseases [...].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b54a56078412c3f8ce10026ceaef38f6f42908b" target='_blank'>Personalized Medicine and Machine Learning: A Roadmap for the Future</a></td>
          <td>
            M. Sebastiani, C. Vacchi, A. Manfredi, G. Cassone
          </td>
          <td>2022-07-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>24</td>
          <td>46</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>