---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-02 09:38:08 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Bispecific T cell engager therapy for refractory rheumatoid arthritis</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Bispecific T-cell engaging therapies harness the immune system to elicit an effective anticancer response. Modulating the immune activation avoiding potential adverse effects such as cytokine release syndrome (CRS) is a critical aspect to realizing the full potential of this therapy. The use of suitable exogenous intervention strategies to mitigate the CRS risk without compromising the antitumoral capability of bispecific antibody treatment is crucial. To this end, computational approaches can be instrumental to systematically exploring the effects of combining bispecific antibodies with CRS intervention strategies. Here, we employ a logical model to describe the action of bispecific antibodies and the complex interplay of various immune system components and use it to perform simulation experiments to improve the understanding of the factors affecting CRS. We performed a sensitivity analysis to identify the comedications that could ameliorate CRS without impairing tumor clearance. Our results agree with publicly available experimental data suggesting anti-TNF and anti-IL6 as possible co-treatments. Furthermore, we suggest anti-IFNγ as a suitable candidate for clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b15e89fe9ee38f8cee9b8688b18c9a82745947" target='_blank'>Computational Analysis of Cytokine Release Following Bispecific T-Cell Engager Therapy: Applications of a Logic-Based Model</a></td>
          <td>
            G. Selvaggio, S. Parolo, Pranami Bora, L. Leonardelli, J. Harrold, Khamir Mehta, D. Rock, L. Marchetti
          </td>
          <td>2022-03-08</td>
          <td>Frontiers in Oncology</td>
          <td>10</td>
          <td>32</td>
        </tr>
    
        <tr id="The innate immune response is vital for the success of prophylactic vaccines and immunotherapies. Control of signaling in innate immune pathways can improve prophylactic vaccines by inhibiting unfavorable systemic inflammation and immunotherapies by enhancing immune stimulation. In this work, we developed a machine learning-enabled active learning pipeline to guide in vitro experimental screening and discovery of small molecule immunomodulators that improve immune responses by altering the signaling activity of innate immune responses stimulated by traditional pattern recognition receptor agonists. Molecules were tested by in vitro high throughput screening (HTS) where we measured modulation of the nuclear factor κ-light-chain-enhancer of activated B-cells (NF-κB) and the interferon regulatory factors (IRF) pathways. These data were used to train data-driven predictive models linking molecular structure to modulation of the NF-κB and IRF responses using deep representational learning, Gaussian process regression, and Bayesian optimization. By interleaving successive rounds of model training and in vitro HTS, we performed an active learning-guided traversal of a 139,998 molecule library. After sampling only ∼2% of the library, we discovered viable molecules with unprecedented immunomodulatory capacity, including those capable of suppressing NF-κB activity by up to 15-fold, elevating NF-κB activity by up to 5-fold, and elevating IRF activity by up to 6-fold. We extracted chemical design rules identifying particular chemical fragments as principal drivers of specific immunomodulation behaviors. We validated the immunomodulatory effect of a subset of our top candidates by measuring cytokine release profiles. Of these, one molecule induced a 3-fold enhancement in IFN-β production when delivered with a cyclic di-nucleotide stimulator of interferon genes (STING) agonist. In sum, our machine learning-enabled screening approach presents an efficient immunomodulator discovery pipeline that has furnished a library of novel small molecules with a strong capacity to enhance or suppress innate immune signaling pathways to shape and improve prophylactic vaccination and immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/856fac1fabe015a9be15d183af164343b060e203" target='_blank'>Data-driven discovery of innate immunomodulators via machine learning-guided high throughput screening</a></td>
          <td>
            Yifeng Tang, Jeremiah Y Kim, C. Ip, A. Bahmani, Qing Chen, M. Rosenberger, Aaron P. Esser‐Kahn, Andrew L. Ferguson
          </td>
          <td>2023-06-28</td>
          <td>Chemical Science</td>
          <td>1</td>
          <td>31</td>
        </tr>
    
        <tr id="Rheumatoid Arthritis (RA) is an inflammatory autoimmune disease that affects 23 million people worldwide. It is a clinically heterogeneous disorder characterized by the attack of inflammatory chemicals on the synovial tissue that lines joints. It is advantageous to develop effective, targeted treatments and identify specific diagnostic biomarkers for RA before extensive joint degradation, bone erosion, and cartilage destruction. Current modes of RA treatments have alleviated and notably halted the progression of RA. Despite this, not many patients reach low disease activity status after treatment, and a significant number of patients fail to respond to medication due to drug non-specificity. While the reasons for these rates remain unknown, the cellular and molecular signatures present in the synovial tissue for RA patients likely play a role in the varied treatment response. Thus, a drug that particularly targets specific genes and networks may have a significant effect in halting the progression of RA. This study evaluates and proposes potential drug targets through in silico mathematical modeling of various pathways of interest in RA. To understand how drugs interact with genes, we built a mathematical model with 30 two-gene and three-gene network interactions and analyzed the effect of 92 different perturbations to rate constants. We determined that inhibition of the LCK-CD4, VAV1-CD4, and MLT-ROR pathways could potentially serve as drug targets. We also found that increased activity of the DEC2-IL1β and the NF-kB-interleukin pathway and the decreased activity of the TNF-α-REV-ERB pathway could serve as diagnostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ed8bc30a87977c47c1d9eee552d50185c806f93" target='_blank'>A Computational Modeling Framework to Analyze Synovial-Tissue Based Drug Targets and Diagnostic Biomarkers in Rheumatoid Arthritis</a></td>
          <td>
            P. Latawa, B. Chrisman
          </td>
          <td>2021-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr id="With the advent of effective immunotherapies to battle cancers and diseases, an obstacle in recovery has become the potential side effects, specifically cytokine release syndrome (CRS). As there is little quantitative understanding of risks for developing CRS and the degree of its severity, this work explored a model-based approach to produce a library of in silico patients through sensitivity analysis of cytokine interaction parameters and a Monte Carlo sampling. The objective of producing the in silico patients was to correlate a known grading system of cytokine release syndrome severity and thus design a new formula for grading CRS. Using our CRS grading system as the foundation, this work produced not only a formula which related the in silico patient data to the different grades, but we effectively demonstrated a selective approach to reduce the grade of CRS with sequential cytokine inhibition targets. We achieved the reduction of grades by applying the insight from the sensitivity analysis, beginning with the most sensitive targets. Cytokines IL-1, IL-8, TNF-α, INF-γ, IL-6, IL-2, IL-4, IL-10, and IL-12 were in turn the best targets for inhibition to alleviate CRS. Using this approach, patient cytokine time profiles in real-time can be related to the CRS grading system and if the grade is severe enough, action can be taken earlier during the treatment to prevent potentially life-threatening symptoms. What’s more, the identified inhibition sequence of the 9 cytokines provides guidance for clinical intervention of CRS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbed3634ba13aed3fc2c3d954812638cfe069fa" target='_blank'>A Model-Based Investigation of Cytokine Dynamics in Immunotherapies</a></td>
          <td>
            B. Hopkins, Yiming Pan, M. Tucker, Z. Huang
          </td>
          <td>2018-12-30</td>
          <td>Processes</td>
          <td>5</td>
          <td>18</td>
        </tr>
    
        <tr id="During the COVID-19 pandemic, methylcobalamin (MCB), an active form of Vitamin B12 (VB12), showed therapeutic potential in mitigating the cytokine storm associated with SARS-CoV-2 infection. While MCB's significant anti-inflammatory properties are confirmed, it is also observed that its treatment may impair macrophage-mediated innate immune responses. Comprehensive RNA-seq, ATAC-seq, and CUT&Tag analyses revealed that MCB reduces inflammation and weakens innate immunity by limiting chromatin accessibility at NF-κB and EGR1 binding sites, leading to decreased IFNB1 production and enhanced viral immune evasion. To address this challenge, a deep learning model, VDLIN (Vitamin B12-derived Deep Learning for Innate Immunity), is developed to identify compounds capable of both suppressing inflammation and boosting innate immunity. As anticipated, VDLIN identified a novel compound, "Co7," which retains MCB's strong anti-inflammatory effects while also enhancing immune activation via the TLR4 signaling pathway. Co7 thus emerges as a promising therapeutic candidate, offering advantages over MCB by balancing anti-inflammatory and immune-stimulatory functions. Taken together, this study sheds light on the intricate interplay between chromatin dynamics and immune regulation, presenting new opportunities for therapeutic interventions in inflammatory diseases and SARS-CoV-2 infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54c2da8de104ff7e1d3c8e32703e5f9348febbe0" target='_blank'>VDLIN: A Deep Learning-Based Platform for Methylcobalamin-Inspired Immunomodulatory Compound Screening.</a></td>
          <td>
            Xuefei Guo, Yang Zhao, Xianle Rong, Chenxi Niu, Shiyu Hu, Qiang Liu, Fuping You
          </td>
          <td>2025-10-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr id="Baricitinib is an oral Janus kinase (JAK)1/JAK2 inhibitor approved for the treatment of rheumatoid arthritis (RA) that was independently predicted, using artificial intelligence (AI) algorithms, to be useful for COVID‐19 infection via proposed anti‐cytokine effects and as an inhibitor of host cell viral propagation. We evaluated the in vitro pharmacology of baricitinib across relevant leukocyte subpopulations coupled to its in vivo pharmacokinetics and showed it inhibited signaling of cytokines implicated in COVID‐19 infection. We validated the AI‐predicted biochemical inhibitory effects of baricitinib on human numb‐associated kinase (hNAK) members measuring nanomolar affinities for AAK1, BIKE, and GAK. Inhibition of NAKs led to reduced viral infectivity with baricitinib using human primary liver spheroids. These effects occurred at exposure levels seen clinically. In a case series of patients with bilateral COVID‐19 pneumonia, baricitinib treatment was associated with clinical and radiologic recovery, a rapid decline in SARS‐CoV‐2 viral load, inflammatory markers, and IL‐6 levels. Collectively, these data support further evaluation of the anti‐cytokine and anti‐viral activity of baricitinib and support its assessment in randomized trials in hospitalized COVID‐19 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94ac4264c40d5ec6d031d92065f86b85c33aad3" target='_blank'>Mechanism of baricitinib supports artificial intelligence‐predicted testing in COVID‐19 patients</a></td>
          <td>
            J. Stebbing, V. Krishnan, S. de Bono, S. Ottaviani, G. Casalini, Peter J. Richardson, V. Monteil, V. M. Lauschke, A. Mirazimi, Sonia Youhanna, Yee-Joo Tan, F. Baldanti, A. Sarasini, J. R. Terres, B. Nickoloff, R. Higgs, G. Rocha, Nicole L. Byers, D. Schlichting, A. Nirula, A. Cardoso, M. Corbellino
          </td>
          <td>2020-04-15</td>
          <td>EMBO Molecular Medicine</td>
          <td>248</td>
          <td>96</td>
        </tr>
    
        <tr id="Rheumatoid arthritis (RA) is a systemic autoimmune disease characterized by swelling in at least one joint. Owing to an overactive immune response, extra-articular manifestations are observed in certain cases, with interstitial lung disease (ILD) being the most common. Rheumatoid arthritis-associated interstitial lung disease (RA-ILD) is characterized by chronic inflammation of the interstitial space, which causes fibrosis and the scarring of lung tissue. Controlling inflammation and pulmonary fibrosis in RA-ILD is important because they are associated with high morbidity and mortality. Pirfenidone and nintedanib are specific drugs against idiopathic pulmonary fibrosis and showed efficacy against RA-ILD in several clinical trials. Immunosuppressants and disease-modifying antirheumatic drugs (DMARDs) with anti-fibrotic effects have also been used to treat RA-ILD. Immunosuppressants moderate the overexpression of cytokines and immune cells to reduce pulmonary damage and slow the progression of fibrosis. DMARDs with mild anti-fibrotic effects target specific fibrotic pathways to regulate fibrogenic cellular activity, extracellular matrix homeostasis, and oxidative stress levels. Therefore, specific medications are required to effectively treat RA-ILD. In this review, the commonly used RA-ILD treatments are discussed based on their molecular mechanisms and clinical trial results. In addition, a computational approach is proposed to develop specific drugs for RA-ILD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce16fd14448f759a092623ed1a07f75e03c3b143" target='_blank'>Potential Rheumatoid Arthritis-Associated Interstitial Lung Disease Treatment and Computational Approach for Future Drug Development</a></td>
          <td>
            Eunji Jeong, Hyunseok Hong, Yeon-Ah Lee, Kyoung-Soo Kim
          </td>
          <td>2024-02-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>6</td>
          <td>3</td>
        </tr>
    
        <tr id="Background: Lupus is a heterogeneous, systemic disease that affects millions of patients globally with a high unmet medical need. We present results from our powerful and efficient computational drug discovery platform that identifies hits with first-in-class mechanisms of action that can advance rapidly and successfully through preclinical validation studies. The twoXAR discovery platform uses an artificial-intelligence framework to integrate diverse patient-derived biomedical data sets to build holistic and unbiased models of human disease biology. The utilization of diverse, proprietary algorithms and deep learning principles provides a highly sensitive platform to elucidate complex disease-specific associations between biology and biomedical data that are integrated with a library of existing drug molecules. This enables the identification of novel, high-value drug discovery hits with known pharmacological properties. The twoXAR platform also preserves interpretable data-driven links to disease biology to facilitate efficient validation and optimization studies. Objectives: Apply twoXAR’s computational drug discovery platform for the discovery of first-in-class lupus therapy hits and perform preclinical characterization of selected hits to identify drug discovery lead molecules. Methods: Using clinical SLE patient data, we employed the twoXAR platform to build an in-silico SLE disease model. Nine molecules with novel mechanisms of action (not previously tested as candidate clinical therapies for lupus) were identified as drug discovery hits and then characterized in preclinical efficacy studies using the MRL mouse model of lupus. Results: In preclinical validation studies with the MRL mouse model, 2 compounds were differentiated by significant efficacy and excellent tolerability. TXR-711 and TXR-712 increased renal function, decreased renal inflammation and decreased inflammation compared to vehicle-treated control mice. In particular, TXR-711 and TXR-712 significantly decreased serum blood urea nitrogen (BUN) levels, decreased proteinuria levels, and significantly improved kidney histology readouts such as glomerulonephritis and tubule basophilia. Additionally, TXR-711 and TXR-712 treatment resulted in significantly decreased inguinal lymph node weight. Conclusion: TXR-711 and TXR-712 were identified as SLE drug discovery leads with novel MOAs for further preclinical development. Ongoing studies with TXR-711 and TXR-712 includes pharmacokinetic, pharmacodynamic, and additional MRL mouse efficacy characterization. Disclosure of Interests: Isaac Hakim Employee of: twoXAR, Inc, Sana Mujahid Employee of: twoXAR, Inc., Aaron C. Daugherty Employee of: twoXAR, Inc., Timothy S. Heuer Employee of: twoXAR, Inc">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f2d42b0d468776019487ae1f8cbb77d0a694c1" target='_blank'>AB0377 COMPUTATIONAL DISCOVERY AND PRECLINICAL VALIDATION OF THERAPEUTIC LEADS WITH NOVEL MOAS FOR SYSTEMIC LUPUS ERYTHEMATOSUS (SLE)</a></td>
          <td>
            Isaac Hakim, S. Mujahid, Aaron C. Daugherty, T. Heuer
          </td>
          <td>2020-06-01</td>
          <td>Annals of the Rheumatic Diseases</td>
          <td>0</td>
          <td>14</td>
        </tr>
    
        <tr id="Background Lupus is a heterogeneous, systemic disease that affects millions of patients globally with a high unmet medical need. We present results from our powerful and efficient computational drug discovery platform that identifies hits with first-in-class mechanisms of action that can advance rapidly and successfully through preclinical validation studies. The twoXAR discovery platform uses an artificial-intelligence framework to integrate diverse patient-derived biomedical data sets to build holistic and unbiased models of human disease biology. The utilization of diverse, proprietary algorithms and deep learning principles provides a highly sensitive platform to elucidate complex disease-specific associations between biology and biomedical data that are integrated with a library of existing drug molecules. This enables the identification of novel, high-value drug discovery hits with known pharmacological properties. The twoXAR platform also preserves interpretable data-driven links to disease biology to facilitate efficient validation and optimization studies. Methods Using clinical SLE patient data, we employed the twoXAR platform to build an in-silico SLE disease model. Nine molecules with novel mechanisms of action (not previously tested as candidate clinical therapies for lupus) were identified as drug discovery hits and then characterized in preclinical efficacy studies using the MRL mouse model of lupus. Results In preclinical validation studies with the MRL mouse model, 2 compounds were differentiated by significant efficacy and excellent tolerability. TXR-711 and TXR-712 increased renal function, decreased renal inflammation and decreased inflammation compared to vehicle-treated control mice. In particular, TXR-711 and TXR-712 significantly decreased serum blood urea nitrogen (BUN) levels, decreased proteinuria levels, and significantly improved kidney histology readouts such as glomerulonephritis and tubule basophilia. Additionally, TXR-711 and TXR-712 treatment resulted in significantly decreased inguinal lymph node weight. Conclusions TXR-711 and TXR-712 were identified as SLE drug discovery leads with novel MOAs for further preclinical development. Ongoing studies with TXR-711 and TXR-712 includes pharmacokinetic, pharmacodynamic, and additional MRL mouse efficacy characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253174b0fedf652a359d8c18e394a2ee08cdc07c" target='_blank'>P133 Computational discovery and preclinical validation of therapeutic leads with novel MOAs for systemic lupus erythematosus (SLE)</a></td>
          <td>
            Isaac Hakim, S. Mujahid, Aaron C. Daugherty, T. Heuer
          </td>
          <td>2020-03-01</td>
          <td>Lupus Science & Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>