---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-10 09:16:26 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Bispecific T cell engager therapy for refractory rheumatoid arthritis</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Ankylosing spondylitis (AS) is an immune-mediated inflammatory disorder that primarily affects the axial skeleton, especially the sacroiliac joints and spine. This results in chronic back pain and, in extreme cases, ankylosis of the spine. Despite its debilitating effects, the pathogenesis of AS remains to be further elucidated. This study used single cell CITE-seq technology to analyze peripheral blood mononuclear cells (PBMCs) in AS and in healthy controls. We identified a number of molecular features associated with AS. CD52 was found to be overexpressed in both RNA and surface protein expression across several cell types in patients with AS. CD16+ monocytes overexpressed TNFSF10 and IL-18Rα in AS, while CD8+ TEM cells and natural killer cells overexpressed genes linked with cytotoxicity, including GZMH, GZMB, and NKG7. Tregs underexpressed CD39 in AS, suggesting reduced functionality. We identified an overrepresented NK cell subset in AS that overexpressed CD16, CD161, and CD38, as well as cytotoxic genes and pathways. Finally, we developed machine learning models derived from CITE-seq data for the classification of AS and achieved an Area Under the Receiver Operating Characteristic (AUROC) curve of > 0.95. In summary, CITE-seq identification of AS-associated genes and surface proteins in specific cell subsets informs our understanding of pathogenesis and potential new therapeutic targets, while providing new approaches for diagnosis via machine learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df7233a9fd7894e3c6a13ed8c284c660104293a" target='_blank'>Single Cell Transcriptome and Surface Epitope Analysis of Ankylosing Spondylitis Facilitates Disease Classification by Machine Learning</a></td>
          <td>
            S. Alber, Sugandh Kumar, Jared Liu, Zhi-ming Huang, D. Páez, Julie J. Hong, H. Chang, T. Bhutani, L. Gensler, W. Liao
          </td>
          <td>2022-05-13</td>
          <td>Frontiers in Immunology</td>
          <td>17</td>
          <td>50</td>
        </tr>
    
        <tr id="Because of their diverse clinical manifestations and intricate pathophysiology, autoimmune diseases which are defined by the immune system wrongly attacking healthy tissues present serious difficulties. Artificial intelligence (AI) has shown revolutionary promise in this field, especially in improving diagnostic precision, facilitating tailored treatment plans, and offering real-time illness tracking. This paper highlights AI's role in assessing various datasets pertaining to immune function and pathology while critically examining the present applications of AI in the therapy of autoimmune diseases. In order to find new biomarkers and enable early and accurate detection of various disorders, advanced AI approaches such as machine learning and deep learning have proven essential. AI-powered predictive models have demonstrated promise in predicting periods of remission and disease flares, allowing for prompt and focused treatment modifications. Furthermore, by accelerating the identification of promising therapeutic candidates and lowering related costs, AI is transforming drug discovery and repurposing. However, issues including data heterogeneity, algorithmic transparency, and patient confidence in AI-driven suggestions limit AI's full potential in the treatment of autoimmune diseases. This paper highlights the need for ethical frameworks and interdisciplinary collaboration by examining these limits and suggesting solutions. This paper shows AI's potential to transform the diagnosis, treatment, and management of autoimmune disorders by combining recent developments and future applications. This will pave the way for a future where healthcare solutions are proactive, accurate, and individualized.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f07f41b711d86b6529aa515d8cfa8d3baf41b6ee" target='_blank'>AI in autoimmune diseases: Transforming diagnosis and treatment</a></td>
          <td>
            D. Mane, Abhijeet Nagnath Deshmukh, Rohit Hanumant Ambare, Akash Ankush Solankar, Chaitanya Sanjaykumar Madane
          </td>
          <td>2025-01-15</td>
          <td>Journal of Pharmaceutical and Biological Sciences</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr id="ABSTRACT Introduction The complex pathophysiology of autoimmune diseases (AIDs) is being progressively deciphered, providing evidence for a multiplicity of pro-inflammatory pathways underlying heterogeneous clinical phenotypes and disease evolution. Areas covered Treatment strategies involving drug combinations are emerging as a preferred option to achieve remission in a vast majority of patients affected by systemic AIDs. The design of appropriate drug combinations can benefit from AID modeling following a comprehensive multi-omics molecular profiling of patients combined with Artificial Intelligence (AI)-powered computational analyses. Such disease models support patient stratification in homogeneous subgroups, shed light on dysregulated pro-inflammatory pathways and yield hypotheses regarding potential therapeutic targets and candidate biomarkers to stratify and monitor patients during treatment. AID models inform the rational design of combination therapies interfering with independent pro-inflammatory pathways related to either one of five prominent immune compartments contributing to the pathophysiology of AIDs, i.e. pro-inflammatory signals originating from tissues, innate immune mechanisms, T lymphocyte activation, autoantibodies and B cell activation, as well as soluble mediators involved in immune cross-talk. Expert opinion The optimal management of AIDs in the future will rely upon rationally designed combination therapies, as a modality of a model-based Computational Precision Medicine taking into account the patients’ biological and clinical specificities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fd9bbcc347751dd764acc51b29fd54e388f6bce" target='_blank'>Model-based computational precision medicine to develop combination therapies for autoimmune diseases</a></td>
          <td>
            Emiko Desvaux, A. Aussy, Sandra Hubert, F. Keime-Guibert, A. Blesius, Perrine Soret, M. Guedj, J. Pers, L. Laigle, P. Moingeon
          </td>
          <td>2021-11-29</td>
          <td>Expert Review of Clinical Immunology</td>
          <td>12</td>
          <td>54</td>
        </tr>
    
        <tr id="The ability to characterize immune cells and explore the molecular interactions that govern their functions has never been greater, fueled in recent years by the revolutionary advance of single‐cell analysis platforms. However, precisely how immune cells respond to different stimuli and where differentiation processes and effector functions operate remain incompletely understood. Inferring cellular fate within single‐cell transcriptomic analyses is now omnipresent, despite the assumptions typically required in such analyses. Recently developed experimental models support dynamic analyses of the immune response, providing insights into the temporal changes that occur within cells and the tissues in which such transitions occur. Here we will review these approaches and discuss how these can be combined with single‐cell technologies to develop a deeper understanding of the immune responses that should support the development of better therapeutic options for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df359d3a21a0e0d9ae308793407ba6d01736c657" target='_blank'>Decoding changes in tumor‐infiltrating leukocytes through dynamic experimental models and single‐cell technologies</a></td>
          <td>
            C. Lee, M. Clatworthy, David R Withers
          </td>
          <td>2024-06-10</td>
          <td>Immunology and Cell Biology</td>
          <td>1</td>
          <td>59</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>