---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-19 08:41:31 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Microglia Morphological Response to Mesenchymal Stromal Cell Extracellular Vesicles Demonstrates EV Therapeutic Potential for Modulating Neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Microglia, the resident immune cells of the brain, exhibit complex and diverse phenotypes depending on their physiological context including brain regions and disease states. While single-cell RNA-sequencing has recently resolved this heterogeneity, it does not capture tissue and intercellular mechanisms. Our complementary imaging-based approach enables automatic 3D-morphology characterization and classification of thousands of individual microglia in situ and revealed species- and disease-specific morphological phenotypes in mouse and human brain samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02a4e4801d9cdc43d9ea072c79e5b55d50a8852e" target='_blank'>MIC-MAC: High-throughput analysis of three-dimensional microglia morphology in mammalian brains</a></td>
          <td>
            L. Salamanca, N. Mechawar, Keith K. Murai, Rudi Balling, David S. Bouvier, A. Skupin
          </td>
          <td>2018-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>
    
        <tr id="As rapid responders to their environments, microglia engage in functions that are mirrored by their cellular morphology. Microglia are classically thought to exhibit a ramified morphology under homeostatic conditions which switches to an ameboid form during inflammatory conditions. However, microglia display a wide spectrum of morphologies outside of this dichotomy, including rod-like, ramified, ameboid, and hypertrophic states, which have been observed across brain regions, neurodevelopmental timepoints, and various pathological contexts. We used dimensionality reduction and clustering approaches to consider contributions of multiple morphology measures together to define a spectrum of microglial morphological states. Using ImageJ tools, we first developed a semi-automated approach to characterize 27 morphology features from hundreds to thousands of individual microglial cells in a brain subregion-specific manner. Within this pool of morphology measures, we defined distinct sets of highly correlated features that describe different aspects of morphology, including branch length, branching complexity, territory span, and cell circularity. When considered together, these sets of features drove different morphological clusters. Furthermore, our analysis toolset captured morphological states similarly and robustly when applied to independent datasets and using different immunofluorescent markers for microglia. We have compiled our morphology analysis pipeline into an accessible, easy to use, and fully open-source ImageJ macro and R package that the neuroscience community can expand upon and directly apply to their own analyses. Outcomes from this work will supply the field with new tools to systematically evaluate the heterogeneity of microglia morphological states across various experimental models and research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2dd5433a20a8b0dada8fe60f4e507d656fed823" target='_blank'>Development of a High-Throughput Pipeline to Characterize Microglia Morphological States at a Single-Cell Resolution</a></td>
          <td>
            Jennifer Kim, Paul Pavlidis, A. Ciernia
          </td>
          <td>2023-11-05</td>
          <td>eNeuro</td>
          <td>1</td>
          <td>8</td>
        </tr>
    
        <tr id="Quantification of microglial activation through morphometric analysis has long been a staple of the neuroimmunologist’s toolkit. Microglial morphological phenomics can be conducted through either manual classification or constructing a digital skeleton and extracting morphometric data from it. Multiple open-access and paid software packages are available to generate these skeletons via semi-automated and/or fully automated methods with varying degrees of accuracy. Despite advancements in methods to generate morphometrics (quantitative measures of cellular morphology), there has been limited development of tools to analyze the datasets they generate, in particular those containing parameters from tens of thousands of cells analyzed by fully automated pipelines. In this review, we compare and critique the approaches using cluster analysis and machine learning driven predictive algorithms that have been developed to tackle these large datasets, and propose improvements for these methods. In particular, we highlight the need for a commitment to open science from groups developing these classifiers. Furthermore, we call attention to a need for communication between those with a strong software engineering/computer science background and neuroimmunologists to produce effective analytical tools with simplified operability if we are to see their wide-spread adoption by the glia biology community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231383910b9583a109a7b1e6e771e1976daa806b" target='_blank'>Microglial morphometric analysis: so many options, so little consistency</a></td>
          <td>
            J. Reddaway, P. Richardson, Ryan J. Bevan, Jessica Stoneman, M. Palombo
          </td>
          <td>2023-08-10</td>
          <td>Frontiers in Neuroinformatics</td>
          <td>13</td>
          <td>25</td>
        </tr>
    
        <tr id="Morphological states of human cells are widely imaged and analyzed to diagnose diseases and to discover biological mechanisms. Morphodynamics of cells capture their functions more fully than their morphology. Discovery of morphodynamic states of human cells is challenging, because genetic labeling or manual annotation may not be feasible. We propose a computational framework, DynaMorph, that combines quantitative label-free imaging and deep learning for automated discovery of morphodynamic states. As a case study, we apply DynaMorph to study the morphodynamic states of live primary human microglia, which are mobile immune cells of the brain that exhibit complex functional states. DynaMorph identifies two distinct morphodynamic states of microglia under perturbation by cytokines and glioblastoma supernatant. We find that microglia actively transition between the two states. Moreover, single-cell RNA-sequencing of the perturbed microglia shows that the morphodynamic states correspond to distinct transcriptomic clusters of the cells, revealing how perturbations alter gene expression and phenotype. DynaMorph can broadly enable automated discovery of functional states of cellular systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42d013413b9ba49f1e28c022cebd9cdef0f0920c" target='_blank'>DynaMorph: learning morphodynamic states of human cells with live imaging and sc-RNAseq</a></td>
          <td>
            Zhenqin Wu, Bryant B. Chhun, Galina Schmunk, Chang N. Kim, Li-Hao Yeh, T. Nowakowski, James Zou, Shalin B. Mehta
          </td>
          <td>2020-07-21</td>
          <td>bioRxiv</td>
          <td>9</td>
          <td>39</td>
        </tr>
    
        <tr id="Microglia are a dynamic population of brain-resident immune cells that provide immune surveillance of the tissue environment and can move between different activation states in response to changing stimuli. These activation states, associated with increases in inflammatory, phagocytotic, and pathological functions, are marked by characteristic morphological changes in cellular process length, branching complexity and soma size. Microglial cell states are traditionally viewed along a continuum between resting-with long, delicately branched processes and a small cell body, and activated-in which cells adopt an ameoboid shape with retracted, broadened processes and an enlarged soma [1,2]. This simplistic, polarized view of microglial activation has been challenged due to recent work identifying intermediate and divergent cell states associated with specific functional and transcriptional subtypes [3].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0796154054cbac82828b4b4279479009479b8dbd" target='_blank'>Quantitative Bioimaging of Microglial Response to Brain-Targeted Treatment Using Deep Learning Based Morphometry</a></td>
          <td>
            David Joy, D. Xia, K. Monroe, P. Sanchez, M. Calvert
          </td>
          <td>2022-07-22</td>
          <td>Microscopy and Microanalysis</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr id="Brain is the most complex organ of living organisms, as the celebrated cells in the brain, microglia play an indispensable role in the brain’s immune microenvironment. Microglia have critical roles not only in neural development and homeostasis, but also in neurodegenerative diseases and malignant of the central nervous system. However, little is known about the dynamic characteristics of microglia during development or disease conditions. Recently, the single-cell RNA sequencing technologies have become possible to characterize the heterogeneity of immune system in brain. But it posed computational challenges on integrating and utilizing the massive published datasets to dissect the spatiotemporal characterization of microglia. Here, we present microgliaST (bio-bigdata.hrbmu.edu.cn/MST), a database consisting of single-cell microglia transcriptomes across multiple brain regions and developmental periods. Based on high-quality microglia markers collected from published papers, we annotated and constructed human and mouse transcriptomic profiles of 273,374 microglias, comprising 12 regions, 12 periods and 3 conditions (normal, disease, treatment). In addition, MicrogliaST provides multiple analytical tools to elucidate the landscape of microglia under disorder conditions, conduct personalized difference analysis and spatiotemporal dynamic analysis. More importantly, microgliaST paves an ingenious way to the study of brain environment, and also provides insights into clinical therapy assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d2a8076afdda730098c10379c14a94fdf97631" target='_blank'>MicrogliaST: a web server for microglia spatiotemportal pattern analysis in normal and disordered brains</a></td>
          <td>
            Xiaoling Zhong, Feng Li, G. Tan, Li Yi, Jiaxin Zhao, Wanqi Mi, Yu Zhang, Congxue Hu, Xia Li, Yingqi Xu, Chunlong Zhang
          </td>
          <td>2022-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr id="Microglia are the primary resident innate immune cells of the CNS. They possess branched, motile cell processes that are important for their cellular functions. To study the pathways that control microglial morphology and motility under physiological and disease conditions, it is necessary to quantify microglial morphology and motility precisely and reliably. Several image analysis approaches are available for the quantification of microglial morphology and motility. However, they are either not automated, not freely accessible, and/or limited in the number of morphology and motility parameters that can be assessed. Thus, we have developed MotiQ, an open-source, freely accessible software for automated quantification of microglial motility and morphology. MotiQ allows quantification of a diverse set of cellular motility and morphology parameters, including the parameters that have become the gold standard in the microglia field. We demonstrate that MotiQ can be applied to in vivo, ex vivo, and in vitro data from confocal, epifluorescence, or two-photon microscopy, and we compare its results to other analysis approaches. We suggest MotiQ as a versatile and customizable tool to study microglia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d65f10fc7343b2bb99f0b42041bbd4e84a1692" target='_blank'>MotiQ: an open-source toolbox to quantify the cell motility and morphology of microglia</a></td>
          <td>
            J. N. Hansen, M. Brückner, Marie J Pietrowski, Jan F. Jikeli, M. Plescher, H. Beckert, M. Schnaars, Lorenz Fülle, Katharina Reitmeier, T. Langmann, I. Förster, D. Boche, G. Petzold, A. Halle
          </td>
          <td>2022-06-22</td>
          <td>Molecular Biology of the Cell</td>
          <td>7</td>
          <td>56</td>
        </tr>
    
        <tr id="Microglial cells are the first line of defense within the central nervous system, with morphological characterization being widely used to define their activation status. Most methods to evaluate microglia status are manual, and, therefore, often biased, inaccurate, and time consuming. In fact, the process to collect morphological data starts with the acquisition of photomicrographs from where images of single cells are extracted. Then, the researcher collects the morphological features that characterize each cell. However, a manual data collection process from single cells can take weeks to complete. This work describes an open-source ImageJ plugin, MorphData, which automatizes the data extraction process of morphological features of single microglial cells. The plugin collects, processes, and organizes features associated with cell complexity and ramification. In a computer with limited computing power, MorphData was able to handle 699 single cells in less than 14 minutes. The same process, if performed manually, would take almost 19 working days. Overall, MorphData significantly reduces the time taken to collect morphological data from microglial cells, which can then be used to study, understand, and characterize microglia behavior in the brain of human patients or of animal models of neurological and psychiatric diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019f3b92ec798e7678f727e03dd3d3df8670f015" target='_blank'>MorphData: Automating the data extraction process of morphological features of microglial cells in ImageJ</a></td>
          <td>
            A. Campos, S. Duarte-Silva, A. Ambrósio, P. Maciel, Bruno José Torres Fernandes
          </td>
          <td>2021-08-06</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>48</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24a126c95672e01a36ed821195f669034a3bc49" target='_blank'>CellRemorph: A Toolkit for Transforming, Selecting, and Slicing 3D Cell Structures on the Road to Morphologically Detailed Astrocyte Simulations</a></td>
          <td>
            Laura Keto, T. Manninen
          </td>
          <td>2023-05-03</td>
          <td>Neuroinformatics</td>
          <td>1</td>
          <td>12</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>