---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-16 09:43:46 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Microglia Morphological Response to Mesenchymal Stromal Cell Extracellular Vesicles Demonstrates EV Therapeutic Potential for Modulating Neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Microglia, the resident immune cells of the brain, exhibit complex and diverse phenotypes depending on their physiological context including brain regions and disease states. While single-cell RNA-sequencing has recently resolved this heterogeneity, it does not capture tissue and intercellular mechanisms. Our complementary imaging-based approach enables automatic 3D-morphology characterization and classification of thousands of individual microglia in situ and revealed species- and disease-specific morphological phenotypes in mouse and human brain samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02a4e4801d9cdc43d9ea072c79e5b55d50a8852e" target='_blank'>MIC-MAC: High-throughput analysis of three-dimensional microglia morphology in mammalian brains</a></td>
          <td>
            L. Salamanca, N. Mechawar, Keith K. Murai, Rudi Balling, David S. Bouvier, A. Skupin
          </td>
          <td>2018-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88cd36f22e17dbb5db45dc35a55979c21dbafd6" target='_blank'>EV-origin: Enumerating the tissue-cellular origin of circulating extracellular vesicles using exLR profile</a></td>
          <td>
            Yuchen Li, Xigan He, Qin Li, Hongyan Lai, He-Yun Zhang, Z. Hu, Yan Li, Shenglin Huang
          </td>
          <td>2020-10-14</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>152</td>
          <td>58</td>
        </tr>
    
        <tr id="Visual Abstract As rapid responders to their environments, microglia engage in functions that are mirrored by their cellular morphology. Microglia are classically thought to exhibit a ramified morphology under homeostatic conditions which switches to an ameboid form during inflammatory conditions. However, microglia display a wide spectrum of morphologies outside of this dichotomy, including rod-like, ramified, ameboid, and hypertrophic states, which have been observed across brain regions, neurodevelopmental timepoints, and various pathological contexts. We applied dimensionality reduction and clustering to consider contributions of multiple morphology measures together to define a spectrum of microglial morphological states in a mouse dataset that we used to demonstrate the utility of our toolset. Using ImageJ, we first developed a semiautomated approach to characterize 27 morphology features from hundreds to thousands of individual microglial cells in a brain region-specific manner. Within this pool of features, we defined distinct sets of highly correlated features that describe different aspects of morphology, including branch length, branching complexity, territory span, and circularity. When considered together, these sets of features drove different morphological clusters. Our tools captured morphological states similarly and robustly when applied to independent datasets and using different immunofluorescent markers for microglia. We have compiled our morphology analysis pipeline into an accessible, easy-to-use, and fully open-source ImageJ macro and R package that the neuroscience community can expand upon and directly apply to their own analyses. Outcomes from this work will supply the field with new tools to systematically evaluate the heterogeneity of microglia morphological states across various experimental models and research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2dd5433a20a8b0dada8fe60f4e507d656fed823" target='_blank'>Development of a High-Throughput Pipeline to Characterize Microglia Morphological States at a Single-Cell Resolution</a></td>
          <td>
            Jennifer Kim, P. Pavlidis, A. Ciernia
          </td>
          <td>2023-11-05</td>
          <td>eNeuro</td>
          <td>21</td>
          <td>11</td>
        </tr>
    
        <tr id="Abstract Extracellular vesicle (EV) research increasingly demands for quantitative characterisation at the single vesicle level to address heterogeneity and complexity of EV subpopulations. Emerging, commercialised technologies for single EV analysis based on, for example, imaging flow cytometry or imaging after capture on chips generally require dedicated instrumentation and proprietary software not readily accessible to every lab. This limits their implementation for routine EV characterisation in the rapidly growing EV field. We and others have shown that single vesicles can be detected as light diffraction limited fluorescent spots using standard confocal and widefield fluorescence microscopes. Advancing this simple strategy into a process for routine EV quantitation, we developed ‘EVAnalyzer’, an ImageJ/Fiji (Fiji is just ImageJ) plugin for automated, quantitative single vesicle analysis from imaging data. Using EVAnalyzer, we established a robust protocol for capture, (immuno‐)labelling and fluorescent imaging of EVs. To exemplify the application scope, the process was optimised and systematically tested for (i) quantification of EV subpopulations, (ii) validation of EV labelling reagents, (iii) in situ determination of antibody specificity, sensitivity and species cross‐reactivity for EV markers and (iv) optimisation of genetic EV engineering. Additionally, we show that the process can be applied to synthetic nanoparticles, allowing to determine siRNA encapsulation efficiencies of lipid‐based nanoparticles (LNPs) and protein loading of SiO2 nanoparticles. EVAnalyzer further provides a pipeline for automated quantification of cell uptake at the single cell–single vesicle level, thereby enabling high content EV cell uptake assays and plate‐based screens. Notably, the entire procedure from sample preparation to the final data output is entirely based on standard reagents, materials, laboratory equipment and open access software. In summary, we show that EVAnalyzer enables rigorous characterisation of EVs with generally accessible tools. Since we further provide the plugin as open‐source code, we expect EVAnalyzer to not only be a resource of immediate impact, but an open innovation platform for the EV and nanoparticle research communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee63636048b60cb2e0dc234df0303822d4fb1f68" target='_blank'>EVAnalyzer: High content imaging for rigorous characterisation of single extracellular vesicles using standard laboratory equipment and a new open‐source ImageJ/Fiji plugin</a></td>
          <td>
            Melanie Schürz, Joachim Danmayr, Maria Jaritsch, E. Klinglmayr, Heloisa Melo Benirschke, C. Matea, Patrick Zimmerebner, Jakob Rauter, Martin Wolf, F. G. Gomes, Zdenek Kratochvil, Z. Heger, Andrew Miller, T. Heuser, Vesna Stanojlovic, Jana Kiefer, Tanja Plank, Litty Johnson, M. Himly, C. Blöchl, C. Huber, Martin Hintersteiner, Nicole C Meisner-Kober
          </td>
          <td>2022-11-27</td>
          <td>Journal of Extracellular Vesicles</td>
          <td>26</td>
          <td>47</td>
        </tr>
    
        <tr id="Quantification of microglial activation through morphometric analysis has long been a staple of the neuroimmunologist’s toolkit. Microglial morphological phenomics can be conducted through either manual classification or constructing a digital skeleton and extracting morphometric data from it. Multiple open-access and paid software packages are available to generate these skeletons via semi-automated and/or fully automated methods with varying degrees of accuracy. Despite advancements in methods to generate morphometrics (quantitative measures of cellular morphology), there has been limited development of tools to analyze the datasets they generate, in particular those containing parameters from tens of thousands of cells analyzed by fully automated pipelines. In this review, we compare and critique the approaches using cluster analysis and machine learning driven predictive algorithms that have been developed to tackle these large datasets, and propose improvements for these methods. In particular, we highlight the need for a commitment to open science from groups developing these classifiers. Furthermore, we call attention to a need for communication between those with a strong software engineering/computer science background and neuroimmunologists to produce effective analytical tools with simplified operability if we are to see their wide-spread adoption by the glia biology community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231383910b9583a109a7b1e6e771e1976daa806b" target='_blank'>Microglial morphometric analysis: so many options, so little consistency</a></td>
          <td>
            J. Reddaway, P. Richardson, Ryan J. Bevan, Jessica Stoneman, M. Palombo
          </td>
          <td>2023-08-10</td>
          <td>Frontiers in Neuroinformatics</td>
          <td>62</td>
          <td>26</td>
        </tr>
    
        <tr id="BACKGROUND
A multitude of high-quality imaging modalities exist that provide structural data at unprecedented levels of detail. Tissue ultrastructure greatly influences the rate of transport of proteins and other molecules that contribute to neurodegeneration. However, our ability to model flow and diffusion processes in the brain lags behind the quality of the neuroimaging data. Here, a new framework will be presented for converting neuroimaging data into computational fluid dynamics (CFD) models in a scalable manner suitable for high performance computing. TEM data from the Allen Institute will illustrate its capabilities.


METHODS
Segmented TEM images were downloaded from the MicronsExplorer public storage cache. Segmented image stack volumes (4 × 4 × 4 µm) were converted into surface meshes using the Lewiner Marching Cubes algorithm. Convection and diffusion along the surface of individual cells was modeled. After each timestep mass was allowed to exchange between neighboring cells, implemented on a high performance computing cluster using the MPI protocol on 40 or more computing cores. Multiple volumes were analyzed simultaneously, with updates of the flux between cells in different volumes occurring at each time step. These steps were performed with new custom CFD code written in C and Julia.


RESULTS
Segmented z-stacks of TEM data from the MICrONS Explorer website were downloaded in 4 × 4 × 4 µm blocks, which typically contain ≈500 distinct cells (Fig. 1A&B). The meshing algorithm leads to perfect matching between adjacent cells (Fig. 1C). The convection-diffusion equations are solved, modeling flow/diffusion of solutes between cells (Fig. 1D&E). Equations are solved for all ≈500 distinct cells in parallel, with concentrations at matching faces re-balanced at each time step (Fig. 2A&B). A manager-worker MPI model allows for solution on an arbitrary number of cores (Fig. 2C). The interfaces between cell surfaces across volumes are also perfectly matched, allowing for adjustment of the flux of mass between volumes at each timestep (Fig. 2D).


CONCLUSIONS
A new modeling framework is described for solution of transport equations on high performance computing clusters for any imaging modality that produces segmented z-stacked images.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/442e21cebdf5c35d5db269b1a675a5bbe438eb87" target='_blank'>Basic Science and Pathogenesis.</a></td>
          <td>
            D. Elbert
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="The rising global prevalence of neuroinflammatory diseases poses a formidable public health challenge. Central to these pathologies are microglia, the innate immune sentinels of the central nervous system, whose dysregulation contributes significantly to neurodegeneration. In 2025, the convergence of artificial intelligence (AI) and neurobiology is catalyzing a paradigm shift in the restoration of neural homeostasis. This paper presents a forward-looking synthesis of how AI-driven platforms are enabling high-throughput analysis of multi-omics data to identify actionable microglial targets such as TREM2 and CSF1R for therapeutic modulation. Coupled with CRISPR/Cas9 gene editing, AI orchestrates the reprogramming of microglia toward neuroprotective phenotypes, thereby mitigating neuroinflammation and facilitating cognitive recovery in preclinical models. While promising, challenges remain: addressing microglial heterogeneity, establishing reliable biomarkers, ensuring long-term safety, and overcoming ethical and regulatory barriers. This article advocates for multidisciplinary collaboration, infrastructure reform, and equitable policy frameworks to advance AI-orchestrated microglial therapies from research to clinical application, marking a transformative stride in precision neurology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e888ab29fe1cb5d7023153f0660df5980ea93ba5" target='_blank'>AI-orchestrated microglial sculptors: pioneering neural homeostasis restoration in 2025</a></td>
          <td>
            Muhammad Talha, Maliha Khalid, Aminath Waafira
          </td>
          <td>2025-08-28</td>
          <td>Annals of Medicine and Surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr id="Microglia exhibit diverse morphologies reflecting environmental conditions, maturity, or functional states. Thus, morphological characterization provides important information to understand microglial roles and functions. Most recent morphological analysis relies on classifying cells based on morphological parameters. However, this classification may lack biological relevance, as microglial morphologies represent a continuum rather than distinct, separate groups, and do not correspond to mathematically defined clusters irrelevant of microglial cells function. Instead, we propose a new open-source tool, MorphoCellSorter, which assesses microglial morphology by automatically computing morphological criteria, using principal component analysis and Andrews plots to score cells. MorphoCellSorter properly ranked cells from various microglia datasets in mice and rats of different ages, from in vivo, in vitro, and ex vivo models, that were acquired using diverse imaging techniques. This approach allowed for the discrimination of cell populations in various pathophysiological conditions. Finally, MorphoCellSorter offers a versatile, easy, and ready-to-use method to evaluate microglial morphological diversity that could easily be generalized to standardize practices across laboratories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/784e1c3765df07daa2094c39ec73d2c51cb73ef3" target='_blank'>MorphoCellSorter is an Andrews plot-based sorting approach to rank microglia according to their morphological features</a></td>
          <td>
            Sarah Benkeder, Son-Michel Dinh, Paul Marchal, P. de Gea, Muriel Thoby-Brisson, V. Hubert, I. Hristovska, Gabriel Pitollat, K. Combet, L. Cardoit, Bruno Pillot, Christelle Léon, Marlène Wiart, Serge Marthy, Jérôme Honnorat, Olivier Pascual, Jean-Christophe Comte
          </td>
          <td>2025-05-19</td>
          <td>eLife</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>