---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-22 09:15:32 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Microglia Morphological Response to Mesenchymal Stromal Cell Extracellular Vesicles Demonstrates EV Therapeutic Potential for Modulating Neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Microglia, the resident immune cells of the brain, exhibit complex and diverse phenotypes depending on their physiological context including brain regions and disease states. While single-cell RNA-sequencing has recently resolved this heterogeneity, it does not capture tissue and intercellular mechanisms. Our complementary imaging-based approach enables automatic 3D-morphology characterization and classification of thousands of individual microglia in situ and revealed species- and disease-specific morphological phenotypes in mouse and human brain samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02a4e4801d9cdc43d9ea072c79e5b55d50a8852e" target='_blank'>MIC-MAC: High-throughput analysis of three-dimensional microglia morphology in mammalian brains</a></td>
          <td>
            L. Salamanca, N. Mechawar, Keith K. Murai, Rudi Balling, David S Bouvier, A. Skupin
          </td>
          <td>2018-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>
    
        <tr id="Visual Abstract As rapid responders to their environments, microglia engage in functions that are mirrored by their cellular morphology. Microglia are classically thought to exhibit a ramified morphology under homeostatic conditions which switches to an ameboid form during inflammatory conditions. However, microglia display a wide spectrum of morphologies outside of this dichotomy, including rod-like, ramified, ameboid, and hypertrophic states, which have been observed across brain regions, neurodevelopmental timepoints, and various pathological contexts. We applied dimensionality reduction and clustering to consider contributions of multiple morphology measures together to define a spectrum of microglial morphological states in a mouse dataset that we used to demonstrate the utility of our toolset. Using ImageJ, we first developed a semiautomated approach to characterize 27 morphology features from hundreds to thousands of individual microglial cells in a brain region-specific manner. Within this pool of features, we defined distinct sets of highly correlated features that describe different aspects of morphology, including branch length, branching complexity, territory span, and circularity. When considered together, these sets of features drove different morphological clusters. Our tools captured morphological states similarly and robustly when applied to independent datasets and using different immunofluorescent markers for microglia. We have compiled our morphology analysis pipeline into an accessible, easy-to-use, and fully open-source ImageJ macro and R package that the neuroscience community can expand upon and directly apply to their own analyses. Outcomes from this work will supply the field with new tools to systematically evaluate the heterogeneity of microglia morphological states across various experimental models and research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2dd5433a20a8b0dada8fe60f4e507d656fed823" target='_blank'>Development of a High-Throughput Pipeline to Characterize Microglia Morphological States at a Single-Cell Resolution</a></td>
          <td>
            Jennifer Kim, Paul Pavlidis, A. Ciernia
          </td>
          <td>2023-11-05</td>
          <td>eNeuro</td>
          <td>13</td>
          <td>11</td>
        </tr>
    
        <tr id="Quantification of microglial activation through morphometric analysis has long been a staple of the neuroimmunologistâ€™s toolkit. Microglial morphological phenomics can be conducted through either manual classification or constructing a digital skeleton and extracting morphometric data from it. Multiple open-access and paid software packages are available to generate these skeletons via semi-automated and/or fully automated methods with varying degrees of accuracy. Despite advancements in methods to generate morphometrics (quantitative measures of cellular morphology), there has been limited development of tools to analyze the datasets they generate, in particular those containing parameters from tens of thousands of cells analyzed by fully automated pipelines. In this review, we compare and critique the approaches using cluster analysis and machine learning driven predictive algorithms that have been developed to tackle these large datasets, and propose improvements for these methods. In particular, we highlight the need for a commitment to open science from groups developing these classifiers. Furthermore, we call attention to a need for communication between those with a strong software engineering/computer science background and neuroimmunologists to produce effective analytical tools with simplified operability if we are to see their wide-spread adoption by the glia biology community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231383910b9583a109a7b1e6e771e1976daa806b" target='_blank'>Microglial morphometric analysis: so many options, so little consistency</a></td>
          <td>
            J. Reddaway, P. Richardson, Ryan J. Bevan, Jessica Stoneman, M. Palombo
          </td>
          <td>2023-08-10</td>
          <td>Frontiers in Neuroinformatics</td>
          <td>48</td>
          <td>25</td>
        </tr>
    
        <tr id="The rising global prevalence of neuroinflammatory diseases poses a formidable public health challenge. Central to these pathologies are microglia, the innate immune sentinels of the central nervous system, whose dysregulation contributes significantly to neurodegeneration. In 2025, the convergence of artificial intelligence (AI) and neurobiology is catalyzing a paradigm shift in the restoration of neural homeostasis. This paper presents a forward-looking synthesis of how AI-driven platforms are enabling high-throughput analysis of multi-omics data to identify actionable microglial targets such as TREM2 and CSF1R for therapeutic modulation. Coupled with CRISPR/Cas9 gene editing, AI orchestrates the reprogramming of microglia toward neuroprotective phenotypes, thereby mitigating neuroinflammation and facilitating cognitive recovery in preclinical models. While promising, challenges remain: addressing microglial heterogeneity, establishing reliable biomarkers, ensuring long-term safety, and overcoming ethical and regulatory barriers. This article advocates for multidisciplinary collaboration, infrastructure reform, and equitable policy frameworks to advance AI-orchestrated microglial therapies from research to clinical application, marking a transformative stride in precision neurology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e888ab29fe1cb5d7023153f0660df5980ea93ba5" target='_blank'>AI-orchestrated microglial sculptors: pioneering neural homeostasis restoration in 2025</a></td>
          <td>
            Muhammad Talha, Maliha Khalid, Aminath Waafira
          </td>
          <td>2025-09-01</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="Microglia exhibit diverse morphologies reflecting environmental conditions, maturity, or functional states. Thus, morphological characterization provides important information to understand microglial roles and functions. Most recent morphological analysis relies on classifying cells based on morphological parameters. However, this classification may lack biological relevance, as microglial morphologies represent a continuum rather than distinct, separate groups, and do not correspond to mathematically defined clusters irrelevant of microglial cells function. Instead, we propose a new open-source tool, MorphoCellSorter, which assesses microglial morphology by automatically computing morphological criteria, using principal component analysis and Andrews plots to score cells. MorphoCellSorter properly ranked cells from various microglia datasets in mice and rats of different ages, from in vivo, in vitro, and ex vivo models, that were acquired using diverse imaging techniques. This approach allowed for the discrimination of cell populations in various pathophysiological conditions. Finally, MorphoCellSorter offers a versatile, easy, and ready-to-use method to evaluate microglial morphological diversity that could easily be generalized to standardize practices across laboratories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/784e1c3765df07daa2094c39ec73d2c51cb73ef3" target='_blank'>MorphoCellSorter is an Andrews plot-based sorting approach to rank microglia according to their morphological features</a></td>
          <td>
            Sarah Benkeder, Son-Michel Dinh, Paul Marchal, P. de Gea, Muriel Thoby-Brisson, V. Hubert, I. Hristovska, Gabriel Pitollat, K. Combet, L. Cardoit, Bruno Pillot, Christelle LÃ©on, MarlÃ¨ne Wiart, Serge Marthy, JÃ©rÃ´me Honnorat, Olivier Pascual, Jean-Christophe Comte
          </td>
          <td>2025-05-19</td>
          <td>eLife</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
        <tr id="Morphological states of human cells are widely imaged and analyzed to diagnose diseases and to discover biological mechanisms. Morphodynamics of cells capture their functions more fully than their morphology. Discovery of morphodynamic states of human cells is challenging, because genetic labeling or manual annotation may not be feasible. We propose a computational framework, DynaMorph, that combines quantitative label-free imaging and deep learning for automated discovery of morphodynamic states. As a case study, we apply DynaMorph to study the morphodynamic states of live primary human microglia, which are mobile immune cells of the brain that exhibit complex functional states. DynaMorph identifies two distinct morphodynamic states of microglia under perturbation by cytokines and glioblastoma supernatant. We find that microglia actively transition between the two states. Moreover, single-cell RNA-sequencing of the perturbed microglia shows that the morphodynamic states correspond to distinct transcriptomic clusters of the cells, revealing how perturbations alter gene expression and phenotype. DynaMorph can broadly enable automated discovery of functional states of cellular systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42d013413b9ba49f1e28c022cebd9cdef0f0920c" target='_blank'>DynaMorph: learning morphodynamic states of human cells with live imaging and sc-RNAseq</a></td>
          <td>
            Zhenqin Wu, Bryant B. Chhun, Galina Schmunk, Chang N. Kim, Li-Hao Yeh, T. Nowakowski, James Zou, Shalin B. Mehta
          </td>
          <td>2020-07-21</td>
          <td>bioRxiv</td>
          <td>9</td>
          <td>43</td>
        </tr>
    
        <tr id="Microglia exhibit diverse morphologies reflecting environmental conditions, maturity or functional states. Thus, morphological characterization provides important information to understand microglial roles and functions. Most recent morphological analysis relies on classifying cells based on morphological parameters. However, this classification may lack biological relevance, as microglial morphologies represent a continuum rather than distinct, separate groups, and do not correspond to mathematically defined, clusters irrelevant of microglial cells function. Instead, we propose a new open-source tool, MorphoCellSorter, which assesses microglial morphology by automatically computing morphological criteria, using principal component analysis and Andrews plots to rank cells. MorphoCellSorter properly ranked cells from various microglia datasets in mice and rats of different age, from in vivo, in vitro and ex vivo models, that were acquired using diverse imaging techniques. This approach allowed for the discrimination of cell populations in various pathophysiological conditions. Finally, MorphoCellSorter offers a versatile, easy and ready-to-use method to evaluate microglial morphological diversity that could easily be generalized to standardize practices across laboratories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7753e5528c0d5e5f2ea918bdfa0798eaa15cfff8" target='_blank'>MorphoCellSorter: An Andrews plot-based sorting approach to rank microglia according to their morphological features</a></td>
          <td>
            Sarah Benkeder, Son-Michel Dinh, Paul Marchal, Priscille De Gea, Muriel Thoby-Brisson, V. Hubert, I. Hristovska, Gabriel Pitollat, K. Combet, L. Cardoit, B. Pillot, Christelle LÃ©on, MarlÃ¨ne Wiart, Serge Marty, JÃ©rÃ´me Honnorat, Olivier Pascual, Jean-Christophe Comte
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>16</td>
        </tr>
    
        <tr id="Microglia are a dynamic population of brain-resident immune cells that provide immune surveillance of the tissue environment and can move between different activation states in response to changing stimuli. These activation states, associated with increases in inflammatory, phagocytotic, and pathological functions, are marked by characteristic morphological changes in cellular process length, branching complexity and soma size. Microglial cell states are traditionally viewed along a continuum between resting-with long, delicately branched processes and a small cell body, and activated-in which cells adopt an ameoboid shape with retracted, broadened processes and an enlarged soma [1,2]. This simplistic, polarized view of microglial activation has been challenged due to recent work identifying intermediate and divergent cell states associated with specific functional and transcriptional subtypes [3].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0796154054cbac82828b4b4279479009479b8dbd" target='_blank'>Quantitative Bioimaging of Microglial Response to Brain-Targeted Treatment Using Deep Learning Based Morphometry</a></td>
          <td>
            David Joy, D. Xia, K. Monroe, P. Sanchez, M. Calvert
          </td>
          <td>2022-07-22</td>
          <td>Microscopy and Microanalysis</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr id="Brain is the most complex organ of living organisms, as the celebrated cells in the brain, microglia play an indispensable role in the brainâ€™s immune microenvironment. Microglia have critical roles not only in neural development and homeostasis, but also in neurodegenerative diseases and malignant of the central nervous system. However, little is known about the dynamic characteristics of microglia during development or disease conditions. Recently, the single-cell RNA sequencing technologies have become possible to characterize the heterogeneity of immune system in brain. But it posed computational challenges on integrating and utilizing the massive published datasets to dissect the spatiotemporal characterization of microglia. Here, we present microgliaST (bio-bigdata.hrbmu.edu.cn/MST), a database consisting of single-cell microglia transcriptomes across multiple brain regions and developmental periods. Based on high-quality microglia markers collected from published papers, we annotated and constructed human and mouse transcriptomic profiles of 273,374 microglias, comprising 12 regions, 12 periods and 3 conditions (normal, disease, treatment). In addition, MicrogliaST provides multiple analytical tools to elucidate the landscape of microglia under disorder conditions, conduct personalized difference analysis and spatiotemporal dynamic analysis. More importantly, microgliaST paves an ingenious way to the study of brain environment, and also provides insights into clinical therapy assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d2a8076afdda730098c10379c14a94fdf97631" target='_blank'>MicrogliaST: a web server for microglia spatiotemportal pattern analysis in normal and disordered brains</a></td>
          <td>
            Xiaoling Zhong, Feng Li, G. Tan, Li Yi, Jiaxin Zhao, Wanqi Mi, Yu Zhang, Congxue Hu, Xia Li, Yingqi Xu, Chunlong Zhang
          </td>
          <td>2022-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>