---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-28 09:21:14 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Microglia Morphological Response to Mesenchymal Stromal Cell Extracellular Vesicles Demonstrates EV Therapeutic Potential for Modulating Neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88cd36f22e17dbb5db45dc35a55979c21dbafd6" target='_blank'>EV-origin: Enumerating the tissue-cellular origin of circulating extracellular vesicles using exLR profile</a></td>
          <td>
            Yuchen Li, Xigan He, Qin Li, Hongyan Lai, He-Yun Zhang, Z. Hu, Yan Li, Shenglin Huang
          </td>
          <td>2020-10-14</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>126</td>
          <td>57</td>
        </tr>
    
        <tr id="Abstract Extracellular vesicle (EV) research increasingly demands for quantitative characterisation at the single vesicle level to address heterogeneity and complexity of EV subpopulations. Emerging, commercialised technologies for single EV analysis based on, for example, imaging flow cytometry or imaging after capture on chips generally require dedicated instrumentation and proprietary software not readily accessible to every lab. This limits their implementation for routine EV characterisation in the rapidly growing EV field. We and others have shown that single vesicles can be detected as light diffraction limited fluorescent spots using standard confocal and widefield fluorescence microscopes. Advancing this simple strategy into a process for routine EV quantitation, we developed ‘EVAnalyzer’, an ImageJ/Fiji (Fiji is just ImageJ) plugin for automated, quantitative single vesicle analysis from imaging data. Using EVAnalyzer, we established a robust protocol for capture, (immuno‐)labelling and fluorescent imaging of EVs. To exemplify the application scope, the process was optimised and systematically tested for (i) quantification of EV subpopulations, (ii) validation of EV labelling reagents, (iii) in situ determination of antibody specificity, sensitivity and species cross‐reactivity for EV markers and (iv) optimisation of genetic EV engineering. Additionally, we show that the process can be applied to synthetic nanoparticles, allowing to determine siRNA encapsulation efficiencies of lipid‐based nanoparticles (LNPs) and protein loading of SiO2 nanoparticles. EVAnalyzer further provides a pipeline for automated quantification of cell uptake at the single cell–single vesicle level, thereby enabling high content EV cell uptake assays and plate‐based screens. Notably, the entire procedure from sample preparation to the final data output is entirely based on standard reagents, materials, laboratory equipment and open access software. In summary, we show that EVAnalyzer enables rigorous characterisation of EVs with generally accessible tools. Since we further provide the plugin as open‐source code, we expect EVAnalyzer to not only be a resource of immediate impact, but an open innovation platform for the EV and nanoparticle research communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee63636048b60cb2e0dc234df0303822d4fb1f68" target='_blank'>EVAnalyzer: High content imaging for rigorous characterisation of single extracellular vesicles using standard laboratory equipment and a new open‐source ImageJ/Fiji plugin</a></td>
          <td>
            Melanie Schürz, Joachim Danmayr, Maria Jaritsch, E. Klinglmayr, Heloisa Melo Benirschke, C. Matea, Patrick Zimmerebner, Jakob Rauter, Martin Wolf, F. G. Gomes, Zdenek Kratochvil, Z. Heger, Andrew Miller, T. Heuser, Vesna Stanojlovic, Jana Kiefer, Tanja Plank, Litty Johnson, M. Himly, C. Blöchl, C. Huber, Martin Hintersteiner, Nicole C Meisner-Kober
          </td>
          <td>2022-11-27</td>
          <td>Journal of Extracellular Vesicles</td>
          <td>21</td>
          <td>47</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbe137d600788d9cf616b6312add987351f83ed" target='_blank'>EVpedia: A community web resource for prokaryotic and eukaryotic extracellular vesicles research.</a></td>
          <td>
            Dae-Kyum Kim, Jaewook Lee, Richard J. Simpson, J. Lötvall, Y. Gho
          </td>
          <td>2015-04-01</td>
          <td>Seminars in cell & developmental biology</td>
          <td>113</td>
          <td>91</td>
        </tr>
    
        <tr id="Extracellular vesicles (EVs) are membranous structures released by cells into the extracellular space and are thought to be involved in cell-to-cell communication. While EVs and their cargo are promising biomarker candidates, protein sorting mechanisms of proteins to EVs remain unclear. In this study, we ask if it is possible to determine EV association based on the protein sequence. Additionally, we ask what the most important determinants are for EV association. We answer these questions with explainable AI models, using human proteome data from EV databases to train and validate the model. It is essential to correct the datasets for contaminants introduced by coarse EV isolation workflows and for experimental bias caused by mass spectrometry. In this study, we show that it is indeed possible to predict EV association from the protein sequence: a simple sequence-based model for predicting EV proteins achieved an area under the curve of 0.77±0.01, which increased further to 0.84±0.00 when incorporating curated post-translational modification (PTM) annotations. Feature analysis shows that EV associated proteins are stable, polar, and structured with low isoelectric point compared to non-EV proteins. PTM annotations emerged as the most important features for correct classification; specifically palmitoylation is one of the most prevalent EV sorting mechanisms for unique proteins. Palmitoylation and nitrosylation sites are especially prevalent in EV proteins that are determined by very strict isolation protocols, indicating they could potentially serve as quality control criteria for future studies. This computational study offers an effective sequence-based predictor of EV associated proteins with extensive characterisation of the human EV proteome that can explain for individual proteins which factors contribute to their EV association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1596f82dd4bad7e1fdd49c48a0ba6de8e5defd3" target='_blank'>Proteome encoded determinants of protein sorting into extracellular vesicles</a></td>
          <td>
            Katharina Waury, Dea Gogishvili, R. Nieuwland, M. Chatterjee, C. Teunissen, Sanne Abeln
          </td>
          <td>2023-02-03</td>
          <td>Journal of Extracellular Biology</td>
          <td>9</td>
          <td>81</td>
        </tr>
    
        <tr id="The study of extracellular vesicles (EVs) is a rapidly growing field due to their great potential in many areas of clinical medicine including diagnostics, prognostics, theranostics, and therapeutics. Flow cytometry is currently one of the most popular methods of analyzing EVs due to it being a high‐throughput, multiparametric technique, that is readily available in the majority of research labs. Despite its wide use, few commercial flow cytometers are designed specifically for the detection of EVs. Many flow cytometers used for EV analysis are working at their detection limits and are unable to detect the majority of EVs. Currently, very little standardization exists for EV flow cytometry, which is an issue because flow cytometers vary considerably in the way they collect scattered or fluorescent light from particles being interrogated. This makes published research hard to interpret, compare, and in some cases, impossible to reproduce. Here we demonstrate a method of flow cytometer light scatter standardization, utilizing flow cytometer postacquisition analysis software (FCMPASS). FCMPASS is built upon Mie theory and enables the approximation of flow cytometer geometric parameters either by analyzing beads of known diameter and refractive index or by inputting the collection angle if known. The software is then able to create a scatter‐diameter curve and scatter‐refractive index curve that enables researchers to convert scattering data and instrument sensitivity into standardized units. Furthermore, with the correct controls, light scatter data can be converted to diameter distributions or refractive index distributions. FCMPASS therefore offers a freely available and ergonomic method of standardizing and further extending EV characterization using flow cytometry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b61277edd8b8b6554841d44429f5faf5329d9f2" target='_blank'>FCMPASS Software Aids Extracellular Vesicle Light Scatter Standardization</a></td>
          <td>
            J. Welsh, P. Horák, J. Wilkinson, V. Ford, Jennifer C. Jones, David C. Smith, J. Holloway, N. Englyst
          </td>
          <td>2019-06-28</td>
          <td>Cytometry Part A</td>
          <td>65</td>
          <td>43</td>
        </tr>
    
        <tr id="Renal cell carcinoma (RCC) is the most common type of kidney cancer. Increasingly evidences indicate that extracellular vesicles (EVs) orchestrate multiple processes in tumorigenesis, metastasis, immune evasion, and drug response of RCC. EVs are lipid membrane-bound vesicles in nanometer size and secreted by almost all cell types into the extracellular milieu. A myriad of bioactive molecules such as RNA, DNA, protein, and lipid are able to be delivered via EVs for the intercellular communication. Hence, the abundant content of EVs is appealing reservoir for biomarker identification through computational analysis and experimental validation. EVs with excellent biocompatibility and biodistribution are natural platforms that can be engineered to offer achievable drug delivery strategies for RCC therapies. Moreover, the multifaceted roles of EVs in RCC progression also provide substantial targets and facilitate EVs-based drug discovery, which will be accelerated by using artificial intelligence approaches. In this review, we summarized the vital roles of EVs in occurrence, metastasis, immune evasion, and drug resistance of RCC. Furthermore, we also recapitulated and prospected the EVs-based potential applications in RCC, including biomarker identification, drug vehicle development as well as drug target discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c093e05e941e18bfd2993efd7b964559fb7d420f" target='_blank'>Extracellular Vesicles in Renal Cell Carcinoma: Multifaceted Roles and Potential Applications Identified by Experimental and Computational Methods</a></td>
          <td>
            Zhiyuan Qin, Qingwen Xu, Hai-hong Hu, Lushan Yu, S. Zeng
          </td>
          <td>2020-05-07</td>
          <td>Frontiers in Oncology</td>
          <td>23</td>
          <td>51</td>
        </tr>
    
        <tr id="Recent literature suggests that extracellular vesicles (EVs), secreted from most cells and containing cell-specific cargo of proteins, lipids, and nucleic acids, are major driver of intracellular communication in normal physiology and pathological conditions. The recent evidence on stem/progenitor cell EVs as potential therapeutic modality mimicking their parental cell function is exciting because EVs could possibly be used as a surrogate for the stem cell-based therapy, and this regimen may overcome certain roadblocks identified with the use of stem/progenitor cell themselves. This review provides a comprehensive update on our understanding on the role of EVs in cardiac repair and emphasizes the applications of stem/progenitor cell-derived EVs as therapeutics and discusses the current challenges associated with the EV therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a1c57afc68ec5472c3b9cbbc094ee2a8fe82fe9" target='_blank'>Extracellular Vesicles and the Application of System Biology and Computational Modeling in Cardiac Repair.</a></td>
          <td>
            V. Garikipati, Farnaz Shoja-Taheri, Michael E. Davis, R. Kishore
          </td>
          <td>2018-07-06</td>
          <td>Circulation research</td>
          <td>54</td>
          <td>49</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43bd5a1206e9a197f4b737e6d8a6199687b6ba8e" target='_blank'>EV-TRACK: transparent reporting and centralizing knowledge in extracellular vesicle research</a></td>
          <td>
            J. Van Deun, P. Mestdagh, P. Agostinis, Özden Akay, Sushma Anand, J. Anckaert, Zoraida Andreu Martinez, Tine Baetens, Els Beghein, Laurence Bertier, G. Berx, J. Boere, Stephanie Boukouris, Michel Bremer, Dominik Buschmann, James Brian Byrd, Clara Casert, Lesley Cheng, A. Cmoch, D. Daveloose, E. De Smedt, Ş. Demirsoy, V. Depoorter, Bert Dhondt, Tom A P Driedonks, A. Dudek, Abdou Elsharawy, I. Floris, Andrew D. Foers, Kathrin Gärtner, A. Garg, Edward Geeurickx, J. Gettemans, F. Ghazavi, B. Giebel, T. G. Kormelink, Grace V. Hancock, H. Helsmoortel, A. Hill, Vincent Hyenne, Hina Kalra, David Kim, J. Kowal, S. Kraemer, P. Leidinger, C. Leonelli, Yaxuan Liang, L. Lippens, Shu Liu, Alessandra Lo Cicero, Shaun Martin, S. Mathivanan, P. Mathiyalagan, Tamás Matusek, G. Milani, M. Monguió-Tortajada, Liselot M. Mus, D. Muth, A. Németh, E. N. Nolte-‘t Hoen, L. O’Driscoll, Roberta Palmulli, M. Pfaffl, Bjarke Primdal-Bengtson, E. Romano, Quentin Rousseau, S. Sahoo, Natália G. Sampaio, Monisha Samuel, Benjamin J Scicluna, B. Soen, Anneleen Steels, J. Swinnen, M. Takatalo, S. Thaminy, C. Théry, Joeri Tulkens, Isabel Van Audenhove, S. G. van der Grein, Alan Van Goethem, M. V. van Herwijnen, Guillaume van Niel, N. Van Roy, A. V. van Vliet, N. Vandamme, Suzanne Vanhauwaert, Glenn Vergauwen, F. Verweij, Annelynn Wallaert, M. Wauben, K. Witwer, Marijke I. Zonneveld, O. De Wever, J. Vandesompele, A. Hendrix
          </td>
          <td>2017-02-28</td>
          <td>Nature Methods</td>
          <td>999</td>
          <td>95</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58ea1c020b932e22dea9001bde104a1d3d3cb0da" target='_blank'>Applications of machine learning-assisted extracellular vesicles analysis technology in tumor diagnosis</a></td>
          <td>
            Liang Xu, Jing Li, Wei Gong
          </td>
          <td>2025-06-01</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>