---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-12-30 09:12:36 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Microglia Morphological Response to Mesenchymal Stromal Cell Extracellular Vesicles Demonstrates EV Therapeutic Potential for Modulating Neuroinflammation</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88cd36f22e17dbb5db45dc35a55979c21dbafd6" target='_blank'>EV-origin: Enumerating the tissue-cellular origin of circulating extracellular vesicles using exLR profile</a></td>
          <td>
            Yuchen Li, Xigan He, Qin Li, Hongyan Lai, He-Yun Zhang, Z. Hu, Yan Li, Shenglin Huang
          </td>
          <td>2020-10-14</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>104</td>
          <td>55</td>
        </tr>
    
        <tr id="Abstract Extracellular vesicle (EV) research increasingly demands for quantitative characterisation at the single vesicle level to address heterogeneity and complexity of EV subpopulations. Emerging, commercialised technologies for single EV analysis based on, for example, imaging flow cytometry or imaging after capture on chips generally require dedicated instrumentation and proprietary software not readily accessible to every lab. This limits their implementation for routine EV characterisation in the rapidly growing EV field. We and others have shown that single vesicles can be detected as light diffraction limited fluorescent spots using standard confocal and widefield fluorescence microscopes. Advancing this simple strategy into a process for routine EV quantitation, we developed ‘EVAnalyzer’, an ImageJ/Fiji (Fiji is just ImageJ) plugin for automated, quantitative single vesicle analysis from imaging data. Using EVAnalyzer, we established a robust protocol for capture, (immuno‐)labelling and fluorescent imaging of EVs. To exemplify the application scope, the process was optimised and systematically tested for (i) quantification of EV subpopulations, (ii) validation of EV labelling reagents, (iii) in situ determination of antibody specificity, sensitivity and species cross‐reactivity for EV markers and (iv) optimisation of genetic EV engineering. Additionally, we show that the process can be applied to synthetic nanoparticles, allowing to determine siRNA encapsulation efficiencies of lipid‐based nanoparticles (LNPs) and protein loading of SiO2 nanoparticles. EVAnalyzer further provides a pipeline for automated quantification of cell uptake at the single cell–single vesicle level, thereby enabling high content EV cell uptake assays and plate‐based screens. Notably, the entire procedure from sample preparation to the final data output is entirely based on standard reagents, materials, laboratory equipment and open access software. In summary, we show that EVAnalyzer enables rigorous characterisation of EVs with generally accessible tools. Since we further provide the plugin as open‐source code, we expect EVAnalyzer to not only be a resource of immediate impact, but an open innovation platform for the EV and nanoparticle research communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee63636048b60cb2e0dc234df0303822d4fb1f68" target='_blank'>EVAnalyzer: High content imaging for rigorous characterisation of single extracellular vesicles using standard laboratory equipment and a new open‐source ImageJ/Fiji plugin</a></td>
          <td>
            Melanie Schürz, Joachim Danmayr, Maria Jaritsch, E. Klinglmayr, Heloisa Melo Benirschke, C. Matea, Patrick Zimmerebner, Jakob Rauter, Martin Wolf, F. G. Gomes, Zdenek Kratochvil, Z. Heger, Andrew Miller, T. Heuser, Vesna Stanojlovic, Jana Kiefer, Tanja Plank, Litty Johnson, M. Himly, C. Blöchl, C. Huber, Martin Hintersteiner, Nicole C. Meisner-Kober
          </td>
          <td>2022-11-27</td>
          <td>Journal of Extracellular Vesicles</td>
          <td>12</td>
          <td>46</td>
        </tr>
    
        <tr id="Astrocytes are important regulators of normal brain function in mammals, including roles in synaptic signaling, synapse formation, and neuronal health and survival. Many of these functions are executed via secreted proteins. To analyze the astrocyte secretome, a combination of shotgun proteomics and bioinformatics was employed to analyze conditioned media from primary murine astrocyte cultures. Both two- and one-dimensional LC-MS/MS were used to analyze astrocyte secreted proteins, resulting in the identification of over 420 proteins. To refine our results, the intracellular protein contaminants were removed in silico using a cytoplasmic control. In additional rounds of refinement, putative secreted proteins were subjected to analysis by SignalP, SecretomeP, and gene ontology analysis, yielding a refined list of 187 secreted proteins. In conclusion, the use of shotgun proteomics combined with multiple rounds of data refinement produced a high quality catalog of astrocyte secreted proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/898779e91e7d7e3d2eec1b48b49668626825e8de" target='_blank'>Identification of astrocyte secreted proteins with a combination of shotgun proteomics and bioinformatics.</a></td>
          <td>
            J. Dowell, Jeffrey A. Johnson, Lingjun Li
          </td>
          <td>2009-06-17</td>
          <td>Journal of proteome research</td>
          <td>99</td>
          <td>54</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbe137d600788d9cf616b6312add987351f83ed" target='_blank'>EVpedia: A community web resource for prokaryotic and eukaryotic extracellular vesicles research.</a></td>
          <td>
            Dae-Kyum Kim, Jaewook Lee, Richard J. Simpson, J. Lötvall, Y. Gho
          </td>
          <td>2015-04-01</td>
          <td>Seminars in cell & developmental biology</td>
          <td>109</td>
          <td>89</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a5ce1bd967b96df66a24d509b4067b0a28b43b" target='_blank'>Exploring the potential link between MitoEVs and the immune microenvironment of periodontitis based on machine learning and bioinformatics methods</a></td>
          <td>
            Haoran Yang, Anna Zhao, Yuxiang Chen, Tingting Cheng, Jianzhong Zhou, Ziliang Li
          </td>
          <td>2024-02-02</td>
          <td>BMC Oral Health</td>
          <td>4</td>
          <td>1</td>
        </tr>
    
        <tr id="Extracellular vesicles (EVs) are membranous structures released by cells into the extracellular space and are thought to be involved in cell-to-cell communication. While EVs and their cargo are promising biomarker candidates, protein sorting mechanisms of proteins to EVs remain unclear. In this study, we ask if it is possible to determine EV association based on the protein sequence. Additionally, we ask what the most important determinants are for EV association. We answer these questions with explainable AI models, using human proteome data from EV databases to train and validate the model. It is essential to correct the datasets for contaminants introduced by coarse EV isolation workflows and for experimental bias caused by mass spectrometry. In this study, we show that it is indeed possible to predict EV association from the protein sequence: a simple sequence-based model for predicting EV proteins achieved an area under the curve of 0.77±0.01, which increased further to 0.84±0.00 when incorporating curated post-translational modification (PTM) annotations. Feature analysis shows that EV associated proteins are stable, polar, and structured with low isoelectric point compared to non-EV proteins. PTM annotations emerged as the most important features for correct classification; specifically palmitoylation is one of the most prevalent EV sorting mechanisms for unique proteins. Palmitoylation and nitrosylation sites are especially prevalent in EV proteins that are determined by very strict isolation protocols, indicating they could potentially serve as quality control criteria for future studies. This computational study offers an effective sequence-based predictor of EV associated proteins with extensive characterisation of the human EV proteome that can explain for individual proteins which factors contribute to their EV association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1596f82dd4bad7e1fdd49c48a0ba6de8e5defd3" target='_blank'>Proteome encoded determinants of protein sorting into extracellular vesicles</a></td>
          <td>
            Katharina Waury, Dea Gogishvili, R. Nieuwland, M. Chatterjee, C. Teunissen, Sanne Abeln
          </td>
          <td>2023-02-03</td>
          <td>Journal of Extracellular Biology</td>
          <td>6</td>
          <td>89</td>
        </tr>
    
        <tr id="Exosomes are a class of extracellular vesicles that are secreted by various cell types. Unlike other extracellular vesicles (ectosomes and apoptotic blebs), exosomes are of endocytic origin. The roles of exosomes in vaccine/drug delivery, intercellular communication and as a possible source of disease biomarkers have sparked immense interest in them, resulting in a plethora of studies. Whilst multidimensional datasets are continuously generated, it is difficult to harness the true potential of the data until they are compiled and made accessible to the biomedical researchers. Here, we describe ExoCarta (http://www.exocarta.org), a manually curated database of exosomal proteins, RNA and lipids. Datasets currently present in ExoCarta are integrated from both published and unpublished exosomal studies. Since its launch in 2009, ExoCarta has been accessed by more than 16,000 unique users. In this article, we discuss the utility of ExoCarta for exosomal research and urge biomedical researchers in the field to deposit their datasets directly to ExoCarta.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daaf9dae9fa205c61aeed97bd4eb4e92cfd86e31" target='_blank'>ExoCarta as a resource for exosomal research</a></td>
          <td>
            R. Simpson, Hina Kalra, S. Mathivanan
          </td>
          <td>2012-01-01</td>
          <td>Journal of Extracellular Vesicles</td>
          <td>345</td>
          <td>90</td>
        </tr>
    
        <tr id="The study of extracellular vesicles (EVs) is a rapidly growing field due to their great potential in many areas of clinical medicine including diagnostics, prognostics, theranostics, and therapeutics. Flow cytometry is currently one of the most popular methods of analyzing EVs due to it being a high‐throughput, multiparametric technique, that is readily available in the majority of research labs. Despite its wide use, few commercial flow cytometers are designed specifically for the detection of EVs. Many flow cytometers used for EV analysis are working at their detection limits and are unable to detect the majority of EVs. Currently, very little standardization exists for EV flow cytometry, which is an issue because flow cytometers vary considerably in the way they collect scattered or fluorescent light from particles being interrogated. This makes published research hard to interpret, compare, and in some cases, impossible to reproduce. Here we demonstrate a method of flow cytometer light scatter standardization, utilizing flow cytometer postacquisition analysis software (FCMPASS). FCMPASS is built upon Mie theory and enables the approximation of flow cytometer geometric parameters either by analyzing beads of known diameter and refractive index or by inputting the collection angle if known. The software is then able to create a scatter‐diameter curve and scatter‐refractive index curve that enables researchers to convert scattering data and instrument sensitivity into standardized units. Furthermore, with the correct controls, light scatter data can be converted to diameter distributions or refractive index distributions. FCMPASS therefore offers a freely available and ergonomic method of standardizing and further extending EV characterization using flow cytometry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b61277edd8b8b6554841d44429f5faf5329d9f2" target='_blank'>FCMPASS Software Aids Extracellular Vesicle Light Scatter Standardization</a></td>
          <td>
            J. Welsh, P. Horák, J. Wilkinson, V. Ford, Jennifer C. Jones, David C. Smith, J. Holloway, N. Englyst
          </td>
          <td>2019-06-28</td>
          <td>Cytometry Part A</td>
          <td>59</td>
          <td>43</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7a6a2a6301afa935d9e20ab5fd0cfe8dab900c" target='_blank'>Identification of Disulfidptosis-Related Genes in Ischemic Stroke by Combining Single-Cell Sequencing, Machine Learning Algorithms, and In Vitro Experiments</a></td>
          <td>
            Songyun Zhao, Zhuang Hao, Wei Ji, Chao Cheng, Yuankun Liu
          </td>
          <td>2024-09-15</td>
          <td>Neuromolecular Medicine</td>
          <td>1</td>
          <td>12</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>