---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-12-25 12:43:27 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Rebalancing Viral and Immune Damage versus Tissue Repair Prevents Death from Lethal Influenza Infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdba71c8198592733c2f3c0e94873428e96682cb" target='_blank'>Advancing therapies for viral infections using mechanistic computational models of the dynamic interplay between the virus and host immune response</a></td>
          <td>
            V. Zarnitsyna, J. F. Gianlupi, A. Hagar, T. Sego, J. Glazier
          </td>
          <td>2021-08-24</td>
          <td>Current Opinion in Virology</td>
          <td>7</td>
          <td>51</td>
        </tr>
    
        <tr id="During the course of respiratory infections such as influenza, we do have a very limited view of immunological indicators to objectively and quantitatively evaluate the outcome of a host. Methods for monitoring immunological markers in a host’s lungs are invasive and expensive, and some of them are not feasible to perform. ABSTRACT The tracking of pathogen burden and host responses with minimally invasive methods during respiratory infections is central for monitoring disease development and guiding treatment decisions. Utilizing a standardized murine model of respiratory influenza A virus (IAV) infection, we developed and tested different supervised machine learning models to predict viral burden and immune response markers, i.e., cytokines and leukocytes in the lung, from hematological data. We performed independently in vivo infection experiments to acquire extensive data for training and testing of the models. We show here that lung viral load, neutrophil counts, cytokines (such as gamma interferon [IFN-γ] and interleukin 6 [IL-6]), and other lung infection markers can be predicted from hematological data. Furthermore, feature analysis of the models showed that blood granulocytes and platelets play a crucial role in prediction and are highly involved in the immune response against IAV. The proposed in silico tools pave the path toward improved tracking and monitoring of influenza virus infections and possibly other respiratory infections based on minimally invasively obtained hematological parameters. IMPORTANCE During the course of respiratory infections such as influenza, we do have a very limited view of immunological indicators to objectively and quantitatively evaluate the outcome of a host. Methods for monitoring immunological markers in a host’s lungs are invasive and expensive, and some of them are not feasible to perform. Using machine learning algorithms, we show for the first time that minimally invasively acquired hematological parameters can be used to infer lung viral burden, leukocytes, and cytokines following influenza virus infection in mice. The potential of the framework proposed here consists of a new qualitative vision of the disease processes in the lung compartment as a noninvasive tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0bb8b34a01264ae5b42da612199af9cf166307" target='_blank'>Predicting Influenza A Virus Infection in the Lung from Hematological Data with Machine Learning</a></td>
          <td>
            Suneet Singh Jhutty, J. Boehme, A. Jeron, Julia Volckmar, Kristin Schultz, J. Schreiber, K. Schughart, Kaiquan Zhou, J. Steinheimer, H. Stöcker, S. Stegemann-Koniszewski, D. Bruder, E. Hernández-Vargas
          </td>
          <td>2022-11-08</td>
          <td>mSystems</td>
          <td>4</td>
          <td>41</td>
        </tr>
    
        <tr id="The tracking of pathogen burden and host responses with minimal-invasive methods during respiratory infections is central for monitoring disease development and guiding treatment decisions. Utilizing a standardized murine model of respiratory Influenza A virus (IAV) infection, we developed and tested different supervised machine learning models to predict viral burden and immune response markers, i.e. cytokines and leukocytes in the lung, from hematological data. We performed independently in vivo infection experiments to acquire extensive data for training and testing purposes of the models. We show here that lung viral load, neutrophil counts, cytokines like IFN-γ and IL-6, and other lung infection markers can be predicted from hematological data. Furthermore, feature analysis of the models shows that blood granulocytes and platelets play a crucial role in prediction and are highly involved in the immune response against IAV. The proposed in silico tools pave the path towards improved tracking and monitoring of influenza infections and possibly other respiratory infections based on minimal-invasively obtained hematological parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8be3b968f58313fb32721babc61a76af0f7bbfb8" target='_blank'>Tracking influenza a virus infection in the lung from hematological data with machine learning</a></td>
          <td>
            Suneet Singh Jhutty, J. Boehme, A. Jeron, Julia Volckmar, Kristin Schultz, J. Schreiber, K. Schughart, Kai Zhou, J. Steinheimer, H. Stöcker, S. Stegemann-Koniszewski, D. Bruder, E. Hernández-Vargas
          </td>
          <td>2022-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>
    
        <tr id="Acute lung injury (ALI) is a serious complication in clinical settings. This study aimed to elucidate the immune molecular mechanisms underlying ALI by bioinformatics analysis. Human ALI and six ALI mouse model datasets were collected. Immune cell infiltration between the ALI samples and non-ALI controls was estimated using the ssGSEA algorithm. Least absolute shrinkage and selection operator (LASSO) regression analysis and Wilcoxon test were performed to obtain the significantly different immune cell infiltration types. Immune feature genes were screened by differential analysis and the weighted correlation network analysis (WGCNA) algorithm. Functional enrichment was then performed and candidate hub biomarkers were identified. Finally, the receiver operator characteristic curve (ROC) analysis was used to predict their diagnostic performances. Three significantly different immune cell types (B cells, CD4 T cells, and CD8 T cells) were identified between the ALI samples and controls. A total of 13 immune feature genes were obtained by WGCNA and differential analysis and found to be significantly associated with immune functions and lung diseases. Four hub genes, including CD180, CD4, CD74, and MCL1 were identified using cytoHubba and were shown to have good specificity and sensitivity for the diagnosis of ALI. Correlation analysis suggested that CD4 was positively associated with T cells, whereas MCL1 was negatively correlated with B and T cells. We found that CD180, CD4, CD74, and MCL1 can serve as specific immune biomarkers for ALI. MCL1–B cell, MCL1–T cell, and CD4–T cell axes may be involved in the progression of ALI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18db2bcf6945f18cfc74571fc3a49f9b4bdb334d" target='_blank'>Identification of Immune Infiltration and Effective Immune Biomarkers in Acute Lung Injury by Bioinformatics Analysis</a></td>
          <td>
            Dandan Ling, Xiang Zhang, Jiamin Wu, Qianyun Xu, Zhiyong He, Jun Zhang
          </td>
          <td>2022-01-01</td>
          <td>Cell Transplantation</td>
          <td>4</td>
          <td>13</td>
        </tr>
    
        <tr id="Anti-TNF drugs are frequently associated with serious Adverse Events (AEs), which necessitates an improved understanding of individual factors that determine efficacy and safety of anti-TNF agents. We mined the US FDA’s Adverse Event Reporting System (AERS) for anti-TNF-associated AEs to identify and stratify patient subgroups and drug combinations that exhibit specifically correlated complications. We demonstrate the existence of patient subgroup and anti-TNF agent-specific associations for relative risks of developing known and novel AEs including infections, edema, and organ damage associated processes. Concomitant use of anti-TNFs with corticosteroids significantly increased risk of AEs (p < 0.001) including pulmonary fibrosis and pulmonary edema. Using these tightly correlated phenotypes, we mined mouse phenotype data to identify the molecular basis of these AEs. Multiple pathways and networks that regulate injury response, fluid regulation, and wound healing were implicated suggesting modification of anti-TNF-based therapeutic strategies to minimize corticosteroid-based combinatorial risk of severe AEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/543f0f1202d7709a6b6282f97da37faf186516ad" target='_blank'>Using Systems Biology-based Analysis Approaches to Identify Mechanistically Significant Adverse Drug Reactions: Pulmonary Complications from Combined Use of Anti-TNFα Agents and Corticosteroids</a></td>
          <td>
            M. Sarangdhar, Akash Kushwaha, Jeanine E Dahlquist, A. Jegga, B. Aronow
          </td>
          <td>2013-03-18</td>
          <td>AMIA Summits on Translational Science Proceedings</td>
          <td>1</td>
          <td>92</td>
        </tr>
    
        <tr id="Hundreds of immune cell types work in coordination to maintain tissue homeostasis. Upon infection, dramatic changes occur with the localization, migration, and proliferation of the immune cells to first alert the body of the danger, confine it to limit spreading, and finally extinguish the threat and bring the tissue back to homeostasis. Since current technologies can follow the dynamics of only a limited number of cell types, we have yet to grasp the full complexity of global in vivo cell dynamics in normal developmental processes and disease. Here, we devise a computational method, digital cell quantification (DCQ), which combines genome‐wide gene expression data with an immune cell compendium to infer in vivo changes in the quantities of 213 immune cell subpopulations. DCQ was applied to study global immune cell dynamics in mice lungs at ten time points during 7 days of flu infection. We find dramatic changes in quantities of 70 immune cell types, including various innate, adaptive, and progenitor immune cells. We focus on the previously unreported dynamics of four immune dendritic cell subtypes and suggest a specific role for CD103+ CD11b− DCs in early stages of disease and CD8+ pDC in late stages of flu infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77a0a3591292c89f8a23e67e05e4e9abcd805fa" target='_blank'>Digital cell quantification identifies global immune cell dynamics during influenza infection</a></td>
          <td>
            Z. Altboum, Yael Steuerman, Eyal David, Z. Barnett-Itzhaki, Liran Valadarsky, Hadas Keren-Shaul, Tal Meningher, E. Mendelson, M. Mandelboim, Irit Gat-Viks, I. Amit
          </td>
          <td>2014-02-01</td>
          <td>Molecular Systems Biology</td>
          <td>101</td>
          <td>101</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/651735405eca6cb18c70caf058c6db248cb3d39b" target='_blank'>A narrative review on lung injury: mechanisms, biomarkers, and monitoring</a></td>
          <td>
            Wenping Fan, Biyu Gui, Xiaolei Zhou, Li Li, Huaiyong Chen
          </td>
          <td>2024-10-31</td>
          <td>Critical Care</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr id="Innate immunity is protective against viruses, but also can facilitate pathological infection responses. Despite intensive research, our understanding of the mechanisms that regulate innate immunity in virus infection remains incomplete. Systems biology-based data-driven modeling approaches hold substantial promise toward discovery of crucial innate immune signaling regulators, yet model-derived predictions are almost completely unexplored. Here, we carried out systematic experimental validation of candidate regulators predicted by a transcriptional association network model of influenza virus-infected cells. We identified dozens of novel innate immune signaling regulators with potent effects on the replication of influenza and other viruses, and importantly, we established the biological relevance of a validated regulator in vivo. Collectively, these findings aid in clarifying mechanisms of influenza virus pathogenicity and might lead to innovative approaches for treating influenza virus disease. Similar data-driven modeling strategies may be applicable for the study of other pathogen systems or immunological disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eac33ed63c7f5f9313741e86f6940a95a3afe88" target='_blank'>Host network-based discovery of critical regulators of innate immunity, virus growth, and pathogenesis in influenza virus infection</a></td>
          <td>
            A. Eisfeld, Shufang Fan, Hongyu Rao, Backiyalakshmi Ammayappan Venkatachalam, D. Smith, P. Halfmann, Kevin B. Walters, S. Nair, Larissa B. Thackray, Jacob F. Kocher, A. Sims, H. Mitchell, G. Neumann, V. Blank, K. Waters, R. Baric, M. Diamond, Y. Kawaoka
          </td>
          <td>2022-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>153</td>
        </tr>
    
        <tr id="
 Inflammation is common in virtually all diseases. Since immune cells travel through the blood to inflamed tissues, peripheral blood reflects inflammation at distant sites. However, blood biomarkers are hard to detect in global gene expression data since blood is not the site of perturbation and statistical corrections for multiple testing reduce sensitivity. We hypothesized that combining transcriptional signatures from inflamed tissue and machine learning would be useful to derive blood biomarkers. We first tested this in the mouse model of influenza infection. We identified a set of early genes in the lungs associated with lethality after infection. We then tested their differential abundance early in the blood of mice infected with a range of influenza doses. We applied elastic net to build a biomarker able to predict disease outcome (4 classes: early lethal, late lethal, non-lethal vs. non-infected; accuracy on training and validation sets: 91% and 79%). We next confirmed our hypothesis in lung cancer and breast cancer in humans. We identified sets of genes associated with tumors but not with tumor-free tissues and tested their differential abundance in the blood of patients compared to healthy controls. Using elastic net, we built two blood biomarkers to classify subjects according to disease status (case vs. control; accuracy on training and validation sets in breast cancer: 92% and 67%; in lung cancer: 89% and 78%). In conclusion, combining inflamed tissue signatures and machine learning, we were able to derive blood biomarkers for three different diseases in two different species. We expect that this novel strategy can be applied in other diseases as well.
 This work was supported by the Intramural Research Program of NIAID, NIH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95ae3cefc1c60c582470175b79c18a533b7a4b2" target='_blank'>Combining gene expression data from inflamed tissue and machine learning for blood biomarker discovery</a></td>
          <td>
            Pedro Milanez-Almeida, A. Martins, Manikandan Narayanan, P. Torabi-Parizi, Luis Franco, J. Tsang, R. N. Germain
          </td>
          <td>2017-05-01</td>
          <td>The Journal of Immunology</td>
          <td>1</td>
          <td>36</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>