---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-24 09:14:15 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Rebalancing Viral and Immune Damage versus Tissue Repair Prevents Death from Lethal Influenza Infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdba71c8198592733c2f3c0e94873428e96682cb" target='_blank'>Advancing therapies for viral infections using mechanistic computational models of the dynamic interplay between the virus and host immune response</a></td>
          <td>
            V. Zarnitsyna, J. F. Gianlupi, A. Hagar, T. Sego, J. Glazier
          </td>
          <td>2021-08-24</td>
          <td>Current Opinion in Virology</td>
          <td>8</td>
          <td>51</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e096ebd7370fb9e33e17eeb039e8d87258842d" target='_blank'>Systems Virology</a></td>
          <td>
            M. J. Korth, G. L. Law
          </td>
          <td>2016-02-12</td>
          <td>Viral Pathogenesis</td>
          <td>0</td>
          <td>37</td>
        </tr>
    
        <tr id="Respiratory viruses present major health challenges, as evidenced by the 2009 influenza pandemic and the ongoing severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic. Severe RNA virus respiratory infections often correlate with high viral load and excessive inflammation. Understanding the dynamics of the innate immune response and its manifestation at the cell and tissue levels are vital to understanding the mechanisms of immunopathology and developing improved, strain independent treatments. Here, we present a novel spatialized multicellular spatial computational model of two principal components of tissue infection and response: RNA virus replication and type-I interferon mediated antiviral response to infection within lung epithelial cells. The model is parameterized using data from influenza virus infected cell cultures and, consistent with experimental observations, exhibits either linear radial growth of viral plaques or arrested plaque growth depending on the local concentration of type I interferons. Modulating the phosphorylation of STAT or altering the ratio of the diffusion constants of interferon and virus in the cell culture could lead to plaque growth arrest. The dependence of arrest on diffusion constants highlights the importance of developing validated spatial models of cytokine signaling and the need for in vitro experiments to measure these diffusion constants. Sensitivity analyses were performed under conditions creating both continuous plaque growth and arrested plaque growth. Findings suggest that plaque growth and cytokine assay measurements should be collected during arrested plaque growth, as the model parameters are significantly more sensitive and more likely to be identifiable. The model’s metrics replicate experimental immunostaining imaging and titer based sampling assays. The model is easy to extend to include SARS-CoV-2-specific mechanisms as they are discovered or to include as a component linking epithelial cell signaling to systemic immune models. Author Summary COVID-19 is possibly the defining healthcare crisis of the current generation, with tens of millions of global cases and more than a million reported deaths. Respiratory lung infections form lesions in the lungs, whose number and size correlate with severity of illness. In some severe cases, the disease triggers a severe inflammatory condition known as cytokine storm. Given the complexity of the immune system, computational modeling is needed to link molecular signaling at the site of inflection to the signaling impact on the overall immune system, ultimately revealing how severe inflammatory conditions may emerge. Here, we created a computational model of the early stages of infection that simulates lung cells infected with RNA viruses, such those responsible for COVID-19 and influenza, to help explore how the disease forms viral plaques, an in vitro analog to lesion growth in the lung. Our model recapitulates in vitro observations that pretreatment of biological signaling molecules called with type-I interferons, which are currently being evaluated for treatment of COVID-19. Analyzing the model, we, can stop viral plaque growth. We found that enhancing certain aspects of the innate immune system, such as the JAK/STAT pathway, may be able to stop viral plaque growth, suggesting molecules involved in this pathway as possible drug candidates. Quantifying the parameters needed to model interferon signaling and viral replication, experiments should be performed under conditions that inhibit viral growth, such as pretreating cells with interferon. We present a computational framework that is essential to constructing larger models of respiratory infection induced immune responses, can be used to evaluate drugs and other medical interventions quickly, cheaply, and without the need for animal testing during the initial phase, and that defines experiments needed to improve our fundamental understanding of the mechanisms regulating the immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afbec26a5cdf60962de2faad5464a72152c1c8fc" target='_blank'>Multicellular spatial model of RNA virus replication and interferon responses reveals factors controlling plaque growth dynamics</a></td>
          <td>
            Josua O. Aponte-Serrano, J. Weaver, T. Sego, J. Glazier, J. Shoemaker
          </td>
          <td>2021-03-16</td>
          <td>PLoS Computational Biology</td>
          <td>11</td>
          <td>51</td>
        </tr>
    
        <tr id="Acute lung injury (ALI) is a serious complication in clinical settings. This study aimed to elucidate the immune molecular mechanisms underlying ALI by bioinformatics analysis. Human ALI and six ALI mouse model datasets were collected. Immune cell infiltration between the ALI samples and non-ALI controls was estimated using the ssGSEA algorithm. Least absolute shrinkage and selection operator (LASSO) regression analysis and Wilcoxon test were performed to obtain the significantly different immune cell infiltration types. Immune feature genes were screened by differential analysis and the weighted correlation network analysis (WGCNA) algorithm. Functional enrichment was then performed and candidate hub biomarkers were identified. Finally, the receiver operator characteristic curve (ROC) analysis was used to predict their diagnostic performances. Three significantly different immune cell types (B cells, CD4 T cells, and CD8 T cells) were identified between the ALI samples and controls. A total of 13 immune feature genes were obtained by WGCNA and differential analysis and found to be significantly associated with immune functions and lung diseases. Four hub genes, including CD180, CD4, CD74, and MCL1 were identified using cytoHubba and were shown to have good specificity and sensitivity for the diagnosis of ALI. Correlation analysis suggested that CD4 was positively associated with T cells, whereas MCL1 was negatively correlated with B and T cells. We found that CD180, CD4, CD74, and MCL1 can serve as specific immune biomarkers for ALI. MCL1–B cell, MCL1–T cell, and CD4–T cell axes may be involved in the progression of ALI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18db2bcf6945f18cfc74571fc3a49f9b4bdb334d" target='_blank'>Identification of Immune Infiltration and Effective Immune Biomarkers in Acute Lung Injury by Bioinformatics Analysis</a></td>
          <td>
            Dandan Ling, Xiang Zhang, Jiamin Wu, Qianyun Xu, Zhiyong He, Jun Zhang
          </td>
          <td>2022-01-01</td>
          <td>Cell Transplantation</td>
          <td>6</td>
          <td>13</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d88b718d283e4f7c86f3f8be9dbc1eaa8a3dd6" target='_blank'>Systems approaches to influenza-virus host interactions and the pathogenesis of highly virulent and pandemic viruses</a></td>
          <td>
            M. J. Korth, Nicolas Tchitchek, Arndt Benecke, M. Katze
          </td>
          <td>2012-12-04</td>
          <td>Seminars in Immunology</td>
          <td>6</td>
          <td>100</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/543f0f1202d7709a6b6282f97da37faf186516ad" target='_blank'>Using Systems Biology-based Analysis Approaches to Identify Mechanistically Significant Adverse Drug Reactions: Pulmonary Complications from Combined Use of Anti-TNFα Agents and Corticosteroids</a></td>
          <td>
            M. Sarangdhar, Akash Kushwaha, Jeanine E Dahlquist, A. Jegga, B. Aronow
          </td>
          <td>2013-03-18</td>
          <td>AMIA Summits on Translational Science Proceedings</td>
          <td>1</td>
          <td>92</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/651735405eca6cb18c70caf058c6db248cb3d39b" target='_blank'>A narrative review on lung injury: mechanisms, biomarkers, and monitoring</a></td>
          <td>
            Wenping Fan, Biyu Gui, Xiaolei Zhou, Li Li, Huaiyong Chen
          </td>
          <td>2024-10-31</td>
          <td>Critical Care</td>
          <td>4</td>
          <td>1</td>
        </tr>
    
        <tr id="Respiratory and emergency medicine have undergone significant transformations in recent years, shaped by advancements in critical care, the integration of artificial intelligence (AI), and the evolving understanding of post-infectious sequelae [...].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/921897d8b5158c1932e17d4ed17b3bbbc7a13bd1" target='_blank'>Pioneering Respiratory and Emergency Medicine Research During the COVID-19 Era</a></td>
          <td>
            Ioannis Pantazopoulos, G. Mavrovounis, Konstantinos I. Gourgoulianis
          </td>
          <td>2025-02-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f595e6a15b63b8ddd0d4e333d8db4bd03984ce5" target='_blank'>Computational identification of key biological modules and transcription factors in acute lung injury.</a></td>
          <td>
            S. Gharib, W. Liles, G. Matute-Bello, R. Glenny, T. R. Martin, W. Altemeier
          </td>
          <td>2006-03-15</td>
          <td>American journal of respiratory and critical care medicine</td>
          <td>61</td>
          <td>51</td>
        </tr>
    
        <tr id="Infectious diseases caused by viral pathogens exacerbate health care and economic burdens. Numerous viral biomolecules suppress the human innate immune system, enabling viruses to evade an immune response from the host. ABSTRACT Suppression of the host innate immune response is a critical aspect of viral replication. Upon infection, viruses may introduce one or more proteins that inhibit key immune pathways, such as the type I interferon pathway. However, the ability to predict and evaluate viral protein bioactivity on targeted pathways remains challenging and is typically done on a single-virus or -gene basis. Here, we present a medium-throughput high-content cell-based assay to reveal the immunosuppressive effects of viral proteins. To test the predictive power of our approach, we developed a library of 800 genes encoding known, predicted, and uncharacterized human virus genes. We found that previously known immune suppressors from numerous viral families such as Picornaviridae and Flaviviridae recorded positive responses. These include a number of viral proteases for which we further confirmed that innate immune suppression depends on protease activity. A class of predicted inhibitors encoded by Rhabdoviridae viruses was demonstrated to block nuclear transport, and several previously uncharacterized proteins from uncultivated viruses were shown to inhibit nuclear transport of the transcription factors NF-κB and interferon regulatory factor 3 (IRF3). We propose that this pathway-based assay, together with early sequencing, gene synthesis, and viral infection studies, could partly serve as the basis for rapid in vitro characterization of novel viral proteins. IMPORTANCE Infectious diseases caused by viral pathogens exacerbate health care and economic burdens. Numerous viral biomolecules suppress the human innate immune system, enabling viruses to evade an immune response from the host. Despite our current understanding of viral replications and immune evasion, new viral proteins, including those encoded by uncultivated viruses or emerging viruses, are being unearthed at a rapid pace from large-scale sequencing and surveillance projects. The use of medium- and high-throughput functional assays to characterize immunosuppressive functions of viral proteins can advance our understanding of viral replication and possibly treatment of infections. In this study, we assembled a large viral-gene library from diverse viral families and developed a high-content assay to test for inhibition of innate immunity pathways. Our work expands the tools that can rapidly link sequence and protein function, representing a practical step toward early-stage evaluation of emerging and understudied viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4e601c9825f076ee4215b502373093f39f9e08" target='_blank'>High-Content Screening and Computational Prediction Reveal Viral Genes That Suppress the Innate Immune Response</a></td>
          <td>
            Tai L. Ng, Erika J. Olson, Tae Yeon Yoo, H. Weiss, Yukiye Koide, P. Koch, Nathan J. Rollins, Pia Mach, Tobias Meisinger, Trenton Bricken, Timothy Z. Chang, Colin Molloy, Jérôme Zürcher, Roger L. Chang, T. Mitchison, J. Glass, D. Marks, J. Way, P. Silver
          </td>
          <td>2022-03-23</td>
          <td>mSystems</td>
          <td>4</td>
          <td>138</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>