---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-26 09:39:54 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Rebalancing Viral and Immune Damage versus Tissue Repair Prevents Death from Lethal Influenza Infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, Benno Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>428</td>
          <td>102</td>
        </tr>
    
        <tr id="[This corrects the article DOI: 10.1155/2022/8660752.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06dae337126e2677af35c5fd5ba2bc5d41eea9da" target='_blank'>Corrigendum to “HD-13 Induces Swine Pneumonia Progression via Activation of TLR9”</a></td>
          <td>
            Jing Liu, Jiayi Xu, Honglei Zhou, Lihua Wang
          </td>
          <td>2022-11-16</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>2</td>
          <td>5</td>
        </tr>
    
        <tr id="Single-cell RNA sequencing of the bronchoalveolar lavage fluid (BALF) samples from COVID-19 patients has enabled us to examine gene expression changes of human tissue in response to the SARS-CoV-2 virus infection. However, the underlying mechanisms of COVID-19 pathogenesis at single-cell resolution, its transcriptional drivers, and dynamics require further investigation. In this study, we applied machine learning algorithms to infer the trajectories of cellular changes and identify their transcriptional programs. Our study generated cellular trajectories that show the COVID-19 pathogenesis of healthy-to-moderate and healthy-to-severe on macrophages and T cells, and we observed more diverse trajectories in macrophages compared to T cells. Furthermore, our deep-learning algorithm DrivAER identified several pathways (e.g., xenobiotic pathway and complement pathway) and transcription factors (e.g., MITF and GATA3) that could be potential drivers of the transcriptomic changes for COVID-19 pathogenesis and the markers of the COVID-19 severity. Moreover, macrophages-related functions corresponded more to the disease severity compared to T cells-related functions. Our findings more proficiently dissected the transcriptomic changes leading to the severity of a COVID-19 infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d245306d2eb9c5ec667c8da10a9a48cb65cf35bc" target='_blank'>Investigating Cellular Trajectories in the Severity of COVID-19 and Their Transcriptional Programs Using Machine Learning Approaches</a></td>
          <td>
            Hyun-hwan Jeong, Johnathan Jia, Yulin Dai, L. Simon, Zhongming Zhao
          </td>
          <td>2021-04-24</td>
          <td>Genes</td>
          <td>13</td>
          <td>67</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a952e012e1800bbd1317b1a754da406bbe7cda" target='_blank'>Systems analysis of human innate immunity in COVID-19</a></td>
          <td>
            Sophie Müller, J. Schultze
          </td>
          <td>2023-05-01</td>
          <td>Seminars in Immunology</td>
          <td>5</td>
          <td>94</td>
        </tr>
    
        <tr id="We sought to define the host immune response, a.k.a, the “cytokine storm” that has been implicated in fatal COVID-19 using an AI-based approach. Over 45,000 transcriptomic datasets of viral pandemics were analyzed to extract a 166-gene signature using ACE2 as a ‘seed’ gene; ACE2 was rationalized because it encodes the receptor that facilitates the entry of SARS-CoV-2 (the virus that causes COVID-19) into host cells. Surprisingly, this 166-gene signature was conserved in all viral pandemics, including COVID-19, and a subset of 20-genes classified disease severity, inspiring the nomenclatures ViP and severe-ViP signatures, respectively. The ViP signatures pinpointed a paradoxical phenomenon wherein lung epithelial and myeloid cells mount an IL15 cytokine storm, and epithelial and NK cell senescence and apoptosis determines severity/fatality. Precise therapeutic goals were formulated and subsequently validated in high-dose SARS-CoV-2-challenged hamsters using neutralizing antibodies that abrogate SARS-CoV-2•ACE2 engagement. IL15/IL15RA were elevated in the lungs of patients with fatal disease, and plasma levels of the cytokine tracked with disease severity. Thus, the ViP signatures provide a quantitative and qualitative framework for titrating the immune response in viral pandemics and may serve as a powerful unbiased tool to rapidly assess disease severity and vet candidate drugs. One Sentence Summary The host immune response in COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe71745d79ad1edbe5a4564c177c150c61d69e5c" target='_blank'>AI-guided discovery of the invariant host response to viral pandemics</a></td>
          <td>
            D. Sahoo, G. Katkar, Soni Khandelwal, Mahdi Behroozikhah, Amanraj Claire, Vanessa Castillo, Courtney Tindle, MacKenzie Fuller, Sahar Taheri, T. Rogers, N. Beutler, S. Ramirez, Stephen A. Rawlings, Victor Pretorius, David M. Smith, Dennis R. Burton, L. C. Crotty Alexander, Jason M. Duran, S. Crotty, J. Dan, Soumita Das, P. Ghosh
          </td>
          <td>2020-09-22</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>100</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0bfbb262c434b216c8e33b76b7c2ad347e9d4a" target='_blank'>Index to Titles and Authors/Editors of Books and Software Reviewed</a></td>
          <td>
            
          </td>
          <td>1996-03-01</td>
          <td>The Quarterly Review of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39e4a41252dbf836ccd7c717542f5551024f59d9" target='_blank'>Guides: Alerting Services: Table of Contents, Saved Search, Citation Alerts, & RSS Feeds: Table of Contents Alerts</a></td>
          <td>
            Patty Lynn
          </td>
          <td>2014-04-23</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="Background Viral acute respiratory illnesses (viral ARIs) contribute significantly to human morbidity and mortality worldwide, but their successful treatment requires timely diagnosis of viral etiology, which is complicated by overlap in clinical presentation with the non-viral ARIs. Multiple pandemics in the twenty-first century to date have further highlighted the unmet need for effective monitoring of clinically relevant emerging viruses. Recent studies have identified conserved host response to viral infections in the blood. Methods We hypothesize that a similarly conserved host response in nasal samples can be utilized for diagnosis and to rule out viral infection in symptomatic patients when current diagnostic tests are negative. Using a multi-cohort analysis framework, we analyzed 1555 nasal samples across 10 independent cohorts dividing them into training and validation. Results Using six of the datasets for training, we identified 119 genes that are consistently differentially expressed in viral ARI patients ( N  = 236) compared to healthy controls ( N  = 146) and further down-selected 33 genes for classifier development. The resulting locked logistic regression-based classifier using the 33-mRNAs had AUC of 0.94 and 0.89 in the six training and four validation datasets, respectively. Furthermore, we found that although trained on healthy controls only, in the four validation datasets, the 33-mRNA classifier distinguished viral ARI from both healthy or non-viral ARI samples with > 80% specificity and sensitivity, irrespective of age, viral type, and viral load. Single-cell RNA-sequencing data showed that the 33-mRNA signature is dominated by macrophages and neutrophils in nasal samples. Conclusion This proof-of-concept signature has potential to be adapted as a clinical point-of-care test (‘RespVerity’) to improve the diagnosis of viral ARIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1995cde0ed47ce65e02d70369df4da3999fd5e65" target='_blank'>A machine learning classifier using 33 host immune response mRNAs accurately distinguishes viral and non-viral acute respiratory illnesses in nasal swab samples</a></td>
          <td>
            Rushika R. Pandya, Yudong He, T. Sweeney, Yehudit Hasin-Brumshtein, P. Khatri
          </td>
          <td>2023-08-28</td>
          <td>Genome Medicine</td>
          <td>2</td>
          <td>67</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57c5f7089fb8bb0cf1d7fad1e7a0676e703b8b9b" target='_blank'>Guides: Alerting Services: Table of Contents, Saved Search, Citation Alerts, & RSS Feeds: About Alerts</a></td>
          <td>
            Patty Lynn
          </td>
          <td>2014-04-23</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>