---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-02 09:38:10 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Rebalancing Viral and Immune Damage versus Tissue Repair Prevents Death from Lethal Influenza Infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Hundreds of immune cell types work in coordination to maintain tissue homeostasis. Upon infection, dramatic changes occur with the localization, migration, and proliferation of the immune cells to first alert the body of the danger, confine it to limit spreading, and finally extinguish the threat and bring the tissue back to homeostasis. Since current technologies can follow the dynamics of only a limited number of cell types, we have yet to grasp the full complexity of global in vivo cell dynamics in normal developmental processes and disease. Here, we devise a computational method, digital cell quantification (DCQ), which combines genome‐wide gene expression data with an immune cell compendium to infer in vivo changes in the quantities of 213 immune cell subpopulations. DCQ was applied to study global immune cell dynamics in mice lungs at ten time points during 7 days of flu infection. We find dramatic changes in quantities of 70 immune cell types, including various innate, adaptive, and progenitor immune cells. We focus on the previously unreported dynamics of four immune dendritic cell subtypes and suggest a specific role for CD103+ CD11b− DCs in early stages of disease and CD8+ pDC in late stages of flu infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77a0a3591292c89f8a23e67e05e4e9abcd805fa" target='_blank'>Digital cell quantification identifies global immune cell dynamics during influenza infection</a></td>
          <td>
            Z. Altboum, Yael Steuerman, Eyal David, Z. Barnett-Itzhaki, Liran Valadarsky, Hadas Keren-Shaul, T. Meningher, E. Mendelson, M. Mandelboim, Irit Gat-Viks, I. Amit
          </td>
          <td>2014-02-01</td>
          <td>Molecular Systems Biology</td>
          <td>114</td>
          <td>106</td>
        </tr>
    
        <tr id="[This corrects the article DOI: 10.1155/2022/8660752.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06dae337126e2677af35c5fd5ba2bc5d41eea9da" target='_blank'>Corrigendum to “HD-13 Induces Swine Pneumonia Progression via Activation of TLR9”</a></td>
          <td>
            Jing Liu, Jiayi Xu, Honglei Zhou, Lihua Wang
          </td>
          <td>2022-11-16</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>2</td>
          <td>5</td>
        </tr>
    
        <tr id="Increasing evidence points towards the key role of the epithelium in the systemic and over-activated immune response to viral infection, including SARS-CoV-2 infection. Yet, how viral infection alters epithelial–immune cell interactions regulating inflammatory responses, is not well known. Available experimental approaches are insufficient to properly analyse this complex system, and computational predictions and targeted data integration are needed as an alternative approach. In this work, we propose an integrated computational biology framework that models how infection alters intracellular signalling of epithelial cells and how this change impacts the systemic immune response through modified interactions between epithelial cells and local immune cell populations. As a proof-of-concept, we focused on the role of intestinal and upper-airway epithelial infection. To characterise the modified epithelial–immune interactome, we integrated intra- and intercellular networks with single-cell RNA-seq data from SARS-CoV-2 infected human ileal and colonic organoids as well as from infected airway ciliated epithelial cells. This integrated methodology has proven useful to point out specific epithelial–immune interactions driving inflammation during disease response, and propose relevant molecular targets to guide focused experimental analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1769cf61d9dd494ac1e38ca9ce3846c8857f9d71" target='_blank'>Mapping the epithelial–immune cell interactome upon infection in the gut and the upper airways</a></td>
          <td>
            M. Poletti, A. Treveil, L. Csabai, L. Gul, D. Módos, M. Madgwick, M. Olbei, B. Bohár, Alberto Valdeolivas, D. Turei, B. Verstockt, Sergio Triana, T. Alexandrov, J. Sáez-Rodríguez, M. Stanifer, S. Boulant, T. Korcsmáros
          </td>
          <td>2022-05-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>5</td>
          <td>49</td>
        </tr>
    
        <tr id="The adaptive immune response is a powerful tool, capable of recognizing, binding to, and neutralizing a vast number of internal and external threats via T or B lymphatic receptors with widespread sets of antigen specificities. The emergence of high-throughput sequencing technology and bioinformatics provides opportunities for research in the fields of life sciences and medicine. The analysis and annotation for immune repertoire data can reveal biologically meaningful information, including immune prediction, target antigens, and effective evaluation. Continuous improvements of the immunological repertoire sequencing methods and analysis tools will help to minimize the experimental and calculation errors and realize the immunological information to meet the clinical requirements. That said, the clinical application of adaptive immune repertoire sequencing requires appropriate experimental methods and standard analytical tools. At the population cell level, we can acquire the overview of cell groups, but the information about a single cell is not obtained accurately. The information that is ignored may be crucial for understanding the heterogeneity of each cell, gene expression and drug response. The combination of high-throughput sequencing and single-cell technology allows us to obtain single-cell information with low-cost and high-throughput. In this review, we summarized the current methods and progress in this area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e219137f88b4df35bbb4901fbbc45c04dabc236a" target='_blank'>The methods and advances of adaptive immune receptors repertoire sequencing</a></td>
          <td>
            Hongmei Liu, Wenjing Pan, Congli Tang, Yujie Tang, Haijing Wu, A. Yoshimura, Yan Deng, N. He, Song Li
          </td>
          <td>2021-08-19</td>
          <td>Theranostics</td>
          <td>32</td>
          <td>121</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39e4a41252dbf836ccd7c717542f5551024f59d9" target='_blank'>Guides: Alerting Services: Table of Contents, Saved Search, Citation Alerts, & RSS Feeds: Table of Contents Alerts</a></td>
          <td>
            Patty Lynn
          </td>
          <td>2014-04-23</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57c5f7089fb8bb0cf1d7fad1e7a0676e703b8b9b" target='_blank'>Guides: Alerting Services: Table of Contents, Saved Search, Citation Alerts, & RSS Feeds: About Alerts</a></td>
          <td>
            Patty Lynn
          </td>
          <td>2014-04-23</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82117ef0e60e11080da326a0ba5412fc95fc9c2a" target='_blank'>Modeling and Remodeling the Cell: How Digital Twins and HCMV Can Elucidate the Complex Interactions of Viral Latency, Epigenetic Regulation, and Immune Responses</a></td>
          <td>
            Hana McMahon-Cole, Alicia E. Johnson, S. Aghamiri, T. Helikar, Lindsey B. Crawford
          </td>
          <td>2023-07-29</td>
          <td>Current Clinical Microbiology Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>
    
        <tr id="Trained Immunity is the nonspecific (pathogen agnostic) memory of innate immune cells, characterized by altered responses upon secondary stimulation. This review provides a RoadMap for the discovery and development of therapeutics targeting Trained Immunity, aimed at researchers with strong scientific backgrounds but limited drug development experience. The article outlines five drug development domains – epigenetic, metabolic, differentiation, inflammatory, and memory changes – that guide the identification of molecular targets, model selection, and biomarker development for the discovery and development of Trained Immunity-based therapeutics. It emphasizes the application of preclinical models and artificial intelligence in target discovery and compound screening. Additionally, the review addresses challenges in translating preclinical Trained Immunity findings to clinical trials and highlights relevant disease indications and ongoing clinical trials. This review integrates scientific findings with development strategy and thereby aims to bridge the gap between discovery and clinical application, advancing the field of Trained Immunity-based therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e32e161fe88188b7ab3aacb49680bec8bd2cf08" target='_blank'>Trained Immunity: RoadMap for drug discovery and development</a></td>
          <td>
            Jelmer H. van Puffelen, Callum Campbell, Irene Gander-Meisterernst, Johanna Holldack, Pauline T Lukey
          </td>
          <td>2025-12-23</td>
          <td>eLife</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr id="The immune system is a complex and dynamic network, crucial for combating infections and maintaining health. Developing a comprehensive digital twin of the immune system requires incorporating essential cellular components and their interactions. This study presents the first blueprint for an immune system digital twin, consisting of a comprehensive and simulatable mechanistic model. It integrates 51 innate and adaptive immune cells, 37 secretory factors, and 11 disease conditions, providing the foundation for developing a multi-scale model. The cellular-level model demonstrates its potential in characterizing immune responses to various single and combinatorial disease conditions. By making the model available in easy-to-use formats directly in the Cell Collective platform, the community can easily and further expand it. This blueprint represents a significant step towards developing general-purpose immune digital twins, with far-reaching implications for the future of digital twin technology in life sciences and healthcare, advancing patient care, and accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6641c34ef9f1144419c9d233fad85729b60ada25" target='_blank'>Immune Digital Twin Blueprint: A Comprehensive Mechanistic Model of the Human Immune System</a></td>
          <td>
            Rada Amin, S. Aghamiri, B. L. Puniya, Lauren Mayo, Dennis Startsev, Kashish Poore, Resa M. K. Helikar, T. Helikar
          </td>
          <td>2020-03-12</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>27</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>