---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-02 09:15:05 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Rebalancing Viral and Immune Damage versus Tissue Repair Prevents Death from Lethal Influenza Infection</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Acute lung injury (ALI) is a serious complication in clinical settings. This study aimed to elucidate the immune molecular mechanisms underlying ALI by bioinformatics analysis. Human ALI and six ALI mouse model datasets were collected. Immune cell infiltration between the ALI samples and non-ALI controls was estimated using the ssGSEA algorithm. Least absolute shrinkage and selection operator (LASSO) regression analysis and Wilcoxon test were performed to obtain the significantly different immune cell infiltration types. Immune feature genes were screened by differential analysis and the weighted correlation network analysis (WGCNA) algorithm. Functional enrichment was then performed and candidate hub biomarkers were identified. Finally, the receiver operator characteristic curve (ROC) analysis was used to predict their diagnostic performances. Three significantly different immune cell types (B cells, CD4 T cells, and CD8 T cells) were identified between the ALI samples and controls. A total of 13 immune feature genes were obtained by WGCNA and differential analysis and found to be significantly associated with immune functions and lung diseases. Four hub genes, including CD180, CD4, CD74, and MCL1 were identified using cytoHubba and were shown to have good specificity and sensitivity for the diagnosis of ALI. Correlation analysis suggested that CD4 was positively associated with T cells, whereas MCL1 was negatively correlated with B and T cells. We found that CD180, CD4, CD74, and MCL1 can serve as specific immune biomarkers for ALI. MCL1–B cell, MCL1–T cell, and CD4–T cell axes may be involved in the progression of ALI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18db2bcf6945f18cfc74571fc3a49f9b4bdb334d" target='_blank'>Identification of Immune Infiltration and Effective Immune Biomarkers in Acute Lung Injury by Bioinformatics Analysis</a></td>
          <td>
            Dandan Ling, Xiang Zhang, Jiamin Wu, Qianyun Xu, Zhiyong He, Jun Zhang
          </td>
          <td>2022-01-01</td>
          <td>Cell Transplantation</td>
          <td>6</td>
          <td>13</td>
        </tr>
    
        <tr id="Anti-TNF drugs are frequently associated with serious Adverse Events (AEs), which necessitates an improved understanding of individual factors that determine efficacy and safety of anti-TNF agents. We mined the US FDA’s Adverse Event Reporting System (AERS) for anti-TNF-associated AEs to identify and stratify patient subgroups and drug combinations that exhibit specifically correlated complications. We demonstrate the existence of patient subgroup and anti-TNF agent-specific associations for relative risks of developing known and novel AEs including infections, edema, and organ damage associated processes. Concomitant use of anti-TNFs with corticosteroids significantly increased risk of AEs (p < 0.001) including pulmonary fibrosis and pulmonary edema. Using these tightly correlated phenotypes, we mined mouse phenotype data to identify the molecular basis of these AEs. Multiple pathways and networks that regulate injury response, fluid regulation, and wound healing were implicated suggesting modification of anti-TNF-based therapeutic strategies to minimize corticosteroid-based combinatorial risk of severe AEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/543f0f1202d7709a6b6282f97da37faf186516ad" target='_blank'>Using Systems Biology-based Analysis Approaches to Identify Mechanistically Significant Adverse Drug Reactions: Pulmonary Complications from Combined Use of Anti-TNFα Agents and Corticosteroids</a></td>
          <td>
            M. Sarangdhar, Akash Kushwaha, Jeanine E Dahlquist, A. Jegga, B. Aronow
          </td>
          <td>2013-03-18</td>
          <td>AMIA Summits on Translational Science Proceedings</td>
          <td>1</td>
          <td>92</td>
        </tr>
    
        <tr id="Lung injury is closely associated with the heterogeneity, severity, mortality, and prognosis of various respiratory diseases. Effective monitoring of lung injury is crucial for the optimal management and improved outcomes of patients with lung diseases. This review describes acute and chronic respiratory diseases characterized by significant lung injury and current clinical tools for assessing lung health. Furthermore, we summarized the mechanisms of lung cell death observed in these diseases and highlighted recently identified biomarkers in the plasma indicative of injury to specific cell types and scaffold structure in the lung. Last, we propose an artificial intelligence-driven lung injury monitoring model to assess disease severity, and predict mortality and prognosis, aiming to achieve precision and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/651735405eca6cb18c70caf058c6db248cb3d39b" target='_blank'>A narrative review on lung injury: mechanisms, biomarkers, and monitoring</a></td>
          <td>
            Wenping Fan, Biyu Gui, Xiaolei Zhou, Li Li, Huaiyong Chen
          </td>
          <td>2024-10-31</td>
          <td>Critical Care</td>
          <td>7</td>
          <td>5</td>
        </tr>
    
        <tr id="Respiratory and emergency medicine have undergone significant transformations in recent years, shaped by advancements in critical care, the integration of artificial intelligence (AI), and the evolving understanding of post-infectious sequelae [...].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/921897d8b5158c1932e17d4ed17b3bbbc7a13bd1" target='_blank'>Pioneering Respiratory and Emergency Medicine Research During the COVID-19 Era</a></td>
          <td>
            I. Pantazopoulos, G. Mavrovounis, Konstantinos I. Gourgoulianis
          </td>
          <td>2025-02-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr id="
 Inflammation is common in virtually all diseases. Since immune cells travel through the blood to inflamed tissues, peripheral blood reflects inflammation at distant sites. However, blood biomarkers are hard to detect in global gene expression data since blood is not the site of perturbation and statistical corrections for multiple testing reduce sensitivity. We hypothesized that combining transcriptional signatures from inflamed tissue and machine learning would be useful to derive blood biomarkers. We first tested this in the mouse model of influenza infection. We identified a set of early genes in the lungs associated with lethality after infection. We then tested their differential abundance early in the blood of mice infected with a range of influenza doses. We applied elastic net to build a biomarker able to predict disease outcome (4 classes: early lethal, late lethal, non-lethal vs. non-infected; accuracy on training and validation sets: 91% and 79%). We next confirmed our hypothesis in lung cancer and breast cancer in humans. We identified sets of genes associated with tumors but not with tumor-free tissues and tested their differential abundance in the blood of patients compared to healthy controls. Using elastic net, we built two blood biomarkers to classify subjects according to disease status (case vs. control; accuracy on training and validation sets in breast cancer: 92% and 67%; in lung cancer: 89% and 78%). In conclusion, combining inflamed tissue signatures and machine learning, we were able to derive blood biomarkers for three different diseases in two different species. We expect that this novel strategy can be applied in other diseases as well.
 This work was supported by the Intramural Research Program of NIAID, NIH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95ae3cefc1c60c582470175b79c18a533b7a4b2" target='_blank'>Combining gene expression data from inflamed tissue and machine learning for blood biomarker discovery</a></td>
          <td>
            Pedro Milanez-Almeida, A. Martins, Manikandan Narayanan, P. Torabi-Parizi, Luis Franco, J. Tsang, R. N. Germain
          </td>
          <td>2017-05-01</td>
          <td>The Journal of Immunology</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr id="RATIONALE
Mechanical ventilation augments the acute lung injury (ALI) caused by bacterial products. The molecular pathogenesis of this synergistic interaction remains incompletely understood.


OBJECTIVE
We sought to develop a computational framework to systematically identify gene regulatory networks activated in ALI.


METHODS
We have developed a mouse model in which the combination of mechanical ventilation and intratracheal LPS produces significantly more injury to the lung than either insult alone. We used global gene ontology analysis to determine overrepresented biological modules and computational transcription factor analysis to identify putative regulatory factors involved in this model of ALI.


RESULTS
By integrating expression profiling with gene ontology and promoter analysis, we constructed a large-scale regulatory modular map of the important processes activated in ALI. This map assigned differentially expressed genes to highly overrepresented biological modules, including "defense response," "immune response," and "oxidoreductase activity." These modules were then systematically incorporated into a gene regulatory network that consisted of putative transcription factors, such as IFN-stimulated response element, IRF7, and Sp1, that may regulate critical processes involved in the pathogenesis of ALI.


CONCLUSIONS
We present a novel, unbiased, and powerful computational approach to investigate the synergistic effects of mechanical ventilation and LPS in promoting ALI. Our methodology is applicable to any expression profiling experiment involving eukaryotic organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f595e6a15b63b8ddd0d4e333d8db4bd03984ce5" target='_blank'>Computational identification of key biological modules and transcription factors in acute lung injury.</a></td>
          <td>
            S. Gharib, W. Liles, G. Matute-Bello, R. Glenny, T. R. Martin, W. Altemeier
          </td>
          <td>2006-03-15</td>
          <td>American journal of respiratory and critical care medicine</td>
          <td>60</td>
          <td>52</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5204e1908226cce19619229db21587c834b5dd" target='_blank'>Machine Learning Analysis of the Bleomycin Mouse Model Reveals the Compartmental and Temporal Inflammatory Pulmonary Fingerprint</a></td>
          <td>
            Natalie Bordag, V. Biasin, D. Schnoegl, F. Valzano, K. Jandl, B. Nagy, N. Sharma, M. Wygrecka, G. Kwapiszewska, L. Marsh
          </td>
          <td>2020-05-26</td>
          <td>iScience</td>
          <td>17</td>
          <td>45</td>
        </tr>
    
        <tr id="Gene networks have arisen as a promising tool in the comprehensive modeling and analysis of complex diseases. Particularly in viral infections, the understanding of the host-pathogen mechanisms, and the immune response to these, is considered a major goal for the rational design of appropriate therapies. For this reason, the use of gene networks may well encourage therapy-associated research in the context of the coronavirus pandemic, orchestrating experimental scrutiny and reducing costs. In this work, gene co-expression networks were reconstructed from RNA-Seq expression data with the aim of analyzing the time-resolved effects of gene Ly6E in the immune response against the coronavirus responsible for murine hepatitis (MHV). Through the integration of differential expression analyses and reconstructed networks exploration, significant differences in the immune response to virus were observed in Ly6EΔHSC compared to wild type animals. Results show that Ly6E ablation at hematopoietic stem cells (HSCs) leads to a progressive impaired immune response in both liver and spleen. Specifically, depletion of the normal leukocyte mediated immunity and chemokine signaling is observed in the liver of Ly6EΔHSC mice. On the other hand, the immune response in the spleen, which seemed to be mediated by an intense chromatin activity in the normal situation, is replaced by ECM remodeling in Ly6EΔHSC mice. These findings, which require further experimental characterization, could be extrapolated to other coronaviruses and motivate the efforts towards novel antiviral approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a2d142649ec83ee9cc41dffea7e0a26c3f539e" target='_blank'>Computational Analysis of the Global Effects of Ly6E in the Immune Response to Coronavirus Infection Using Gene Networks</a></td>
          <td>
            F. Delgado-Chaves, Francisco Gómez-Vela, F. Divina, M. García-Torres, Domingo S. Rodríguez-Baena
          </td>
          <td>2020-07-01</td>
          <td>Genes</td>
          <td>12</td>
          <td>24</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39e4a41252dbf836ccd7c717542f5551024f59d9" target='_blank'>Guides: Alerting Services: Table of Contents, Saved Search, Citation Alerts, & RSS Feeds: Table of Contents Alerts</a></td>
          <td>
            Patty Lynn
          </td>
          <td>2014-04-23</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>