---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-26 06:02:02 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Myeloid progenitor dysregulation fuels immunosuppressive macrophages in tumors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Background Lipid metabolism pivotally contributes to the incidence and development of lung adenocarcinoma (LUAD). The interaction of lipid metabolism and tumor microenvironment (TME) has become a new research direction. Methods Using the 1107 LUAD records from the Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases, a comprehensive exploration was performed on the heterogeneous lipid metabolism subtypes based on lipid metabolism genes (LMGs) and immune-related genes (LRGs). The clinical significance, functional status, TME interaction and genomic changes of different subtypes were further studied. A new scoring system, lipid-immune score (LIS), was developed and validated. Results Two heterogeneous subtypes, which express more LMGs and show the characteristics of tumor metabolism and proliferation, are defined as lipid metabolism phenotypes. The prognosis of lipid metabolism phenotype is poor, and it is more common in patients with tumor progression. Expressing more IRGs, enrichment of immunoactive pathways and infiltration of effector immune cells are defined as immunoactive phenotypes. The immunoactive phenotype has a better prognosis and stronger anti-tumor immunity and is more sensitive to immunotherapy. In addition, KEAP1 is a driving mutant gene in the lipid metabolism subtype. Finally, LIS was developed and confirmed to be a robust predictor of overall survival (OS) and immunotherapy in LUAD patients. Conclusion Two heterogeneous subtypes of LUAD (lipid metabolism subtype and immune activity subtype) were identified to evaluate prognosis and immunotherapy sensitivity. Our research promotes the understanding of the interaction between lipid metabolism and TME and offers a novel direction for clinical management and precision therapy aimed to LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a048b2a423976a05680233fce270b75a4d63680" target='_blank'>Machine learning reveals two heterogeneous subtypes to assist immune therapy based on lipid metabolism in lung adenocarcinoma</a></td>
          <td>
            Xuyu Gu, Shiyou Wei, Zhixin Li, Huan Xu
          </td>
          <td>2022-09-27</td>
          <td>Frontiers in Immunology</td>
          <td>4</td>
          <td>14</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1831b820faa617bad051fddf25c5f42afc8f9050" target='_blank'>Discovery of tumor immune infiltration-related snoRNAs for predicting tumor immune microenvironment status and prognosis in lung adenocarcinoma</a></td>
          <td>
            Rongjun Wan, L. Bai, Changjing Cai, Wang Ya, Juan Jiang, Chengping Hu, Qiong Chen, Bingrong Zhao, Yuanyuan Li
          </td>
          <td>2021-11-01</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>8</td>
          <td>17</td>
        </tr>
    
        <tr id="ABSTRACT Non-small cell lung cancer is one of the leading causes of cancer-related death in the world. Lung adenocarcinoma, the most common type of non-small cell lung cancer, has been well characterized as having a dense lymphocytic infiltrate, suggesting that the immune system plays an active role in shaping this cancer's growth and development. Despite these findings, our understanding of how this infiltrate affects patient prognosis and its association with lung adenocarcinoma-specific clinical factors remains limited. To address these questions, we inferred the infiltration level of six distinct immune cell types from a series of four lung adenocarcinoma gene expression datasets. We found that naive B cell, CD8+ T cell, and myeloid cell-derived expression signals of immune infiltration were significantly predictive of patient survival in multiple independent datasets, with B cell and CD8+ T cell infiltration associated with prolonged prognosis and myeloid cell infiltration associated with shorter survival. These associations remained significant even after accounting for additional clinical variables. Patients stratified by smoking status exhibited decreased CD8+ T cell infiltration and altered prognostic associations, suggesting potential immunosuppressive mechanisms in smokers. Survival analyses accounting for immune checkpoint gene expression and cellular immune infiltrate indicated checkpoint protein-specific modulatory effects on CD8+ T cell and B cell function that may be associated with patient sensitivity to immunotherapy. Together, these analyses identified reproducible associations that can be used to better characterize the role of immune infiltration in lung adenocarcinoma and demonstrate the utility in using computational approaches to systematically characterize tissue-specific tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f31433e71319d79734506ea0ea2a773252885f" target='_blank'>Computational immune profiling in lung adenocarcinoma reveals reproducible prognostic associations with implications for immunotherapy</a></td>
          <td>
            F. Varn, L. Tafe, C. Amos, Chao Cheng
          </td>
          <td>2018-02-01</td>
          <td>Oncoimmunology</td>
          <td>34</td>
          <td>115</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06fa901027681eacec7ece647736f51dfea2e99c" target='_blank'>Systems biology analysis identifies TNFRSF9 as a functional marker of tumor-infiltrating regulatory T-cell enabling clinical outcome prediction in lung cancer</a></td>
          <td>
            Jae-Won Cho, Jimin Son, S. Ha, Insuk Lee
          </td>
          <td>2021-01-21</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>16</td>
          <td>42</td>
        </tr>
    
        <tr id="Background In recent years, the immunotherapy of lung adenocarcinoma has developed rapidly, but the good therapeutic effect only exists in some patients, and most of the current predictors cannot predict it very well. Tumor-infiltrating macrophages have been reported to play a crucial role in lung adenocarcinoma (LUAD). Thus, we want to build novel molecular markers based on macrophages. Methods By non-negative matrix factorization (NMF) algorithm and Cox regression analysis, we constructed macrophage-related subtypes of LUAD patients and built a novel gene signature consisting of 12 differentially expressed genes between two subtypes. The gene signature was further validated in Gene-Expression Omnibus (GEO) datasets. Its predictive effect on prognosis and immunotherapy outcome was further evaluated with rounded analyses. We finally explore the role of TRIM28 in LUAD with a series of in vitro experiments. Results Our research indicated that a higher LMS score was significantly correlated with tumor staging, pathological grade, tumor node metastasis stage, and survival. LMS was identified as an independent risk factor for OS in LUAD patients and verified in GEO datasets. Clinical response to immunotherapy was better in patients with low LMS score compared to those with high LMS score. TRIM28, a key gene in the gene signature, was shown to promote the proliferation, invasion and migration of LUAD cell. Conclusion Our study highlights the significant role of gene signature in predicting the prognosis and immunotherapy efficacy of LUAD patients, and identifies TRIM28 as a potential biomarker for the treatment of LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ac19ef9cb502db1abc4ccf806df7b06d4446f0" target='_blank'>Macrophage-Related Gene Signatures for Predicting Prognosis and Immunotherapy of Lung Adenocarcinoma by Machine Learning and Bioinformatics</a></td>
          <td>
            Yunzhi Xiang, Guanghui Wang, Baoliang Liu, Hao Zheng, Qian-qian Liu, G. Ma, Jiajun Du
          </td>
          <td>2024-02-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr id="Lung cancer remains the leading cause of cancer death globally, with lung adenocarcinoma (LUAD) being its most prevalent subtype. Due to the heterogeneity of LUAD, patients given the same treatment regimen may have different responses and clinical outcomes. Therefore, identifying new subtypes of LUAD is important for predicting prognosis and providing personalized treatment for patients. Pyroptosis-related genes play an essential role in anticancer, but there is limited research investigating pyroptosis in LUAD. In this study, 33 pyroptosis gene expression profiles and clinical information were collected from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases. By bioinformatics and machine learning analyses, we identified novel subtypes of LUAD based on 10 pyroptosis-related genes and further validated them in the GEO dataset, with machine learning models performing up to an AUC of 1 for classifying in GEO. A web-based tool was established for clinicians to use our clustering model (http://www.aimedicallab.com/tool/aiml-subphe-luad.html). LUAD patients were clustered into 3 subtypes (A, B, and C), and survival analysis showed that B had the best survival outcome and C had the worst survival outcome. The relationships between pyroptosis gene expression and clinical characteristics were further analyzed in the three molecular subtypes. Immune profiling revealed significant differences in immune cell infiltration among the three molecular subtypes. GO enrichment and KEGG pathway analyses were performed based on the differential genes of the three subtypes, indicating that differentially expressed genes (DEGs) were involved in multiple cellular and biological functions, including RNA catabolic process, mRNA catabolic process, and pathways of neurodegeneration-multiple diseases. Finally, we developed an 8-gene prognostic model that accurately predicted 1-, 3-, and 5-year overall survival. In conclusion, pyroptosis-related genes may play a critical role in LUAD, and provide new insights into the underlying mechanisms of LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/772f3a2e967b991cbeeec01c0336c1a90e48757c" target='_blank'>Identification and Validation of the Pyroptosis-Related Molecular Subtypes of Lung Adenocarcinoma by Bioinformatics and Machine Learning</a></td>
          <td>
            Le-ping Liu, Lu Lu, Qiang Zhao, Qinjie Kou, Zhengqian Jiang, R. Gui, Yanwei Luo, Qin-yu Zhao
          </td>
          <td>2021-11-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>28</td>
          <td>25</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faacd271ce482436d2209b74e0a77669df7741e9" target='_blank'>Artificial Intelligence-Powered Hematoxylin and Eosin Analyzer Reveals Distinct Immunologic and Mutational Profiles among Immune Phenotypes in Non-Small-Cell Lung Cancer.</a></td>
          <td>
            Jonghanne Park, Hyung-Gyo Cho, Jewel Park, G. Lee, Hye Sung Kim, K. Paeng, Sanghoon Song, G. Park, C. Ock, Y. Chae
          </td>
          <td>2022-04-01</td>
          <td>The American journal of pathology</td>
          <td>7</td>
          <td>46</td>
        </tr>
    
        <tr id="The ability to characterize immune cells and explore the molecular interactions that govern their functions has never been greater, fueled in recent years by the revolutionary advance of single-cell analysis platforms. However, precisely how immune cells respond to different stimuli and where differentiation processes and effector functions operate remain incompletely understood. Inferring cellular fate within single-cell transcriptomic analyses is now omnipresent, despite the assumptions typically required in such analyses. Recently developed experimental models support dynamic analyses of the immune response, providing insights into the temporal changes that occur within cells and the tissues in which such transitions occur. Here we will review these approaches and discuss how these can be combined with single-cell technologies to develop a deeper understanding of the immune responses that should support the development of better therapeutic options for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df359d3a21a0e0d9ae308793407ba6d01736c657" target='_blank'>Decoding changes in tumor-infiltrating leukocytes through dynamic experimental models and single-cell technologies.</a></td>
          <td>
            Colin Y C Lee, M. Clatworthy, David R Withers
          </td>
          <td>2024-06-10</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>53</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>