---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-22 09:15:33 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Myeloid progenitor dysregulation fuels immunosuppressive macrophages in tumors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Tumor‐associated macrophages (TAMs) are a vital immune component within the tumor microenvironment (TME) of lung adenocarcinoma (LUAD), exerting significant influence on tumor growth, metastasis, and drug resistance. N6‐methyladenosine (m6A) modifications regulate gene expression and various facets of cancer biology; nonetheless, the mechanisms by which they modulate gene expression in TAMs and their impact on LUAD progression remain inadequately elucidated. Single‐cell transcriptome analysis identified the macrophage m6A‐related genes (MMRGs) with high expression in TAMs and linked to m6A modifications in LUAD. The MMRGs were employed to construct 801 prognostic models by 13 different machine learning (ML) algorithms. An integrative multi‐omics approach was utilized to analyze the potential biological functions of MMRGs. Five ML algorithms were employed to discover potential biomarkers for patient stratification and precision therapy. Potential drugs for treating LUAD were identified and assessed by molecular docking and molecular dynamics, and ML was employed to determine the most promising candidates. Immunohistochemistry and immunofluorescence staining were conducted to assess MMRG expression in LUAD tissues. Seventeen MMRGs were identified in LUAD and subsequently employed to construct a prognostic model for patient stratification into high‐risk and low‐risk groups. The impact of MMRG expression on various tumor immune phenotypes, such as tumor stemness, heterogeneity, hallmark pathway enrichment, TAM infiltration, and immune landscape, was thoroughly characterized. PIM3, HMGB2, DUSP2, NR4A2, and others were recognized as promising biomarkers for patient classification and precision therapy. Furthermore, it was predicted that drugs such as BRD9876 and MK‐1775 would demonstrate therapeutic efficacy in treating LUAD, and drugs showing potential binding with DUSP2, ZNF331, FLT, and LYZ were identified. Finally, experimental validation was conducted to confirm the protein expression and distribution of DUSP2 and NR4A2 in tissues of LUAD. Our study offers valuable insights into the biological significance of MMRGs, shedding light on novel mechanisms of tumor development and immune evasion in LUAD. Furthermore, our findings have identified potential biomarkers, drug candidates, and therapeutic targets that may improve the management of LUAD in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae46da9569e5c6c4bf7aeb7e8d86c57f59f25b2" target='_blank'>Characterization of m6A‐Related Genes in Tumor‐Associated Macrophages for Prognosis, Immunotherapy, and Drug Prediction in Lung Adenocarcinomas Based on Machine Learning Algorithms</a></td>
          <td>
            Mengmeng Sang, Qinhua Huang, Mimi Mao, Haoming Yan, Jia Ge, Rui Zhang, Wenhao Ju, Xiaorong Zhou
          </td>
          <td>2025-06-05</td>
          <td>The FASEB Journal</td>
          <td>1</td>
          <td>3</td>
        </tr>
    
        <tr id="Lung cancer continues to be the leading cause of cancer‐related mortality worldwide. Early detection and a comprehensive understanding of tumor‐immune interactions are crucial for improving patient outcomes. This study aimed to develop a novel biomarker panel utilizing peripheral blood transcriptomics and machine learning algorithms for early lung cancer diagnosis, while simultaneously providing insights into tumor‐immune crosstalk mechanisms. Leveraging a training cohort (GSE135304), we employed multiple machine learning algorithms to formulate a Lung Cancer Diagnostic Score (LCDS) based on peripheral blood transcriptomic features. The LCDS model's performance was evaluated using the area under the receiver operating characteristic (ROC) curve (AUC) in multiple validation cohorts (GSE42834, GSE157086, and an in‐house dataset). Peripheral blood samples were obtained from 20 lung cancer patients and 10 healthy control subjects, representing an in‐house cohort recruited at the Sixth People's Hospital of Chengdu. We employed advanced bioinformatics techniques to explore tumor‐immune interactions through comprehensive immune infiltration and pathway enrichment analyses. Initial screening identified 844 differentially expressed genes, which were subsequently refined to 87 genes using the Boruta feature selection algorithm. The random forest (RF) algorithm demonstrated the highest accuracy in constructing the LCDS model, yielding a mean AUC of 0.938. Lower LCDS values were significantly associated with elevated immune scores and increased CD4+ and CD8+ T‐cell infiltration, indicative of enhanced antitumor‐immune responses. Higher LCDS scores correlated with activation of hypoxia, peroxisome proliferator‐activated receptor (PPAR), and Toll‐like receptor (TLR) signaling pathways, as well as reduced DNA damage repair pathway scores. Our study presents a novel, machine learning‐derived peripheral blood transcriptomic biomarker panel with potential applications in early lung cancer diagnosis. The LCDS model not only demonstrates high accuracy in distinguishing lung cancer patients from healthy individuals but also offers valuable insights into tumor‐immune interactions and underlying cancer biology. This approach may facilitate early lung cancer detection and contribute to a deeper understanding of the molecular and cellular mechanisms underlying tumor‐immune crosstalk. Furthermore, our findings on the relationship between LCDS and immune infiltration patterns may have implications for future research on therapeutic strategies targeting the immune system in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/370a2f4ae2cf142f63ac75064687574a6aab3e66" target='_blank'>Machine learning‐derived peripheral blood transcriptomic biomarkers for early lung cancer diagnosis: Unveiling tumor‐immune interaction mechanisms</a></td>
          <td>
            Xiaohua Li, Xuebing Li, Jiangyue Qin, Lei Lei, Hua Guo, Xi Zheng, Xuefeng Zeng
          </td>
          <td>2024-10-16</td>
          <td>Biofactors (Oxford, England)</td>
          <td>4</td>
          <td>3</td>
        </tr>
    
        <tr id="Background Lipid metabolism pivotally contributes to the incidence and development of lung adenocarcinoma (LUAD). The interaction of lipid metabolism and tumor microenvironment (TME) has become a new research direction. Methods Using the 1107 LUAD records from the Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases, a comprehensive exploration was performed on the heterogeneous lipid metabolism subtypes based on lipid metabolism genes (LMGs) and immune-related genes (LRGs). The clinical significance, functional status, TME interaction and genomic changes of different subtypes were further studied. A new scoring system, lipid-immune score (LIS), was developed and validated. Results Two heterogeneous subtypes, which express more LMGs and show the characteristics of tumor metabolism and proliferation, are defined as lipid metabolism phenotypes. The prognosis of lipid metabolism phenotype is poor, and it is more common in patients with tumor progression. Expressing more IRGs, enrichment of immunoactive pathways and infiltration of effector immune cells are defined as immunoactive phenotypes. The immunoactive phenotype has a better prognosis and stronger anti-tumor immunity and is more sensitive to immunotherapy. In addition, KEAP1 is a driving mutant gene in the lipid metabolism subtype. Finally, LIS was developed and confirmed to be a robust predictor of overall survival (OS) and immunotherapy in LUAD patients. Conclusion Two heterogeneous subtypes of LUAD (lipid metabolism subtype and immune activity subtype) were identified to evaluate prognosis and immunotherapy sensitivity. Our research promotes the understanding of the interaction between lipid metabolism and TME and offers a novel direction for clinical management and precision therapy aimed to LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a048b2a423976a05680233fce270b75a4d63680" target='_blank'>Machine learning reveals two heterogeneous subtypes to assist immune therapy based on lipid metabolism in lung adenocarcinoma</a></td>
          <td>
            Xuyu Gu, Shiyou Wei, Zhixin Li, Huan Xu
          </td>
          <td>2022-09-27</td>
          <td>Frontiers in Immunology</td>
          <td>5</td>
          <td>17</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1831b820faa617bad051fddf25c5f42afc8f9050" target='_blank'>Discovery of tumor immune infiltration-related snoRNAs for predicting tumor immune microenvironment status and prognosis in lung adenocarcinoma</a></td>
          <td>
            Rongjun Wan, L. Bai, Changjing Cai, Wang Ya, Juan Jiang, Chengping Hu, Qiong Chen, Bingrong Zhao, Yuanyuan Li
          </td>
          <td>2021-11-01</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>10</td>
          <td>19</td>
        </tr>
    
        <tr id="ABSTRACT Non-small cell lung cancer is one of the leading causes of cancer-related death in the world. Lung adenocarcinoma, the most common type of non-small cell lung cancer, has been well characterized as having a dense lymphocytic infiltrate, suggesting that the immune system plays an active role in shaping this cancer's growth and development. Despite these findings, our understanding of how this infiltrate affects patient prognosis and its association with lung adenocarcinoma-specific clinical factors remains limited. To address these questions, we inferred the infiltration level of six distinct immune cell types from a series of four lung adenocarcinoma gene expression datasets. We found that naive B cell, CD8+ T cell, and myeloid cell-derived expression signals of immune infiltration were significantly predictive of patient survival in multiple independent datasets, with B cell and CD8+ T cell infiltration associated with prolonged prognosis and myeloid cell infiltration associated with shorter survival. These associations remained significant even after accounting for additional clinical variables. Patients stratified by smoking status exhibited decreased CD8+ T cell infiltration and altered prognostic associations, suggesting potential immunosuppressive mechanisms in smokers. Survival analyses accounting for immune checkpoint gene expression and cellular immune infiltrate indicated checkpoint protein-specific modulatory effects on CD8+ T cell and B cell function that may be associated with patient sensitivity to immunotherapy. Together, these analyses identified reproducible associations that can be used to better characterize the role of immune infiltration in lung adenocarcinoma and demonstrate the utility in using computational approaches to systematically characterize tissue-specific tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f31433e71319d79734506ea0ea2a773252885f" target='_blank'>Computational immune profiling in lung adenocarcinoma reveals reproducible prognostic associations with implications for immunotherapy</a></td>
          <td>
            F. Varn, L. Tafe, C. Amos, Chao Cheng
          </td>
          <td>2018-02-01</td>
          <td>Oncoimmunology</td>
          <td>39</td>
          <td>115</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06fa901027681eacec7ece647736f51dfea2e99c" target='_blank'>Systems biology analysis identifies TNFRSF9 as a functional marker of tumor-infiltrating regulatory T-cell enabling clinical outcome prediction in lung cancer</a></td>
          <td>
            Jae-Won Cho, Jimin Son, S. Ha, Insuk Lee
          </td>
          <td>2021-01-21</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>23</td>
          <td>44</td>
        </tr>
    
        <tr id="Objective Lung adenocarcinoma (LUAD) continues to be a primary cause of cancer-related mortality globally, highlighting the urgent need for novel insights finto its molecular mechanisms. This study aims to investigate the relationship between gene expression and mitophagy in LUAD, with an emphasis on identifying key biomarkers and elucidating their roles in tumorigenesis and immune cell infiltration. Methods We utilized datasets GSE151101 and GSE203609 from the Gene Expression Omnibus (GEO) database to identify differentially expressed genes (DEGs) associated with lung cancer and mitophagy. DEGs were identified using GEO2R, filtered based on criteria of P < 0.05 and log2 fold change ≥ 1. Subsequently, Weighted Gene Co-expression Network Analysis (WGCNA) was conducted to classify DEGs into modules. Functional annotation of these modules was performed using Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses. Gene Set Enrichment Analysis (GSEA) was applied to the most relevant module, designated as the greenyellow module. To identify critical biomarkers, machine learning algorithms including Random Forest, Least Absolute Shrinkage and Selection Operator (LASSO) regression, and Support Vector Machine (SVM) were employed. Validation of the findings was conducted using The Cancer Genome Atlas (TCGA) database, Human Protein Atlas (HPA), quantitative PCR (qPCR), and immune cell infiltration analysis via CIBERSORTx. Results Our analysis identified 11,012 overlapping DEGs between the two datasets. WGCNA revealed 11 modules, with the green-yellow module exhibiting the highest correlation. Functional enrichment analysis highlighted significant associations with FOXM1 signaling pathways and retinoblastoma in cancer. Machine learning algorithms identified COASY, FTSJ1, and MOGS as pivotal genes. These findings were validated using TCGA data, qPCR experiments, which demonstrated high expression levels in LUAD samples. Immunohistochemistry from HPA confirmed consistency between protein levels and RNA-seq data. Furthermore, pan-cancer analysis indicated that these genes are highly expressed across various cancer types. Immune infiltration analysis suggested significant correlations between these genes and specific immune cell populations. Conclusion COASY, FTSJ1 and MOGS have emerged as critical biomarkers in LUAD, potentially influencing tumorigenesis through mitophagy-related mechanisms and immune modulation. These findings provide promising avenues for future research into targeted therapies and diagnostic tools, thereby enhancing LUAD management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0972d886cd27d4f854b91feb85db4c65079f1fb5" target='_blank'>The role and machine learning analysis of mitochondrial autophagy-related gene expression in lung adenocarcinoma</a></td>
          <td>
            Binyu Wang, Di Liu, Danfei Shi, Xinmin Li, Yong Li
          </td>
          <td>2025-04-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72b483258e391c7690780daf270bca7fc969d63b" target='_blank'>Enhancing immuno-oncology investigations through multidimensional decoding of tumor microenvironment with IOBR 2.0</a></td>
          <td>
            D. Zeng, Yiran Fang, Wenjun Qiu, Peng Luo, Shixiang Wang, Rongfang Shen, Wenchao Gu, Xiatong Huang, Qianqian Mao, Gaofeng Wang, Yonghong Lai, Guangda Rong, Xi Xu, Min Shi, Zuqiang Wu, Guangchuang Yu, Wangjun Liao
          </td>
          <td>2024-11-26</td>
          <td>Cell Reports Methods</td>
          <td>22</td>
          <td>17</td>
        </tr>
    
        <tr id="Background The molecular pathogenesis of lung adenocarcinoma (LUAD) involves genomic mutations, autophagy dysregulation, and signaling pathway disruptions. Autophagy, a key cellular process, is tightly linked to cancer development; genes like ATG5 and ATG10 influence lung cancer progression, and epigenetic regulators modulate autophagy-related carcinogenesis. However, the role of epigenetic-autophagy genes in LUAD’s tumor microenvironment is under-researched. Methods We used the “limma”” package to identify differential epigenetic-related genes associated with altered autophagy regulation (A-ERGs) in LUAD. Single-cell RNA sequencing was further employed to evaluate the heterogeneity of immune cells. Machine learning algorithms were utilized to construct and identify diagnostic markers for LUAD, which were then validated by receiver operating characteristic (ROC) curve analysis. Cell experiments, real-time PCR, and Western blot were conducted to verify the expression of KDM6B and KANSL1 and their effects on T-cell differentiation. Results Based on single-cell and transcriptome analyses, we screened 19 A-ERGs that were significantly differentially expressed in lung cancer tissues. These genes were primarily enriched in exhausted T cells. Subsequently, through machine learning, KDM6B and KANSL1 were identified to have excellent diagnostic performance. Single-cell level and transcriptome correlation analyses revealed that the expression of these two genes was associated with exhausted T cells. Results from in vitro cell experiments showed that high expression of these two genes promoted the occurrence of T cell exhaustion. Conclusion In this study, we utilized bulk and single-cell transcriptomic data to uncover the potential molecular mechanisms of A-ERGs in lung cancer. We explored the characteristic distribution of these genes in the tumor immune microenvironment and identified two A-ERGs, KDM6B and KANSL1, as potential diagnostic biomarkers for lung adenocarcinoma (LUAD). Our findings offer novel strategies for targeted therapeutic interventions in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1f597e92b4e6e03d62da92caf9bd7b0e87f372c" target='_blank'>Revealing key regulatory factors in lung adenocarcinoma: the role of epigenetic regulation of autophagy-related genes from transcriptomics, scRNA-seq, and machine learning</a></td>
          <td>
            Xianchang Zeng, Lingyun Wei, Lu Lv, Di Wu, Yingying Shen, Xinliang Lu, Xianghui Kong, Zhijian Cai, Jianli Wang
          </td>
          <td>2025-08-04</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>