---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-26 06:02:03 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>HIV-1 infection is associated with depletion of germinal center B cells and a decrease in IgA+ plasma cells in the GI tract</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="HIV RNA viral load (VL) is an important outcome variable in studies of HIV infected persons. There exists only a handful of methods which classify patients by VL patterns. Â Most methods place limits on the use of viral load measurements, are often specific to a particular study design, and do not account for complex, temporal variation. To address this issue, we propose a set of four unambiguous computable characteristics (features) of time-varying HIV viral load patterns, along with a novel centroid-based classification algorithm, which we use to classify a population of 1,576 HIV positive clinic patients into one of five different viral load patterns (clusters) often found in the literature: durably suppressed viral load (DSVL), sustained low viral load (SLVL), sustained high viral load (SHVL), high viral load suppression (HVLS), and rebounding viral load (RVL). The centroid algorithm summarizes these clusters in terms of their centroids and radii. We show that this allows new VL patterns to be assigned pattern membership based on the distance from the centroid relative to its radius, which we term radial normalization classification. This method has the benefit of providing an objective and quantitative method to assign VL pattern membership with a concise and interpretable model that aids clinical decision making. This method also facilitates meta-analyses by providing computably distinct HIV categories. Finally we propose that this novel centroid algorithm could also be useful in the areas of cluster comparison for outcomes research and data reduction in machine learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0927b99db41425f9e4ddb50be574ef673935bd6f" target='_blank'>Revealing HIV viral load patterns using unsupervised machine learning and cluster summarization</a></td>
          <td>
            S. Farooq, Samuel J. Weisenthal, Melissa Trayhan, Robert J. White, Kristen Bush, P. Mariuz, M. Zand
          </td>
          <td>2018-07-27</td>
          <td>F1000Research</td>
          <td>1</td>
          <td>33</td>
        </tr>
    
        <tr id="HIV RNA viral load (VL) is an important outcome variable in studies of HIV infected persons. There exists only a handful of methods which classify patients by viral load patterns. Most methods place limits on the use of viral load measurements, are often specific to a particular study design, and do not account for complex, temporal variation. To address this issue, we propose a set of four unambiguous computable characteristics (features) of time-varying HIV viral load patterns, along with a novel centroid-based classification algorithm, which we use to classify a population of 1,576 HIV positive clinic patients into one of five different viral load patterns (clusters) often found in the literature: durably suppressed viral load (DSVL), sustained low viral load (SLVL), sustained high viral load (SHVL), high viral load suppression (HVLS), and rebounding viral load (RVL). The centroid algorithm summarizes these clusters in terms of their centroids and radii. We show that this allows new viral load patterns to be assigned pattern membership based on the distance from the centroid relative to its radius, which we term radial normalization classification. This method has the benefit of providing an objective and quantitative method to assign viral load pattern membership with a concise and interpretable model that aids clinical decision making. This method also facilitates meta-analyses by providing computably distinct HIV categories. Finally we propose that this novel centroid algorithm could also be useful in the areas of cluster comparison for outcomes research and data reduction in machine learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bbb750d04dfab66fad37cac4116769313e39d8d" target='_blank'>Revealing patterns in HIV viral load data and classifying patients via a novel machine learning cluster summarization method</a></td>
          <td>
            S. Farooq, Samuel J. Weisenthal, Melissa Trayhan, Robert J. White, Kristen Bush, P. Mariuz, M. Zand
          </td>
          <td>2018-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>33</td>
        </tr>
    
        <tr id="Background: The primary hurdle to curing HIV is due to the establishment of a reservoir early in infection. In an effort to find new treatment strategies, we and others have focused on understanding the selection pressures exerted on the reservoir by studying how proviral sequences change over time. Methods: To gain insights into the dynamics of the HIV reservoir we analyzed longitudinal near full-length sequences from 7 people living with HIV between 1 and 20 years following the initiation of antiretroviral treatment. We used this data to employ Bayesian mixed effects models to characterize the decay of the reservoir using single-phase and multiphasic decay models based on near full-length sequencing. In addition, we developed a machine-learning approach utilizing logistic regression to identify elements within the HIV genome most associated with proviral decay and persistence. By systematically analyzing proviruses that are deleted for a specific element, we gain insights into their role in reservoir contraction and expansion. Results: Our analyses indicate that biphasic decay models of intact reservoir dynamics were better than single-phase models with a stronger statistical fit. Based on the biphasic decay pattern of the intact reservoir, we estimated the half-lives of the first and second phases of decay to be 18.2 (17.3 to 19.2, 95%CI) and 433 (227 to 6400, 95%CI) months, respectively. In contrast, the dynamics of defective proviruses differed favoring neither model definitively, with an estimated half-life of 87.3 (78.1 to 98.8, 95% CI) months during the first phase of the biphasic model. Machine-learning analysis of HIV genomes at the nucleotide level revealed that the presence of the splice donor site D1 was the principal genomic element associated with contraction. This role of D1 was then validated in an in vitro system. Using the same approach, we additionally found supporting evidence that HIV nef may confer a protective advantage for latently infected T cells while tat was associated with clonal expansion. Conclusions: The nature of intact reservoir decay suggests that the long-lived HIV reservoir contains at least 2 distinct compartments. The first compartment decays faster than the second compartment. Our machine-learning analysis of HIV proviral sequences reveals specific genomic elements are associated with contraction while others are associated with persistence and expansion. Together, these opposing forces shape the reservoir over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50cd5d7c1f22410445455e8d182f080f7187b65d" target='_blank'>Machine Learning Bolsters Evidence That D1, Nef, and Tat Influence HIV Reservoir Dynamics</a></td>
          <td>
            L. Cannon, Sophia Fehrman, Marilia Pinzone, S. Weissman, Una O'Doherty
          </td>
          <td>2024-01-23</td>
          <td>Pathogens and Immunity</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr id="The Human Immunodeficiency Virus (HIV) Viral Infectivity Factor (Vif) is a 192-amino acid accessory protein essential to viral replication which counteracts host APOBEC3 proteins. APOBEC3 proteins interfere with the replication of HIV, hepatitis C virus, hepatitis B virus and retrotransposons. Vif is a recent candidate target for therapeutic and preventative interventions in HIV/AIDS yet little is known about its clinical relevance. We describe the results of applying different machine learning algorithms (Apriori, Multifactor Dimensionality Reductor, C4.5, Artificial Neural Networks and ID3) to the search of associations between HIV-1 Vif protein attributes and clinical endpoints. Final iterations showed that the presence of mutations in BC Boxes, APOBEC motifs and Cullin5 binding motifs were together associated with higher initial CD4 T cells while mutations of specific APOBEC motifs coupled with the conservation of other APOBEC motifs were associated with lower historic CD4 T cells. Conservation of specific APOBEC motifs and BC boxes were linked to lower initial viral loads while different combinations of mutations in the Nuclear Localisation Inhibition Signal and BC Boxes were associated with higher historic viral loads. Further scrutiny of these combinations through traditional statistical methods revealed striking differences in both CD4 T cells and viral loads in patients stratified into those having the previous combinations. While artificial intelligence algorithms do not phase out traditional statistical methods, our Artificial Intelligence (AI)-based approach highlights their use at reducing the dimensionality of large and complex datasets and at proposing novel, unimaginable, associations of biological patterns with functional relevance or clinical roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f9f46049a315e8ea04aeef437c9ec5c16635db4" target='_blank'>Identification of HIV-1 Vif Protein Attributes Associated With CD4 T Cell Numbers and Viral Loads Using Artificial Intelligence Algorithms</a></td>
          <td>
            J. S. Altamirano-Flores, Sandra E Guerra-Palomares, P. G. HernÃ¡ndez-SÃ¡nchez, J. Ramirez-Garcialuna, J. R. Arguello-Astorga, D. Noyola, J. Cuevas-Tello, C. GarcÃ­a-SepÃºlveda
          </td>
          <td>2020-05-13</td>
          <td>IEEE Access</td>
          <td>6</td>
          <td>28</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e55ec2d84f9b48012d81fc6c681b590ee332c65" target='_blank'>CGI 2017 Editorial (TVCJ)</a></td>
          <td>
            Xiaoyang Mao, Daniel Thalmann, M. Gavrilova
          </td>
          <td>2017-06-01</td>
          <td>The Visual Computer</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bb86aba81933b8198b0234351e9c255869479d7" target='_blank'>Machine learning-selected variables associated with CD4 T cell recovery under antiretroviral therapy in very advanced HIV infection</a></td>
          <td>
            D. Romero-RodrÃ­guez, Carlos RamÃ­rez, Ivan Imaz-Rosshandler, C. Ormsby, Amy B Peralta-Prado, Gustavo Olvera-GarcÃ­a, Fernanda Cervantes, Daniela WÃ¼rsch-Molina, Jessica Romero-RodrÃ­guez, Wei Jiang, G. Reyes-TerÃ¡n, Enrique Espinosa
          </td>
          <td>2020-04-23</td>
          <td>Translational Medicine Communications</td>
          <td>1</td>
          <td>36</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/419cbb3dd77cb717749a4dc387c306524c4e5b61" target='_blank'>Machine learning models based on fluid immunoproteins that predict non-AIDS adverse events in people with HIV</a></td>
          <td>
            T. Premeaux, S. Bowler, Courtney Friday, C. Moser, Martin Hoenigl, Michael M. Lederman, A. L. Landay, S. Gianella, L. Ndhlovu
          </td>
          <td>2024-05-01</td>
          <td>iScience</td>
          <td>1</td>
          <td>41</td>
        </tr>
    
        <tr id="[This retracts the article DOI: 10.1155/2022/3179200.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67126e7421add9da8e58f1b15bbf63167a833ec" target='_blank'>Retracted: Research Progress on the Relationship between the NLRP3 Inflammasome and Immune Reconstitution in HIV-Infected Patients Receiving Antiretroviral Therapy</a></td>
          <td>
            Computational and Mathematical Methods in Medicine
          </td>
          <td>2023-06-28</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cb277352dab0473e59cce1d403c76aed685fd78" target='_blank'>CGI 2016 Editorial (TVCJ)</a></td>
          <td>
            Daniel Thalmann, Panos E. Trahanias, George Papagiannakis
          </td>
          <td>2016-06-01</td>
          <td>The Visual Computer</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7244a1a9df6b8a0c4b1aa072cf9681a18ab81cfe" target='_blank'>ushr: Understanding suppression of HIV in R</a></td>
          <td>
            Sinead E. Morris, L. Dziobek-Garrett, A. Yates, in collaboration with the EPIICAL Consortium
          </td>
          <td>2020-02-11</td>
          <td>BMC Bioinformatics</td>
          <td>4</td>
          <td>32</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>