---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-16 06:02:09 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>HIV-1 infection is associated with depletion of germinal center B cells and a decrease in IgA+ plasma cells in the GI tract</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdaaa8164fd46945764987c98bec307fda098c2" target='_blank'>A Reproducibility-Based Computational Framework Identifies an Inducible, Enhanced Antiviral State in Dendritic Cells from HIV-1 Elite Controllers</a></td>
          <td>
            E. Martín-Gayo, Michael Cole, Kellie E. Kolb, Zhengyu Ouyang, Jacqueline Cronin, S. Kazer, J. Ordovas-Montañes, M. Lichterfeld, B. Walker, N. Yosef, A. Shalek, Xu G. Yu
          </td>
          <td>2018-01-29</td>
          <td>Genome Biology</td>
          <td>41</td>
          <td>159</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8509b9159f6182e4b7c0253557a1a11eb56de119" target='_blank'>Bioinformatics and HIV Latency</a></td>
          <td>
            A. Ciuffi, P. Mohammadi, M. Golumbeanu, J. di Iulio, A. Telenti
          </td>
          <td>2015-01-14</td>
          <td>Current HIV/AIDS Reports</td>
          <td>12</td>
          <td>110</td>
        </tr>
    
        <tr id="[This corrects the article DOI: 10.1371/journal.pcbi.1003973.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba722b1705c6ea6ca37824c562081c67623fb36" target='_blank'>Correction: Comprehensive Sieve Analysis of Breakthrough HIV-1 Sequences in the RV144 Vaccine Efficacy Trial</a></td>
          <td>
            P. Edlefsen, M. Rolland, T. Hertz, S. Tovanabutra, A. Gartland, Allan C. deCamp, C. Magaret, H. Ahmed, R. Gottardo, M. Juraska, Connor McCoy, B. Larsen, E. Sanders-Buell, Chris Carrico, S. Menis, G. Kijak, M. Bose, Sequencing Team, M. Arroyo, Robert J O 'connell, S. Nitayaphan, P. Pitisuttithum, J. Kaewkungwal, S. Rerks-Ngarm, M. Robb, Tatsiana Kirys, I. Georgiev, P. Kwong, K. Scheffler, Sergei L. Kosakovsky Pond, J. Carlson, N. Michael, W. Schief, J. Mullins, Jerome H. Kim, P. Gilbert
          </td>
          <td>2016-01-01</td>
          <td>PLoS Computational Biology</td>
          <td>1</td>
          <td>117</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82117ef0e60e11080da326a0ba5412fc95fc9c2a" target='_blank'>Modeling and Remodeling the Cell: How Digital Twins and HCMV Can Elucidate the Complex Interactions of Viral Latency, Epigenetic Regulation, and Immune Responses</a></td>
          <td>
            Hana McMahon-Cole, Alicia E. Johnson, S. Aghamiri, T. Helikar, Lindsey B. Crawford
          </td>
          <td>2023-07-29</td>
          <td>Current Clinical Microbiology Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b09094e875e44fd29b0ff738a40bbb345c4cf2e" target='_blank'>A computational analysis of transcriptional profiles from CD8(+) T lymphocytes reveals potential mechanisms of HIV/AIDS control and progression</a></td>
          <td>
            S. Ivanov, D. Filimonov, O. Tarasova
          </td>
          <td>2021-04-25</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>3</td>
          <td>30</td>
        </tr>
    
        <tr id="HIV RNA viral load (VL) is an important outcome variable in studies of HIV infected persons. There exists only a handful of methods which classify patients by VL patterns.  Most methods place limits on the use of viral load measurements, are often specific to a particular study design, and do not account for complex, temporal variation. To address this issue, we propose a set of four unambiguous computable characteristics (features) of time-varying HIV viral load patterns, along with a novel centroid-based classification algorithm, which we use to classify a population of 1,576 HIV positive clinic patients into one of five different viral load patterns (clusters) often found in the literature: durably suppressed viral load (DSVL), sustained low viral load (SLVL), sustained high viral load (SHVL), high viral load suppression (HVLS), and rebounding viral load (RVL). The centroid algorithm summarizes these clusters in terms of their centroids and radii. We show that this allows new VL patterns to be assigned pattern membership based on the distance from the centroid relative to its radius, which we term radial normalization classification. This method has the benefit of providing an objective and quantitative method to assign VL pattern membership with a concise and interpretable model that aids clinical decision making. This method also facilitates meta-analyses by providing computably distinct HIV categories. Finally we propose that this novel centroid algorithm could also be useful in the areas of cluster comparison for outcomes research and data reduction in machine learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0927b99db41425f9e4ddb50be574ef673935bd6f" target='_blank'>Revealing HIV viral load patterns using unsupervised machine learning and cluster summarization</a></td>
          <td>
            S. Farooq, Samuel J. Weisenthal, Melissa Trayhan, Robert J. White, Kristen Bush, P. Mariuz, M. Zand
          </td>
          <td>2018-07-27</td>
          <td>F1000Research</td>
          <td>1</td>
          <td>33</td>
        </tr>
    
        <tr id="HIV RNA viral load (VL) is an important outcome variable in studies of HIV infected persons. There exists only a handful of methods which classify patients by viral load patterns. Most methods place limits on the use of viral load measurements, are often specific to a particular study design, and do not account for complex, temporal variation. To address this issue, we propose a set of four unambiguous computable characteristics (features) of time-varying HIV viral load patterns, along with a novel centroid-based classification algorithm, which we use to classify a population of 1,576 HIV positive clinic patients into one of five different viral load patterns (clusters) often found in the literature: durably suppressed viral load (DSVL), sustained low viral load (SLVL), sustained high viral load (SHVL), high viral load suppression (HVLS), and rebounding viral load (RVL). The centroid algorithm summarizes these clusters in terms of their centroids and radii. We show that this allows new viral load patterns to be assigned pattern membership based on the distance from the centroid relative to its radius, which we term radial normalization classification. This method has the benefit of providing an objective and quantitative method to assign viral load pattern membership with a concise and interpretable model that aids clinical decision making. This method also facilitates meta-analyses by providing computably distinct HIV categories. Finally we propose that this novel centroid algorithm could also be useful in the areas of cluster comparison for outcomes research and data reduction in machine learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bbb750d04dfab66fad37cac4116769313e39d8d" target='_blank'>Revealing patterns in HIV viral load data and classifying patients via a novel machine learning cluster summarization method</a></td>
          <td>
            S. Farooq, Samuel J. Weisenthal, Melissa Trayhan, Robert J. White, Kristen Bush, P. Mariuz, M. Zand
          </td>
          <td>2018-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>33</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>