---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-19 08:41:33 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>HIV-1 infection is associated with depletion of germinal center B cells and a decrease in IgA+ plasma cells in the GI tract</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8abba679e2263c0dd6b357ce0b9b74a569cd8b2" target='_blank'>Ten computational challenges in human virome studies.</a></td>
          <td>
            Yifan Wu, Yousong Peng
          </td>
          <td>2024-02-23</td>
          <td>Virologica Sinica</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="Summary
In solid-organ transplant recipients, a delicate balance between immunosuppression and immunocompetence must be achieved, which can be difficult to monitor in real-time. Shotgun sequencing of cell-free DNA (cfDNA) has been recently proposed as a new way to indirectly assess immune function in transplant recipients through analysis of the status of the human virome. To facilitate exploration of the utility of the human virome as an indicator of immune status, and to enable rapid, straightforward analyses by clinicians, we developed a fully automated computational pipeline, EzMap, for performing metagenomic analysis of the human virome. EzMap combines a number of tools to clean, filter, and subtract WGS reads by mapping to a reference human assembly. The relative abundance of each virus present is estimated using a maximum likelihood approach that accounts for genome size, and results are presented with interactive visualizations and taxonomy-based summaries that enable rapid insights. The pipeline is automated to run on both workstations and computing clusters for all steps. EzMap automates an otherwise tedious and time-consuming protocol and aims to facilitate rapid and reproducible insights from cfDNA.


Availability and Implementation
EzMap is freely available at https://github.com/dekoning-lab/ezmap.


Contact
jason.dekoning@ucalgary.ca.


Supplementary information
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cda26283ec1d50b9245ff29644d68027d6f25d" target='_blank'>EzMap: a simple pipeline for reproducible analysis of the human virome</a></td>
          <td>
            Patrick Czeczko, S. Greenway, A. J. D. Koning
          </td>
          <td>2017-08-15</td>
          <td>Bioinformatics</td>
          <td>6</td>
          <td>28</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42062b7298efe0a9a4e5ce471bd6c8d7131d6cda" target='_blank'>Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients</a></td>
          <td>
            Pierre Bost, Amir Giladi, Yang Liu, Yanis Bendjelal, Gang Xu, Eyal David, Ronnie Blecher-Gonen, Merav Cohen, Chiara Medaglia, Hanjie Li, A. Deczkowska, Shuye Zhang, B. Schwikowski, Zheng Zhang, I. Amit
          </td>
          <td>2020-05-08</td>
          <td>Cell</td>
          <td>404</td>
          <td>97</td>
        </tr>
    
        <tr id="Understanding the link between the human gut virome and diseases has garnered significant interest in the research community. Extracting virus-related information from metagenomic sequencing data is crucial for unravelling virus composition, host interactions, and disease associations. However, current metagenomic analysis workflows for viral genomes vary in effectiveness, posing challenges for researchers seeking the most up-to-date tools. To address this, we present ViromeFlowX, a user-friendly Nextflow workflow that automates viral genome assembly, identification, classification, and annotation. This streamlined workflow integrates cutting-edge tools for processing raw sequencing data for taxonomic annotation and functional analysis. Application to a dataset of 200 metagenomic samples yielded high-quality viral genomes. ViromeFlowX enables efficient mining of viral genomic data, offering a valuable resource to investigate the gut virome’s role in virus-host interactions and virus-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ea369c597f3fa29579967e6b4bff2cb63bdf5c" target='_blank'>ViromeFlowX: a Comprehensive Nextflow-based Automated Workflow for Mining Viral Genomes from Metagenomic Sequencing Data</a></td>
          <td>
            Xiaokai Wang, Zhimin Ding, Ying Yang, Lifeng Liang, Yingshuai Sun, Chaojian Hou, Yuning Zheng, Yan Xia, Lixin Dong
          </td>
          <td>2024-02-01</td>
          <td>Microbial Genomics</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20d2a886a2819555f1b29519193323bf6535394" target='_blank'>The Immune Signatures data resource, a compendium of systems vaccinology datasets</a></td>
          <td>
            J. Diray-Arce, H. Miller, E. Henrich, B. Gerritsen, Matthew P. Mulè, S. Fourati, Jeremy P. Gygi, T. Hagan, L. Tomalin, D. Rychkov, Dmitri Kazmin, D. G. Chawla, Hailong Meng, P. Dunn, John Campbell, , Alison Deckhut-Augustine, R. Gottardo, Elias K. Haddad, D. Hafler, Eva Harris, Donna Farber, Ofer Levy, J. McElrath, R. R. Montgomery, Bjoern Peters, Adeeb Rahman, Elaine F. Reed, N. Rouphael, A. Fernández-Sesma, Alessandro Sette, Ken Stuart, A. Togias, John S. Tsang, M. Sarwal, John S. Tsang, Ofer Levy, B. Pulendran, R. Sékaly, A. Floratos, R. Gottardo, S. Kleinstein, M. Suárez-Fariñas
          </td>
          <td>2022-10-20</td>
          <td>Scientific Data</td>
          <td>14</td>
          <td>100</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a952e012e1800bbd1317b1a754da406bbe7cda" target='_blank'>Systems analysis of human innate immunity in COVID-19</a></td>
          <td>
            Sophie Müller, J. Schultze
          </td>
          <td>2023-05-01</td>
          <td>Seminars in Immunology</td>
          <td>0</td>
          <td>92</td>
        </tr>
    
        <tr id="I survey several illuminating applications of diverse next generation sequencing, phylogenetic, machine learning, and network analysis approaches to the study of HIV-1 at the level of a single individual, populations of infected individuals, and the a geographic epidemic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05ae3c9f0f4159c245dcca9a5fc2262129307daf" target='_blank'>Computational analysis of HIV-1 evolution and epidemiology</a></td>
          <td>
            Sergei L Kosakovsky Pond
          </td>
          <td>2011-11-12</td>
          <td>2011 IEEE International Conference on Bioinformatics and Biomedicine Workshops (BIBMW)</td>
          <td>2</td>
          <td>38</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32015c79bd160189ddb13fa99f034c432b94c59" target='_blank'>Probing infectious disease by single-cell RNA sequencing: Progresses and perspectives</a></td>
          <td>
            Geyang Luo, Q. Gao, Shuye Zhang, Bo Yan
          </td>
          <td>2020-10-21</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>35</td>
          <td>42</td>
        </tr>
    
        <tr id="ABSTRACT Many cell lines commonly used for biological studies have been found to harbor exogenous agents such as the human tumor viruses Epstein-Barr virus (EBV) and human papillomavirus. Nevertheless, broad-based, unbiased approaches to globally assess the presence of ectopic organisms within cell model systems have not previously been available. We reasoned that high-throughput sequencing should provide unparalleled insights into the microbiomes of tissue culture cell systems. Here we have used our RNA-seq analysis pipeline, PARSES (Pipeline for Analysis of RNA-Seq Exogenous Sequences), to investigate the presence of ectopic organisms within two EBV-positive B-cell lines commonly used by EBV researchers. Sequencing data sets from both the Akata and JY B-cell lines were found to contain reads for EBV, and the JY data set was found to also contain reads from the murine leukemia virus (MuLV). Further investigation revealed that MuLV transcription in JY cells is highly active. We also identified a number of MuLV alternative splicing events, and we uncovered evidence of APOBEC3G (apolipoprotein B mRNA-editing enzyme, catalytic polypeptide-like 3G)-dependent DNA editing. Finally, reverse transcription-PCR analysis showed the presence of MuLV in three other human B-cell lines (DG75, Ramos, and P3HR1 Cl.13) commonly used by investigators in the Epstein-Barr virus field. We believe that a thorough examination of tissue culture microbiomes using RNA-seq/PARSES-like approaches is critical for the appropriate utilization of these systems in biological studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eded2800549f6b696c7558dca9b3f32ffd9629f3" target='_blank'>Detection of Murine Leukemia Virus in the Epstein-Barr Virus-Positive Human B-Cell Line JY, Using a Computational RNA-Seq-Based Exogenous Agent Detection Pipeline, PARSES</a></td>
          <td>
            Zhen Lin, Adriane Puetter, Joseph R. Coco, Guorong Xu, Michael J. Strong, Xia Wang, Claire Fewell, M. Baddoo, Christopher M. Taylor, E. Flemington
          </td>
          <td>2012-01-11</td>
          <td>Journal of Virology</td>
          <td>30</td>
          <td>51</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>